<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>Microbiota</title>
    <meta charset="utf-8" />
    <meta name="author" content="Marco Chiapello" />
    <script src="libs/header-attrs/header-attrs.js"></script>
    <link href="libs/tile-view/tile-view.css" rel="stylesheet" />
    <script src="libs/tile-view/tile-view.js"></script>
    <link rel="stylesheet" href="scrollable.css" type="text/css" />
    <link rel="stylesheet" href="mtheme_max.css" type="text/css" />
    <link rel="stylesheet" href="fonts_mtheme_max.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">






class: title-slide



# .font170[MICROBIOMA E MICROBIOTA]


&lt;br&gt;&lt;br&gt;&lt;br&gt;&lt;br&gt;
.marco[
Marco Chiapello
&lt;br&gt;
2020-11-17
]


.marco[
.center[.font90[[[&lt;svg style="height:0.8em;top:.04em;position:relative;fill:steelblue;" viewBox="0 0 448 512"&gt;&lt;path d="M94.12 315.1c0 25.9-21.16 47.06-47.06 47.06S0 341 0 315.1c0-25.9 21.16-47.06 47.06-47.06h47.06v47.06zm23.72 0c0-25.9 21.16-47.06 47.06-47.06s47.06 21.16 47.06 47.06v117.84c0 25.9-21.16 47.06-47.06 47.06s-47.06-21.16-47.06-47.06V315.1zm47.06-188.98c-25.9 0-47.06-21.16-47.06-47.06S139 32 164.9 32s47.06 21.16 47.06 47.06v47.06H164.9zm0 23.72c25.9 0 47.06 21.16 47.06 47.06s-21.16 47.06-47.06 47.06H47.06C21.16 243.96 0 222.8 0 196.9s21.16-47.06 47.06-47.06H164.9zm188.98 47.06c0-25.9 21.16-47.06 47.06-47.06 25.9 0 47.06 21.16 47.06 47.06s-21.16 47.06-47.06 47.06h-47.06V196.9zm-23.72 0c0 25.9-21.16 47.06-47.06 47.06-25.9 0-47.06-21.16-47.06-47.06V79.06c0-25.9 21.16-47.06 47.06-47.06 25.9 0 47.06 21.16 47.06 47.06V196.9zM283.1 385.88c25.9 0 47.06 21.16 47.06 47.06 0 25.9-21.16 47.06-47.06 47.06-25.9 0-47.06-21.16-47.06-47.06v-47.06h47.06zm0-23.72c-25.9 0-47.06-21.16-47.06-47.06 0-25.9 21.16-47.06 47.06-47.06h117.84c25.9 0 47.06 21.16 47.06 47.06 0 25.9-21.16 47.06-47.06 47.06H283.1z"/&gt;&lt;/svg&gt;](https://join.slack.com/t/ipma-2020/shared_invite/zt-j9o8ig7k-LN1~U3d6QJztquYbC6iW7w) Slack (https://bit.ly/3eRDBRL) - &lt;svg style="height:0.8em;top:.04em;position:relative;fill:steelblue;" viewBox="0 0 512 512"&gt;&lt;path d="M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7 22.4 17.4 52.1 39.5 154.1 113.6 21.1 15.4 56.7 47.8 92.2 47.6 35.7.3 72-32.8 92.3-47.6 102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4 132.7-96.3 142.8-104.7 173.4-128.7 5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9 30.6 23.9 40.7 32.4 173.4 128.7 16.8 12.2 50.2 41.8 73.4 41.4z"/&gt;&lt;/svg&gt;](mailto:marco.chiapello@unito.it) marco.chiapello@unito.it]
]
]

???

.n30[

- Sarò uno dei vostri docenti per il corso "Interazioni tra piante, microrganismi e ambiente"
]

---
layout: true

# Last lesson recap

---
&lt;br&gt;

.pull-left[
.m0p[
.h25o[
1. Plant microbiota changes spatially and temporally

1. Plant compartment is a major selective force that shapes the composition of plant-associated microbiota

1. Plants and their associated microorganisms form a holobiont

]
]
]

---
&lt;br&gt;

.pull-left[
.m0p[
.h25o[
1. Plant microbiota change spatially and temporally

1. .opacity10[ Plant compartment is a major selective force that shapes the composition of plant-associated microbiota]

1. .opacity10[ Plants and their associated microorganisms form a holobiont]

]
]
]

.pull-right[

.m0p[
&lt;img src="images/PlantMicrobiomeComposition.png" width="230px" height="230px" style="display: block; margin: auto;" /&gt;

&lt;img src="images/plantMicrobiotaTemporal.png" width="300px" height="270px" style="display: block; margin: auto;" /&gt;
]
]

---

&lt;br&gt;

.pull-left[
.m0p[
.h25o[
1. .opacity10[ Plant microbiota change spatially and temporally]

1. Plant compartment is a major selective force that shapes the composition of plant-associated microbiota

1. .opacity10[ Plants and their associated microorganisms form a holobiont]

]
]
]

.pull-right[

&lt;img src="images/bacterialDiv.png" width="350px" style="display: block; margin: auto;" /&gt;

&lt;img src="images/fungalDiv.png" width="350px" style="display: block; margin: auto;" /&gt;
]

---

.pull-left[
.m0p[
.h25o[
1. .opacity10[ Plant microbiota change spatially and temporally]

1. .opacity10[ Plant compartment is a major selective force that shapes the composition of plant-associated microbiota]

1. Plants and their associated microorganisms form a holobiont

]
]
]

.pull-right[
&lt;br&gt;
&lt;img src="images/holobiont1.png" width="800px" style="display: block; margin: auto;" /&gt;

]

---
layout: false
class: clear 

.pull-left-wide[
.font180[.bold[Agenda]]



- .font140[How do you study the microbiota composition?]

- .font140[What can the microbiota potentially do?]

- .font140[What are they doing?]
]




---
layout: true

# How do you study the microbiota composition?



---
class: inverse, middle, center

----

---

.center[**There are two major approaches to assess the microbiota composition**]

.h20[
&lt;img src="images/microbiotaCompositionTecniques.png" width="520px" style="display: block; margin: auto;" /&gt;

.right[.font60[(Lebeis, 2014)]]
]

---

.pull-left[
&lt;img src="images/microbiotaCompositionTecniques.png" width="520px" style="display: block; margin: auto;" /&gt;
]

.pull-right[
**Culture dependent**

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; This approach isolates individual microbes 

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm121.6 313.1c4.7 4.7 4.7 12.3 0 17L338 377.6c-4.7 4.7-12.3 4.7-17 0L256 312l-65.1 65.6c-4.7 4.7-12.3 4.7-17 0L134.4 338c-4.7-4.7-4.7-12.3 0-17l65.6-65-65.6-65.1c-4.7-4.7-4.7-12.3 0-17l39.6-39.6c4.7-4.7 12.3-4.7 17 0l65 65.7 65.1-65.6c4.7-4.7 12.3-4.7 17 0l39.6 39.6c4.7 4.7 4.7 12.3 0 17L312 256l65.6 65.1z"/&gt;&lt;/svg&gt; The community diversity estimate is limited

]

---

.pull-left[
**Culture independent**

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt;  The ability to identify and quantify community members, even down to extremely rare taxa

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt;  Insights beyond the information provided by individual microbes

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm121.6 313.1c4.7 4.7 4.7 12.3 0 17L338 377.6c-4.7 4.7-12.3 4.7-17 0L256 312l-65.1 65.6c-4.7 4.7-12.3 4.7-17 0L134.4 338c-4.7-4.7-4.7-12.3 0-17l65.6-65-65.6-65.1c-4.7-4.7-4.7-12.3 0-17l39.6-39.6c4.7-4.7 12.3-4.7 17 0l65 65.7 65.1-65.6c4.7-4.7 12.3-4.7 17 0l39.6 39.6c4.7 4.7 4.7 12.3 0 17L312 256l65.6 65.1z"/&gt;&lt;/svg&gt; This approach does not isolate individual microbes 

]


.pull-right[
&lt;img src="images/microbiotaCompositionTecniques.png" width="520px" style="display: block; margin: auto;" /&gt;
]

---

.center[.font150[Microbial DNA metabarcoding]]

&gt; **DNA metabarcoding** is a method of **DNA barcoding** that uses universal genetic markers to identify DNA of a mixture of organisms

--

&lt;br&gt;
.content-box-grey[What is it the &lt;br&gt; **DNA barcoding**?]

---

.center[.font150[Microbial DNA metabarcoding]]

- **DNA barcoding** is a method of species identification using a short section of DNA from a specific gene 

- An individual sequence can be used to uniquely identify an organism by comparison with a reference library of classified DNA sequences


&lt;img src="images/DNAbarcoding.png" width="520px" style="display: block; margin: auto;" /&gt;

---

.center[.font150[Microbial DNA metabarcoding]]

.pull-left[
**Marker selection**

.n30[
- Markers used for DNA barcoding are called barcodes and their selection is crucial

- Ideally, one gene sequence would be used for developing universal PCR primers for all taxonomic group
  
- However, no such gene region has been found yet!

- Different gene regions are used to identify the different organismal groups using barcoding
]
]

.pull-right[

&lt;img src="images/DNAbarcoding2.png" width="200px" style="display: block; margin: auto;" /&gt;

.right[.font50[(Mosa et al., 2019)]]
]

---

.center[.font150[Microbial DNA metabarcoding]]

**Marker selection**

| Organism | Region |
|----------|--------|
| animals  | cytochrome c oxidase I|
| fungi    |  internal transcribed spacer (ITS) rRNA |
| plants   | RuBisCO |
| prokaryotes | 16S rRNA |
| microbial eukaryotes | 18S rRNA |


.font80[When barcoding is used to identify organisms from a sample containing DNA from more than one organism, the term DNA metabarcoding is used]


---

.center[.font150[Microbial DNA metabarcoding]]

.pull-left[
.m0p[
**Bioinformatic analysis**
]
.n30[
- After sequencing, raw data must be processed (**quality and cleaning step**)

- Samples are clustered in operational taxonomic unit based on identity threshold (**OTU formation**)

- Using "the Basic Local Alignment Search Tool (BLAST)" regions of similarity between sequences and reference databases are identified (**annotation step**)    

- Microbiota composition definition (**richness and diversity step**)

]
]

.pull-right[

.h20[
.m0p[
.right[.font50[(Jesse et al., 2018)]]     
]
&lt;img src="images/OTU.png" width="220px" style="display: block; margin: auto;" /&gt;


]]



???

.n30[
- These databases contain the DNA barcodes assigned to previously identified taxa

- An operational taxonomic unit (OTU) is an operational definition used to classify groups of closely related individuals
]

---
layout: true

# What can the microbiota potentially do?

---

class: inverse, middle, center

----

---

.center[.font120[Metagenomics]]

&lt;img src="images/metagenomics1.png" width="720px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Ye, Siddle, Park, and Sabeti, 2019)]]
.h25[
- **Metagenomics** is the study of genetic material recovered directly from environmental samples

- The collection of sequenced genes from the environment could be analyzed as a **single genome**
  
- Metagenomics **does not require the isolation** and lab cultivation of individual species

- Due to its random nature, metagenomics, allows that many **under-represented organisms will be detected**
]

---

.center[Metagenomics vs Barcoding]
.pull-left[
.m0p[
&lt;img src="images/compareBarcodingMetagenome.png" width="450px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Gill, Baldock, Brown, Cresswell, Dicks, Fountain, Garratt, Gough, Heard, Holland, Ollerton, Stone, Tang, Vanbergen, Vogler, Woodward, Arce, Boatman, Brand-Hardy, and Potts, 2016)]]
]]


 .pull-right[
.h21[

- Metagenomics does not need the marker selection step

- Metagenomics sequences all the genes present in the sample from each organism

- Metagenomics can help to reconstruct large fragments or even complete genomes from organisms in a community

- Metagenomics allows the characterization of a large number of coding and non-coding sequences that can be used to decipher the microbial diversity or to understand its metabolic potential
]
]

---



&lt;img src="images/CompareHTS.png" width="920px" style="display: block; margin: auto;" /&gt;

---

.center[
.middle[
&lt;iframe width="840" height="473" src="https://www.youtube.com/embed/fCd6B5HRaZ8" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen&gt;&lt;/iframe&gt;
]
]
---

.center[.font120[Genomes assembly]]

.m0p[
&lt;img src="images/Align.png" width="620px" style="display: block; margin: auto;" /&gt;
]
---
.font120[**Functional Metagenomics Analysis**]

.pull-left[
.m10ulp[
Workflow:

- Metagenomic assembly

- Open Reading Frames (ORF) prediction

- Annotation step performed by homology detection (at proteomic level)

- Reconstruction of metabolic pathways is achieved from enzyme-coding genes 
]]

.pull-right[
&lt;img src="images/metagenomicsPipeline.png" width="380px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Roumpeka, Wallace, Escalettes, Fotheringham, and Watson, 2017)]]
]

???

.n30[
- In metagenomica, il binning è il processo di raggruppamento di letture o contigs e di assegnarli a unità tassonomiche operative
]

---

.font120[**Functional Metagenomics Analysis**]

.pull-left[
.m10ulp[
Workflow:

- Metagenomic assembly

- Open Reading Frames (ORF) prediction

- Annotation step performed by homology detection (at proteomic level)

- Reconstruction of metabolic pathways is achieved from enzyme-coding genes 
]]

.pull-right[
&lt;img src="images/functionalMetagenomics.png" width="370px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Castañeda and Barbosa, 2017)]]
]

---
layout: true

# What are they doing?

---

class: inverse, center, middle

----

---

.white[Metatranscriptomics]

&lt;img src="images/Meta1.png" width="620px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Vanwonterghem, Jensen, Ho, Batstone, and Tyson, 2014)]]

---

Metatranscriptomics

&lt;img src="images/Meta2.png" width="620px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Vanwonterghem, Jensen, Ho, et al., 2014)]]

---

Metatranscriptomics
.pull-left[
.h25[
- Metatranscriptomics is the science that studies gene expression of microbes within natural environments

- It allows to obtain whole gene expression profiling of complex microbial communities

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; The advantage of metatranscriptomics, over metagenomics, is that it can provide information about differences in the active functions of microbial communities 

-  The overview of the gene expression in a given sample is obtained by capturing the total mRNA of the microbiome and by performing a whole metatranscriptomics shotgun sequencing.
]
]

.pull-right[
.m0p[
&lt;img src="images/metatrranscriptomePL.png" width="380px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Pérez-Pantoja and Tamames, 2015)]]
]
]

---

Metaproteomics

&lt;img src="images/Meta3.png" width="620px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Vanwonterghem, Jensen, Ho, et al., 2014)]]

---

Metaproteomics

.pull-left[
.h25[
- Metaproteomics is the study of all protein samples recovered directly from environmental sources

- Metaproteomics have rapidly increased in recent years due to many technological advances in mass spectrometry (MS)

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; Metatranscriptomics assess the “expressed” metabolism and physiology of microbial community members, but also allows quantification of per-species biomass to determine community structure

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; Proteomics quantifies the real proteins expression level, while mRNA may be degraded or translated inefficiently, resulting in a incorrect estimation of protein expression level
]
]

.pull.right[
&lt;img src="images/metaproteomics.png" width="480px" style="display: block; margin: auto;" /&gt;
.right[.font30[https://www.ecu.edu.au/schools/science/research-activity/research-support/proteomics/what-is-lc-ms-proteomics]]
]

---

Metabolomics

&lt;img src="images/Meta4.png" width="600px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Vanwonterghem, Jensen, Ho, et al., 2014)]]

---

Metabolomics

.pull-left[
.m0p[
.h25[

- Metabolomics is the study of the complete set of metabolites composition (the metabolome) of a cell type, tissue, or biological fluid

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; The metabolites (&lt;1.5 kDa) are the intermediates or end products of multiple enzymatic reactions and therefore are the most informative proxies of the biochemical activity of an organism

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm121.6 313.1c4.7 4.7 4.7 12.3 0 17L338 377.6c-4.7 4.7-12.3 4.7-17 0L256 312l-65.1 65.6c-4.7 4.7-12.3 4.7-17 0L134.4 338c-4.7-4.7-4.7-12.3 0-17l65.6-65-65.6-65.1c-4.7-4.7-4.7-12.3 0-17l39.6-39.6c4.7-4.7 12.3-4.7 17 0l65 65.7 65.1-65.6c4.7-4.7 12.3-4.7 17 0l39.6 39.6c4.7 4.7 4.7 12.3 0 17L312 256l65.6 65.1z"/&gt;&lt;/svg&gt; The metabolome consists of multiple compounds. Determine the entire set of metabolites is extremely difficult, further complicating the analyses is the dynamic nature of these metabolites.
]
]]

.pull-rigth[
&lt;br&gt;&lt;br&gt;
&lt;img src="images/metabolomics.png" width="480px" style="display: block; margin: auto;" /&gt;
]
.right[.font50[(Alonso, Marsal, and Julià, 2015)]]


???

.n30[
- hydrophilic
  carbohydrates, volatile alcohols, ketones, amino and non- amino organic acids, hydrophobic lipids and complex natural compounds (secondary metabolites such as antibiotics, pigments, non-ribosomal peptides, cofactors)
]

---

Stable/radioactive isotopes

&lt;img src="images/Meta5.png" width="600px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Vanwonterghem, Jensen, Ho, et al., 2014)]]

---

Stable/radioactive isotopes

.pull-left[
.m0p[
.h25[

- Stable-isotope probing (SIP) is a technique in microbial ecology for tracing fluxes of nutrients in biogeochemical cycling by microorganisms using heavy stable isotopes

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; SIP is an approach that can identify microbial populations with a defined function

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; The use of either DNA-SIP, RNA-SIP or protein-SIP can enable both the phylogenetic identification and the key metabolic genes/proteins

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm121.6 313.1c4.7 4.7 4.7 12.3 0 17L338 377.6c-4.7 4.7-12.3 4.7-17 0L256 312l-65.1 65.6c-4.7 4.7-12.3 4.7-17 0L134.4 338c-4.7-4.7-4.7-12.3 0-17l65.6-65-65.6-65.1c-4.7-4.7-4.7-12.3 0-17l39.6-39.6c4.7-4.7 12.3-4.7 17 0l65 65.7 65.1-65.6c4.7-4.7 12.3-4.7 17 0l39.6 39.6c4.7 4.7 4.7 12.3 0 17L312 256l65.6 65.1z"/&gt;&lt;/svg&gt; A major challenge associated with SIP is a very limited availability and high cost of labeled substrates. SIP is also very labor-intensive and low-throughput using current techniques
]
]]

.pull-rigth[
&lt;img src="images/sip1.png" width="280px" style="display: block; margin: auto;" /&gt;
]
.right[.font50[(Uhlik, Leewis, Strejcek, Musilova, Mackova, Leigh, and Macek, 2013)]]

---

Stable/radioactive isotopes

.m0p[
&lt;img src="images/sip2.png" width="570px" style="display: block; margin: auto;" /&gt;
.right[.font50[(el Zahar Haichar, Heulin, Guyonnet, et al., 2016)]]
]

---
layout: false
class: clear, middle, center

.m0p[
&lt;img src="images/sb1.png" width="850px" style="display: block; margin: auto;" /&gt;
]

???
metafora della torta e dei suoi ingredienti

---
class: clear


.center[.font150[System biology]]

.pull-left[
.m0p[
.h25u[
- .font80[Systems biology studies biological systems by systematically perturbing them]    

- .font80[Monitoring the gene, protein, metabolite, and informational pathway responses]    

- .font80[Integrating these data]    

- .font80[Formulating mathematical models that describe the structure of the system]    

- .font80[Predict its response to individual perturbations]
  ]]]

.pull-right[
.m0p[
&lt;img src="images/sb2.png" width="670px" style="display: block; margin: auto;" /&gt;

]
]

---

layout: true

# Questions about the lesson


---


.m0tbp[
.center[.font50px[&lt;svg style="height:0.8em;top:.04em;position:relative;fill:black;" viewBox="0 0 512 512"&gt;&lt;path d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zM262.655 90c-54.497 0-89.255 22.957-116.549 63.758-3.536 5.286-2.353 12.415 2.715 16.258l34.699 26.31c5.205 3.947 12.621 3.008 16.665-2.122 17.864-22.658 30.113-35.797 57.303-35.797 20.429 0 45.698 13.148 45.698 32.958 0 14.976-12.363 22.667-32.534 33.976C247.128 238.528 216 254.941 216 296v4c0 6.627 5.373 12 12 12h56c6.627 0 12-5.373 12-12v-1.333c0-28.462 83.186-29.647 83.186-106.667 0-58.002-60.165-102-116.531-102zM256 338c-25.365 0-46 20.635-46 46 0 25.364 20.635 46 46 46s46-20.636 46-46c0-25.365-20.635-46-46-46z"/&gt;&lt;/svg&gt;]]
]

.marco[
.center[.font90[[[&lt;svg style="height:0.8em;top:.04em;position:relative;fill:steelblue;" viewBox="0 0 448 512"&gt;&lt;path d="M94.12 315.1c0 25.9-21.16 47.06-47.06 47.06S0 341 0 315.1c0-25.9 21.16-47.06 47.06-47.06h47.06v47.06zm23.72 0c0-25.9 21.16-47.06 47.06-47.06s47.06 21.16 47.06 47.06v117.84c0 25.9-21.16 47.06-47.06 47.06s-47.06-21.16-47.06-47.06V315.1zm47.06-188.98c-25.9 0-47.06-21.16-47.06-47.06S139 32 164.9 32s47.06 21.16 47.06 47.06v47.06H164.9zm0 23.72c25.9 0 47.06 21.16 47.06 47.06s-21.16 47.06-47.06 47.06H47.06C21.16 243.96 0 222.8 0 196.9s21.16-47.06 47.06-47.06H164.9zm188.98 47.06c0-25.9 21.16-47.06 47.06-47.06 25.9 0 47.06 21.16 47.06 47.06s-21.16 47.06-47.06 47.06h-47.06V196.9zm-23.72 0c0 25.9-21.16 47.06-47.06 47.06-25.9 0-47.06-21.16-47.06-47.06V79.06c0-25.9 21.16-47.06 47.06-47.06 25.9 0 47.06 21.16 47.06 47.06V196.9zM283.1 385.88c25.9 0 47.06 21.16 47.06 47.06 0 25.9-21.16 47.06-47.06 47.06-25.9 0-47.06-21.16-47.06-47.06v-47.06h47.06zm0-23.72c-25.9 0-47.06-21.16-47.06-47.06 0-25.9 21.16-47.06 47.06-47.06h117.84c25.9 0 47.06 21.16 47.06 47.06 0 25.9-21.16 47.06-47.06 47.06H283.1z"/&gt;&lt;/svg&gt;](https://join.slack.com/t/ipma-2020/shared_invite/zt-j9o8ig7k-LN1~U3d6QJztquYbC6iW7w) Slack (https://bit.ly/3eRDBRL) - &lt;svg style="height:0.8em;top:.04em;position:relative;fill:steelblue;" viewBox="0 0 512 512"&gt;&lt;path d="M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7 22.4 17.4 52.1 39.5 154.1 113.6 21.1 15.4 56.7 47.8 92.2 47.6 35.7.3 72-32.8 92.3-47.6 102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4 132.7-96.3 142.8-104.7 173.4-128.7 5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9 30.6 23.9 40.7 32.4 173.4 128.7 16.8 12.2 50.2 41.8 73.4 41.4z"/&gt;&lt;/svg&gt;](mailto:marco.chiapello@unito.it) marco.chiapello@unito.it]
]
]


    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="libs/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"ratio": "16:9",
"highlightStyle": "github",
"highlightLines": true
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
