---
title: "Microbiota"
subtitle: "Lesson 1"
author: "Marco Chiapello"
date: "???"
output:
  xaringan::moon_reader:
    css: ["scrollable.css", "mtheme_max.css", "fonts_mtheme_max.css"]
    self_contained: false
    lib_dir: libs
    chakra: libs/remark-latest.min.js
    nature:
      ratio: '16:9'
      highlightStyle: github
      highlightLines: true
    countIncrementalSlides: false
    seal: false  
---

```{r setup, include=FALSE, cache=FALSE}
# TO render: rmarkdown::render('Lesson1.Rmd', 'xaringan::moon_reader')

# set working directory to docs folder
setwd(here::here("docs"))

# Set global R options
options(htmltools.dir.version = FALSE, servr.daemon = TRUE)

# Set global knitr chunk options
knitr::opts_chunk$set(
  fig.align = "center", 
  cache = TRUE,
  error = FALSE,
  message = FALSE, 
  warning = FALSE, 
  collapse = TRUE 
)

# This is good for getting the ggplot background consistent with
# the html background color
library(ggplot2)
thm <- theme_bw()
theme_set(thm)

library(RefManageR)
BibOptions(check.entries = FALSE,
           bib.style = "authoryear",
           cite.style = "authoryear",
           style = "markdown",
           hyperlink = FALSE,
           dashed = FALSE)
bib <- ReadBib("biblio/bib.bib", check = FALSE)
```

```{r xaringanExtra, echo=FALSE}
library(xaringanExtra)
use_tile_view() 
```

class: title-slide



# .font170[MICROBIOMA E MICROBIOTA]


<br><br><br><br>
.marco[
Marco Chiapello
<br>
2020-11-17
]


.marco[
.center[.font90[[[`r fontawesome::fa("slack", fill = "steelblue")`](https://join.slack.com/t/ipma-2020/shared_invite/zt-j9o8ig7k-LN1~U3d6QJztquYbC6iW7w) Slack (https://bit.ly/3eRDBRL) - `r fontawesome::fa("envelope", fill = "steelblue")`](mailto:marco.chiapello@unito.it) marco.chiapello@unito.it]
]
]

???

.n30[

- Sarò uno dei vostri docenti per il corso "Interazioni tra piante, microrganismi e ambiente"
]

---
layout: true

# Last lesson recap

---
<br>

.pull-left[
.m0p[
.h25o[
1. Plant microbiota changes spatially and temporally

1. Plant compartment is a major selective force that shapes the composition of plant-associated microbiota

1. Plants and their associated microorganisms form a holobiont

]
]
]

---
<br>

.pull-left[
.m0p[
.h25o[
1. Plant microbiota change spatially and temporally

1. .opacity10[ Plant compartment is a major selective force that shapes the composition of plant-associated microbiota]

1. .opacity10[ Plants and their associated microorganisms form a holobiont]

]
]
]

.pull-right[

.m0p[
```{r echo = FALSE, out.height="230px", out.width="230px"}
knitr::include_graphics("images/PlantMicrobiomeComposition.png")
```

```{r echo = FALSE, out.height="270px", out.width="300px"}
knitr::include_graphics("images/plantMicrobiotaTemporal.png")
```
]
]

---

<br>

.pull-left[
.m0p[
.h25o[
1. .opacity10[ Plant microbiota change spatially and temporally]

1. Plant compartment is a major selective force that shapes the composition of plant-associated microbiota

1. .opacity10[ Plants and their associated microorganisms form a holobiont]

]
]
]

.pull-right[

```{r echo=FALSE, out.width="350px"}
knitr::include_graphics("images/bacterialDiv.png")
```

```{r echo=FALSE, out.width="350px"}
knitr::include_graphics("images/fungalDiv.png")
```
]

---

.pull-left[
.m0p[
.h25o[
1. .opacity10[ Plant microbiota change spatially and temporally]

1. .opacity10[ Plant compartment is a major selective force that shapes the composition of plant-associated microbiota]

1. Plants and their associated microorganisms form a holobiont

]
]
]

.pull-right[
<br>
```{r echo=FALSE, out.width="800px"}
knitr::include_graphics("images/holobiont1.png")
```

]

---
layout: false
class: clear 

.pull-left-wide[
.font180[.bold[Agenda]]



- .font140[How do you study the microbiota composition?]

- .font140[What can the microbiota potentially do?]

- .font140[What are they doing?]
]




---
layout: true

# How do you study the microbiota composition?



---
class: inverse, middle, center

----

---

.center[**There are two major approaches to assess the microbiota composition**]

.h20[
```{r echo=FALSE, out.width="520px"}
knitr::include_graphics("images/microbiotaCompositionTecniques.png")
```

.right[.font60[`r Cite(bib, key = "Lebeis_2014")`]]
]

---

.pull-left[
```{r echo=FALSE, out.width="520px"}
knitr::include_graphics("images/microbiotaCompositionTecniques.png")
```
]

.pull-right[
**Culture dependent**

- `r fontawesome::fa("plus-circle", fill = "green")` This approach isolates individual microbes 

- `r fontawesome::fa("times-circle", fill = "red")` The community diversity estimate is limited

]

---

.pull-left[
**Culture independent**

- `r fontawesome::fa("plus-circle", fill = "green")`  The ability to identify and quantify community members, even down to extremely rare taxa

- `r fontawesome::fa("plus-circle", fill = "green")`  Insights beyond the information provided by individual microbes

- `r fontawesome::fa("times-circle", fill = "red")` This approach does not isolate individual microbes 

]


.pull-right[
```{r echo=FALSE, out.width="520px"}
knitr::include_graphics("images/microbiotaCompositionTecniques.png")
```
]

---

.center[.font150[Microbial DNA metabarcoding]]

> **DNA metabarcoding** is a method of **DNA barcoding** that uses universal genetic markers to identify DNA of a mixture of organisms

--

<br>
.content-box-grey[What is it the <br> **DNA barcoding**?]

---

.center[.font150[Microbial DNA metabarcoding]]

- **DNA barcoding** is a method of species identification using a short section of DNA from a specific gene 

- An individual sequence can be used to uniquely identify an organism by comparison with a reference library of classified DNA sequences


```{r echo=FALSE, out.width="520px"}
knitr::include_graphics("images/DNAbarcoding.png")
```

---

.center[.font150[Microbial DNA metabarcoding]]

.pull-left[
**Marker selection**

.n30[
- Markers used for DNA barcoding are called barcodes and their selection is crucial

- Ideally, one gene sequence would be used for developing universal PCR primers for all taxonomic group
  
- However, no such gene region has been found yet!

- Different gene regions are used to identify the different organismal groups using barcoding
]
]

.pull-right[

```{r echo=FALSE, out.width="200px"}
knitr::include_graphics("images/DNAbarcoding2.png")
```

.right[.font50[(Mosa et al., 2019)]]
]

---

.center[.font150[Microbial DNA metabarcoding]]

**Marker selection**

| Organism | Region |
|----------|--------|
| animals  | cytochrome c oxidase I|
| fungi    |  internal transcribed spacer (ITS) rRNA |
| plants   | RuBisCO |
| prokaryotes | 16S rRNA |
| microbial eukaryotes | 18S rRNA |


.font80[When barcoding is used to identify organisms from a sample containing DNA from more than one organism, the term DNA metabarcoding is used]


---

.center[.font150[Microbial DNA metabarcoding]]

.pull-left[
.m0p[
**Bioinformatic analysis**
]
.n30[
- After sequencing, raw data must be processed (**quality and cleaning step**)

- Samples are clustered in operational taxonomic unit based on identity threshold (**OTU formation**)

- Using "the Basic Local Alignment Search Tool (BLAST)" regions of similarity between sequences and reference databases are identified (**annotation step**)    

- Microbiota composition definition (**richness and diversity step**)

]
]

.pull-right[

.h20[
.m0p[
.right[.font50[(Jesse et al., 2018)]]     
]
```{r echo=FALSE, out.width="220px"}
knitr::include_graphics("images/OTU.png")
```


]]



???

.n30[
- These databases contain the DNA barcodes assigned to previously identified taxa

- An operational taxonomic unit (OTU) is an operational definition used to classify groups of closely related individuals
]

---
layout: true

# What can the microbiota potentially do?

---

class: inverse, middle, center

----

---

.center[.font120[Metagenomics]]

```{r echo=FALSE, out.width="720px"}
knitr::include_graphics("images/metagenomics1.png")
```
.right[.font50[`r Cite(bib,  key = "YE2019779")`]]
.h25[
- **Metagenomics** is the study of genetic material recovered directly from environmental samples

- The collection of sequenced genes from the environment could be analyzed as a **single genome**
  
- Metagenomics **does not require the isolation** and lab cultivation of individual species

- Due to its random nature, metagenomics, allows that many **under-represented organisms will be detected**
]

---

.center[Metagenomics vs Barcoding]
.pull-left[
.m0p[
```{r echo=FALSE, out.width="450px"}
knitr::include_graphics("images/compareBarcodingMetagenome.png")
```
.right[.font50[`r Cite(bib,  key = "gill2016")`]]
]]


 .pull-right[
.h21[

- Metagenomics does not need the marker selection step

- Metagenomics sequences all the genes present in the sample from each organism

- Metagenomics can help to reconstruct large fragments or even complete genomes from organisms in a community

- Metagenomics allows the characterization of a large number of coding and non-coding sequences that can be used to decipher the microbial diversity or to understand its metabolic potential
]
]

---

```{r echo=FALSE}
# # library(rvest)
# library(gt)
# library(tidyverse)
# library(janitor)
# # url <- "https://en.wikipedia.org/wiki/DNA_sequencing"
# # 
# # df <- url %>%
# #     read_html() %>%
# #     html_node(".wikitable") %>%
# #     html_table() %>%
# #     as_tibble()
# # 
# # df %>%
# #     filter(grepl("Sanger|Nanopore|Illumina|Pacific|Torrent|454", Method)) %>%
# #     write_csv("extra/MetagenomicsRAW.csv")
# read_csv("extra/MetagenomicsClean.csv") %>%
#     clean_names() %>%
#     arrange(order) %>%
#     select(-order) %>%
#     separate_rows(read_length, reads_per_run, sep = ";\n") %>% 
#     separate(method, into = c("method", "tech"), sep = "\\(") %>%
#     mutate(tech = str_remove(tech, "\\)")) %>%
#     rename_with(~toupper(gsub("_", " ", .))) %>%
#     gt() %>%
#     tab_header(
#     title = "Comparison of high-throughput sequencing methods",
#     subtitle = "Source: Wikipedia") %>%
#     tab_options(
#                 table.font.size = 14,
#                 table.width = 1200
#     )
# 
```

```{r echo=FALSE, out.width="920px"}
knitr::include_graphics("images/CompareHTS.png")
```

---

.center[
.middle[
<iframe width="840" height="473" src="https://www.youtube.com/embed/fCd6B5HRaZ8" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
]
]
---

.center[.font120[Genomes assembly]]

.m0p[
```{r echo=FALSE, out.width="620px"}
knitr::include_graphics("images/Align.png")
```
]
---
.font120[**Functional Metagenomics Analysis**]

.pull-left[
.m10ulp[
Workflow:

- Metagenomic assembly

- Open Reading Frames (ORF) prediction

- Annotation step performed by homology detection (at proteomic level)

- Reconstruction of metabolic pathways is achieved from enzyme-coding genes 
]]

.pull-right[
```{r echo=FALSE, out.width="380px"}
knitr::include_graphics("images/metagenomicsPipeline.png")
```
.right[.font50[`r Cite(bib,  key = "Roumpeka_2017")`]]
]

???

.n30[
- In metagenomica, il binning è il processo di raggruppamento di letture o contigs e di assegnarli a unità tassonomiche operative
]

---

.font120[**Functional Metagenomics Analysis**]

.pull-left[
.m10ulp[
Workflow:

- Metagenomic assembly

- Open Reading Frames (ORF) prediction

- Annotation step performed by homology detection (at proteomic level)

- Reconstruction of metabolic pathways is achieved from enzyme-coding genes 
]]

.pull-right[
```{r echo=FALSE, out.width="370px"}
knitr::include_graphics("images/functionalMetagenomics.png")
```
.right[.font50[`r Cite(bib,  key = "Casta_eda_2017")`]]
]

---
layout: true

# What are they doing?

---

class: inverse, center, middle

----

---

.white[Metatranscriptomics]

```{r echo=FALSE, out.width="620px"}
# library(magick)
# x <- image_read("images/quadratoRED.png")
# img <- image_read("images/Molecular-methods-for-determining-phylogeny-and-functional-diversity-of-microbial.ppm")
# y <- image_composite(img, image_scale(x, "135x330!"), offset = "+25+85")
# image_write(y, "images/Meta1.png")
# z <- image_composite(img, image_scale(x, "95x330!"), offset = "+160+85")
# image_write(z, "images/Meta2.png")
# a <- image_composite(img, image_scale(x, "95x330!"), offset = "+275+85")
# image_write(a, "images/Meta3.png")
# b <- image_composite(img, image_scale(x, "95x330!"), offset = "+390+85")
# image_write(b, "images/Meta4.png")
# d <- image_composite(img, image_scale(x, "95x330!"), offset = "+505+85")
# image_write(d, "images/Meta5.png")

knitr::include_graphics("images/Meta1.png")
```
.right[.font50[`r Cite(bib,  key = "VANWONTERGHEM201455")`]]

---

Metatranscriptomics

```{r echo=FALSE, out.width="620px"}
knitr::include_graphics("images/Meta2.png")
```
.right[.font50[`r Cite(bib,  key = "VANWONTERGHEM201455")`]]

---

Metatranscriptomics
.pull-left[
.h25[
- Metatranscriptomics is the science that studies gene expression of microbes within natural environments

- It allows to obtain whole gene expression profiling of complex microbial communities

- `r fontawesome::fa("plus-circle", fill = "green")` The advantage of metatranscriptomics, over metagenomics, is that it can provide information about differences in the active functions of microbial communities 

-  The overview of the gene expression in a given sample is obtained by capturing the total mRNA of the microbiome and by performing a whole metatranscriptomics shotgun sequencing.
]
]

.pull-right[
.m0p[
```{r echo=FALSE, out.width="380px"}
knitr::include_graphics("images/metatrranscriptomePL.png")
```
.right[.font50[`r Cite(bib,  key = "Perez-Pantoja:2015aa")`]]
]
]

---

Metaproteomics

```{r echo=FALSE, out.width="620px"}
knitr::include_graphics("images/Meta3.png")
```
.right[.font50[`r Cite(bib,  key = "VANWONTERGHEM201455")`]]

---

Metaproteomics

.pull-left[
.h25[
- Metaproteomics is the study of all protein samples recovered directly from environmental sources

- Metaproteomics have rapidly increased in recent years due to many technological advances in mass spectrometry (MS)

- `r fontawesome::fa("plus-circle", fill = "green")` Metatranscriptomics assess the “expressed” metabolism and physiology of microbial community members, but also allows quantification of per-species biomass to determine community structure

- `r fontawesome::fa("plus-circle", fill = "green")` Proteomics quantifies the real proteins expression level, while mRNA may be degraded or translated inefficiently, resulting in a incorrect estimation of protein expression level
]
]

.pull.right[
```{r echo=FALSE, out.width="480px"}
knitr::include_graphics("images/metaproteomics.png")
```
.right[.font30[https://www.ecu.edu.au/schools/science/research-activity/research-support/proteomics/what-is-lc-ms-proteomics]]
]

---

Metabolomics

```{r echo=FALSE, out.width="600px"}
knitr::include_graphics("images/Meta4.png")
```
.right[.font50[`r Cite(bib,  key = "VANWONTERGHEM201455")`]]

---

Metabolomics

.pull-left[
.m0p[
.h25[

- Metabolomics is the study of the complete set of metabolites composition (the metabolome) of a cell type, tissue, or biological fluid

- `r fontawesome::fa("plus-circle", fill = "green")` The metabolites (<1.5 kDa) are the intermediates or end products of multiple enzymatic reactions and therefore are the most informative proxies of the biochemical activity of an organism

- `r fontawesome::fa("times-circle", fill = "red")` The metabolome consists of multiple compounds. Determine the entire set of metabolites is extremely difficult, further complicating the analyses is the dynamic nature of these metabolites.
]
]]

.pull-rigth[
<br><br>
```{r echo=FALSE, out.width="480px"}
knitr::include_graphics("images/metabolomics.png")
```
]
.right[.font50[`r Cite(bib,  key = "alonso2015")`]]


???

.n30[
- hydrophilic
  carbohydrates, volatile alcohols, ketones, amino and non- amino organic acids, hydrophobic lipids and complex natural compounds (secondary metabolites such as antibiotics, pigments, non-ribosomal peptides, cofactors)
]

---

Stable/radioactive isotopes

```{r echo=FALSE, out.width="600px"}
knitr::include_graphics("images/Meta5.png")
```
.right[.font50[`r Cite(bib,  key = "VANWONTERGHEM201455")`]]

---

Stable/radioactive isotopes

.pull-left[
.m0p[
.h25[

- Stable-isotope probing (SIP) is a technique in microbial ecology for tracing fluxes of nutrients in biogeochemical cycling by microorganisms using heavy stable isotopes

- `r fontawesome::fa("plus-circle", fill = "green")` SIP is an approach that can identify microbial populations with a defined function

- `r fontawesome::fa("plus-circle", fill = "green")` The use of either DNA-SIP, RNA-SIP or protein-SIP can enable both the phylogenetic identification and the key metabolic genes/proteins

- `r fontawesome::fa("times-circle", fill = "red")` A major challenge associated with SIP is a very limited availability and high cost of labeled substrates. SIP is also very labor-intensive and low-throughput using current techniques
]
]]

.pull-rigth[
```{r echo=FALSE, out.width="280px"}
knitr::include_graphics("images/sip1.png")
```
]
.right[.font50[`r Cite(bib,  key = "UHLIK2013154")`]]

---

Stable/radioactive isotopes

.m0p[
```{r echo=FALSE, out.width="570px"}
knitr::include_graphics("images/sip2.png")
```
.right[.font50[(el Zahar Haichar, Heulin, Guyonnet, et al., 2016)]]
]

---
layout: false
class: clear, middle, center

.m0p[
```{r echo=FALSE, out.width="850px"}
knitr::include_graphics("images/sb1.png")
```
]

???
metafora della torta e dei suoi ingredienti

---
class: clear


.center[.font150[System biology]]

.pull-left[
.m0p[
.h25u[
- .font80[Systems biology studies biological systems by systematically perturbing them]    

- .font80[Monitoring the gene, protein, metabolite, and informational pathway responses]    

- .font80[Integrating these data]    

- .font80[Formulating mathematical models that describe the structure of the system]    

- .font80[Predict its response to individual perturbations]
  ]]]

.pull-right[
.m0p[
```{r echo=FALSE, out.width="670px"}
knitr::include_graphics("images/sb2.png")
```

]
]

---

layout: true

# Questions about the lesson


---


.m0tbp[
.center[.font50px[`r fontawesome::fa("question-circle", fill = "black")`]]
]

.marco[
.center[.font90[[[`r fontawesome::fa("slack", fill = "steelblue")`](https://join.slack.com/t/ipma-2020/shared_invite/zt-j9o8ig7k-LN1~U3d6QJztquYbC6iW7w) Slack (https://bit.ly/3eRDBRL) - `r fontawesome::fa("envelope", fill = "steelblue")`](mailto:marco.chiapello@unito.it) marco.chiapello@unito.it]
]
]


