<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>Microbiota</title>
    <meta charset="utf-8" />
    <meta name="author" content="Marco Chiapello" />
    <script src="libs/header-attrs/header-attrs.js"></script>
    <link rel="stylesheet" href="scrollable.css" type="text/css" />
    <link rel="stylesheet" href="mtheme_max.css" type="text/css" />
    <link rel="stylesheet" href="fonts_mtheme_max.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">




class: title-slide



# .font170[MICROBIOMA E MICROBIOTA]


&lt;br&gt;&lt;br&gt;&lt;br&gt;
.marco[
Marco Chiapello
&lt;br&gt;
2020-09-13
]


.marco[
[&lt;svg style="height:0.8em;top:.04em;position:relative;fill:steelblue;" viewBox="0 0 496 512"&gt;&lt;path d="M336.5 160C322 70.7 287.8 8 248 8s-74 62.7-88.5 152h177zM152 256c0 22.2 1.2 43.5 3.3 64h185.3c2.1-20.5 3.3-41.8 3.3-64s-1.2-43.5-3.3-64H155.3c-2.1 20.5-3.3 41.8-3.3 64zm324.7-96c-28.6-67.9-86.5-120.4-158-141.6 24.4 33.8 41.2 84.7 50 141.6h108zM177.2 18.4C105.8 39.6 47.8 92.1 19.3 160h108c8.7-56.9 25.5-107.8 49.9-141.6zM487.4 192H372.7c2.1 21 3.3 42.5 3.3 64s-1.2 43-3.3 64h114.6c5.5-20.5 8.6-41.8 8.6-64s-3.1-43.5-8.5-64zM120 256c0-21.5 1.2-43 3.3-64H8.6C3.2 212.5 0 233.8 0 256s3.2 43.5 8.6 64h114.6c-2-21-3.2-42.5-3.2-64zm39.5 96c14.5 89.3 48.7 152 88.5 152s74-62.7 88.5-152h-177zm159.3 141.6c71.4-21.2 129.4-73.7 158-141.6h-108c-8.8 56.9-25.6 107.8-50 141.6zM19.3 352c28.6 67.9 86.5 120.4 158 141.6-24.4-33.8-41.2-84.7-50-141.6h-108z"/&gt;&lt;/svg&gt;](https://orcid.org/0000-0001-7768-3047) orcID - [&lt;svg style="height:0.8em;top:.04em;position:relative;fill:steelblue;" viewBox="0 0 448 512"&gt;&lt;path d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"/&gt;&lt;/svg&gt;](https://www.linkedin.com/in/marco-chiapello-20b58750/) marcochiapello
&lt;br&gt;
.font90[[&lt;svg style="height:0.8em;top:.04em;position:relative;fill:steelblue;" viewBox="0 0 512 512"&gt;&lt;path d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"/&gt;&lt;/svg&gt;](https://twitter.com/marpello1980) @marpello1980 - [&lt;svg style="height:0.8em;top:.04em;position:relative;fill:steelblue;" viewBox="0 0 512 512"&gt;&lt;path d="M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7 22.4 17.4 52.1 39.5 154.1 113.6 21.1 15.4 56.7 47.8 92.2 47.6 35.7.3 72-32.8 92.3-47.6 102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4 132.7-96.3 142.8-104.7 173.4-128.7 5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9 30.6 23.9 40.7 32.4 173.4 128.7 16.8 12.2 50.2 41.8 73.4 41.4z"/&gt;&lt;/svg&gt;](mailto:chiapello.m@gmail.com) chiapello.m@gmail.com - &lt;svg style="height:0.8em;top:.04em;position:relative;fill:steelblue;" viewBox="0 0 448 512"&gt;&lt;path d="M424.7 299.8c2.9-14 4.7-28.9 4.7-43.8 0-113.5-91.9-205.3-205.3-205.3-14.9 0-29.7 1.7-43.8 4.7C161.3 40.7 137.7 32 112 32 50.2 32 0 82.2 0 144c0 25.7 8.7 49.3 23.3 68.2-2.9 14-4.7 28.9-4.7 43.8 0 113.5 91.9 205.3 205.3 205.3 14.9 0 29.7-1.7 43.8-4.7 19 14.6 42.6 23.3 68.2 23.3 61.8 0 112-50.2 112-112 .1-25.6-8.6-49.2-23.2-68.1zm-194.6 91.5c-65.6 0-120.5-29.2-120.5-65 0-16 9-30.6 29.5-30.6 31.2 0 34.1 44.9 88.1 44.9 25.7 0 42.3-11.4 42.3-26.3 0-18.7-16-21.6-42-28-62.5-15.4-117.8-22-117.8-87.2 0-59.2 58.6-81.1 109.1-81.1 55.1 0 110.8 21.9 110.8 55.4 0 16.9-11.4 31.8-30.3 31.8-28.3 0-29.2-33.5-75-33.5-25.7 0-42 7-42 22.5 0 19.8 20.8 21.8 69.1 33 41.4 9.3 90.7 26.8 90.7 77.6 0 59.1-57.1 86.5-112 86.5z"/&gt;&lt;/svg&gt; marpello]
]

---
layout: true
class: clear

---

.font180[.bold[Agenda]]


- .font140[Introduzione]
  
- .font140[Il microbioma delle piante]

  - .font120[Da chi è composto]
  
  - .font120[Come prende forma]

  - .font120[Che funzioni ha]


???

- .n[**Intro**: spiegazione dei concetti generali e dei termini]
---


.pull-left[.center[.font180[.bold[microBIOTA]]]
&lt;br&gt;
&lt;img src="images/microbiota.jpg" width="500px" style="display: block; margin: auto;" /&gt;

]

.pull-right[.center[.font180[.bold[microBIOMA]]]
&lt;br&gt;
&lt;img src="images/microbioma.jpg" width="500px" style="display: block; margin: auto;" /&gt;


]

???

- .n[si riferisce a una popolazione di microrganismi che colonizza un determinato luogo]

- .n[batteri, funghi, protozoi, elminti, neamtodi, virus, ]
 
- .n[indica la totalità del patrimonio genetico posseduto dal microbiota, cioè i geni che quest’ultimo è in grado di esprimere]

- .n[la differenza che passa tra i due termini è la stessa che esiste tra popolazione umana e genoma umano]
---


.pull-left[.center[.font180[.bold[microBIOTA]]]
&gt; si riferisce a una popolazione di microrganismi che colonizza un determinato luogo

]

.pull-right[.center[.font180[.bold[microBIOMA]]]
&gt; indica la totalità del patrimonio genetico posseduto dal microbiota, cioè i geni che quest’ultimo è in grado di esprimere

]

---


.pull-left[.center[.font180[.bold[microBIOTA]]]]

.pull-right[.center[.font180[.bold[microBIOMA]]]]

&lt;br&gt;
&lt;img src="images/microPublications.gif" width="550px" style="display: block; margin: auto;" /&gt;

---
&lt;br&gt;&lt;br&gt;&lt;br&gt;
.center[.font600[.fontW7[HMP]]]
.center[.font140[Human Microbiome Project]]

---

Il **progetto microbioma umano** è una iniziativa dei National Institutes of Health statunitensi con il fine di **identificare e caratterizzare i microrganismi ed il loro rapporto con lo stato di salute e di malattia dell'uomo**. 

&lt;img src="images/microNubers.jpg" width="700px" style="display: block; margin: auto;" /&gt;

???


- .n[**RAPPORTO** cellule microbiche e cellule umane]

- .n[Il numero totale di cellule microbiche presenti in un organismo umano può superare di dieci volte il numero di cellule dell'organismo stesso]

- .n[i geni di origine microbica possono superare di cento volte il numero di geni presenti nel genoma umano (composto da circa 20.000 geni).]

---

&lt;br&gt;&lt;br&gt;&lt;br&gt;
.content-box-grey[.font180[.fontW7[L'essere umano va concepito come composto da cellule umane e microbiche]]]

???

- .n[Visto l'alto numero di cellule microbiche e della quantità del loro materiale genetico]

- .n[L'essere umano va concepito come composto da cellule umane e microbiche]

- .n[MA **quali** sono i micro-organismi]


---
.center[Temporal human microbiota]
&lt;img src="images/humanEvo.jpg" width="780px" style="display: block; margin: auto;" /&gt;

???

- .n[La figura mostra i phyla chiave della composizione del microbiota umano in diversi stadi della vita]

---

.center[Spatial human microbiota]
&lt;img src="images/humanDiversity.png" width="750px" height="580px" style="display: block; margin: auto;" /&gt;

???

- .n[Vediamo ora **dove** sono distribuiti i batteri]

- .n[Partiamo dalla parte più interna della figura, dall'albero filigenetico]



---
layout: true
class: clear
Focus on human **gut microbiome** (Kho and Lal, 2018)

---
&lt;img src="images/KhoPaperT3-1.png" width="800px" height="530px" style="display: block; margin: auto;" /&gt;
---
&lt;br&gt;&lt;br&gt;
&lt;img src="images/KhoPaperT3-2.png" width="900px" height="400px" style="display: block; margin: auto;" /&gt;
---
  
&lt;img src="images/KhoPaperT3-3.png" width="900px" height="530px" style="display: block; margin: auto;" /&gt;

---
layout: false
class: inverse, middle, center

.font300[.white[Il microbioma delle piante]]


---
layout: true 
class: clear

---


&lt;img src="images/PlantmicroPublications.gif" width="630px" style="display: block; margin: auto;" /&gt;

---
layout: true

# Composizione del microbiota
---

## Glossario


- .font80[**Rhizosphere**: The region of soil in the vicinity of plant roots that is influenced by plant-derived nutrients and oxygen availability; it is not a region of definable size or shape, but instead consists of a gradient in chemical, biological and physical properties that change both radially and longitudinally along the root.]

--

- .font80[**Phyllosphere**: All the aboveground organs of plants, including the leaf, flower, stem and fruit.]

--

- .font80[**Endophytes**: The microorganisms residing within plant tissues (the endosphere), such as leaves, roots or stems.]

--

- .font80[**Bulk soil**:  is soil outside the rhizosphere. Bulk soil is not penetrated by plant roots]

---

layout: true
# Composizione del microbiota

.pull-left[
&lt;img src="images/PlantMicrobiomeComposition.png" width="510px" height="510px" style="display: block; margin: auto;" /&gt;
]

---

.pull-right[

&lt;br&gt;&lt;br&gt;&lt;br&gt;&lt;br&gt;&lt;br&gt;&lt;br&gt;&lt;br&gt;&lt;br&gt;&lt;br&gt;&lt;br&gt;
.font60[(Trivedi, Leach, Tringe, et al., 2020)]
]

---

.pull-right[

Plant **endophytic communities**

- .n[They are frequently **enriched** in members of the **Proteobacteria and Firmicutes**]

- .n[They are **depleted** in members of the **Acidobacteria, Planctomycetes, Chloroflexi and Verrucomicrobia**]


]

---

.pull-right[

Plant **phyllosphere community**

- .n[It Mainly comprises bacteria belonging to phylum Proteobacteria and to the Bacteroidetes, Firmicutes and Actinomycetes]

- .n[Members of phylum **Proteobacteria** constitute ~50% of the community composition]


]

---

.pull-right[

Fungi community

- .n[The vast diversity of fungi that colonize both aboveground and belowground plant tissues mainly belong to the phyla **Ascomycota and Basidiomycota**]

- .n[Although **arbuscular fungi** (phylum Glomeromycota) and **ectomycorrhizal fungi** are well-studied, these groups represent only **minor diversity among fungal communities from roots**]
]

---

.pull-right[

- .n[Although the assemblies of root-associated bacteria and fungi differ substantially from the above-ground communities, both represent a **subset of the microbiota derived from soil communities** and enriched in different plant-associated niches]

- .n[This suggests that **soil functions as a common reservoir** for both belowground and aboveground plant microbiota]

]
---


.pull-right[

**Viruses**

- .n[Viruses play a **critical part in bacterial community assembly and turnover** in the soil, but their function in plant-associated environments is not completely understood]

- .n[Viruses, in soil, affect both the **microbiome structure and function**]

]

---

layout: true
# Composizione del microbiota
---

.pull-left[

&lt;img src="images/bacterialDiv.png" width="350px" style="display: block; margin: auto;" /&gt;

&lt;img src="images/fungalDiv.png" width="350px" style="display: block; margin: auto;" /&gt;
]

.pull-right[

- .n[There are clear differences among the microbial communities in different plant compartments, which indicates that the **plant compartment is a major selective force** that shapes the composition of plant-associated microbiota]

]

???

- .n[**Shannon diversity index**: combina ricchezza e diversità]

- .n[Misura sia il numero delle specie presenti e la differenza tra l'abbondanza delle diverse specie.]

- .n[Un valore grande significa che sono presenti molte specie con un'abbondanza simile. Il range del valore varia da 1 (una specie dominante) fino al numero totale di tutte le species (nel caso che tutte le specie abbiano la stessa abbondanza)]


---

.pull-left-45[.font150[.f400[Core microbiota]]]

.h25[]

.pull-right-45[.font150[.f400[Hub microorganisms]]]

.h25[]
---

.pull-left-45[.font150[.f400[Core microbiota]]

.h25[
- The plant core microbiota consists of members of the microbial community that are persistent and are ubiquitous in almost all the communities associated with a particular host

- The core microbiota contains key microbial taxa that carry genes with functions that are essential for host fitness

- Many members of the core microbiota of different plant species are common at the genus level

- The identification of a co-occurring core of plant-associated microorganisms also provides a useful starting point for studies of how to build SynComs to manipulate plant–microbiome interactions for increased growth and productivity

]
]

.pull-right-45[.font150[.f400[Hub microorganisms]]]

.h25[]
---

.pull-left-45[.font150[.f400[Core microbiota]]

.h25[
- The plant core microbiota consists of members of the microbial community that are persistent and are ubiquitous in almost all the communities associated with a particular host

- The core microbiota contains key microbial taxa that carry genes with functions that are essential for host fitness

- Many members of the core microbiota of different plant species are common at the genus level

- The identification of a co-occurring core of plant-associated microorganisms also provides a useful starting point for studies of how to build SynComs to manipulate plant–microbiome interactions for increased growth and productivity

]
]
.pull-right-45[.font150[.f400[Hub microorganisms]]

.h25[

- Within the core microbiota, a few members, such as ‘hub microorganisms’, can influence the community structure through strong biotic interactions with the host or with other microbial species, rather than simply by their own high abundance

- These hub species may represent keystone species that can exert strong direct and indirect effects on microbiome assembly and that function as mediators between the plant and its associated microbiome.

- For example, Albugo laibachii and Dioszegia spp. have been identified62 as highly interactive hubs in the phyllosphere of *A. thaliana*.
]
]

???

- .n[such as barley, rice, sugarcane, grapevine, citrus, soybean and A. thaliana]

- .n[(for example, Pseudomonas, Agrobacterium, Methylobacterium, Sphingomonas, Erwinia, Cladosporium, Coniothyrium, Resinicium and Fusarium)]
---
.content-box-grey[
La piante non possono più essere considerate come unità a se stanti, ma è richiesta una visione **olistica**.]
&lt;br&gt;
.content-box-grey[
Il microbiota ha un impatto sia sulla **crescita** che **sopravvivenza** della pianta, due concetti chiave della fitness della pianta]
&lt;br&gt;
.content-box-grey[
La fitness della pianta è quindi una conseguenza delle **caratteristiche della pianta e del suo microbiota**] 




???

- .n[Le piante ospitano una grande varietà di micro organismi sia sulla loro superficie che all'interno]

---
layout: false
class: clear

.center[.font200[Holobiont]]



???

A plant and the members
of its associated microbiota considered as a single entity; this represents the ‘unit of selection’ at which plant– microbiome interactions have probably co-evolved in order to maintain host functionality and fitness over ecological and even evolutionary timescales
---
layout: false

# References
Bigeard, J, J. Colcombet, and H. Hirt (2015). "Signaling Mechanisms in
Pattern-Triggered Immunity (PTI)". In: _Molecular Plant_ 8.4. Cell
Signaling, pp. 521 - 539. ISSN: 1674-2052. DOI:
[https://doi.org/10.1016/j.molp.2014.12.022](https://doi.org/https%3A%2F%2Fdoi.org%2F10.1016%2Fj.molp.2014.12.022).
URL:
[http://www.sciencedirect.com/science/article/pii/S1674205215000878](http://www.sciencedirect.com/science/article/pii/S1674205215000878).

    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="libs/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"ratio": "16:9",
"highlightStyle": "github",
"highlightLines": true
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
