<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>Microbiota</title>
    <meta charset="utf-8" />
    <meta name="author" content="Marco Chiapello" />
    <script src="libs/header-attrs/header-attrs.js"></script>
    <link href="libs/tile-view/tile-view.css" rel="stylesheet" />
    <script src="libs/tile-view/tile-view.js"></script>
    <link rel="stylesheet" href="scrollable.css" type="text/css" />
    <link rel="stylesheet" href="mtheme_max.css" type="text/css" />
    <link rel="stylesheet" href="fonts_mtheme_max.css" type="text/css" />
    <link rel="stylesheet" href="lesson3.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">






class: title-slide



# .font170[MICROBIOMA E MICROBIOTA]


&lt;br&gt;&lt;br&gt;&lt;br&gt;&lt;br&gt;
.marco[
Marco Chiapello
&lt;br&gt;
2020-11-17
]


.marco[
.center[.font90[[[&lt;svg style="height:0.8em;top:.04em;position:relative;fill:steelblue;" viewBox="0 0 448 512"&gt;&lt;path d="M94.12 315.1c0 25.9-21.16 47.06-47.06 47.06S0 341 0 315.1c0-25.9 21.16-47.06 47.06-47.06h47.06v47.06zm23.72 0c0-25.9 21.16-47.06 47.06-47.06s47.06 21.16 47.06 47.06v117.84c0 25.9-21.16 47.06-47.06 47.06s-47.06-21.16-47.06-47.06V315.1zm47.06-188.98c-25.9 0-47.06-21.16-47.06-47.06S139 32 164.9 32s47.06 21.16 47.06 47.06v47.06H164.9zm0 23.72c25.9 0 47.06 21.16 47.06 47.06s-21.16 47.06-47.06 47.06H47.06C21.16 243.96 0 222.8 0 196.9s21.16-47.06 47.06-47.06H164.9zm188.98 47.06c0-25.9 21.16-47.06 47.06-47.06 25.9 0 47.06 21.16 47.06 47.06s-21.16 47.06-47.06 47.06h-47.06V196.9zm-23.72 0c0 25.9-21.16 47.06-47.06 47.06-25.9 0-47.06-21.16-47.06-47.06V79.06c0-25.9 21.16-47.06 47.06-47.06 25.9 0 47.06 21.16 47.06 47.06V196.9zM283.1 385.88c25.9 0 47.06 21.16 47.06 47.06 0 25.9-21.16 47.06-47.06 47.06-25.9 0-47.06-21.16-47.06-47.06v-47.06h47.06zm0-23.72c-25.9 0-47.06-21.16-47.06-47.06 0-25.9 21.16-47.06 47.06-47.06h117.84c25.9 0 47.06 21.16 47.06 47.06 0 25.9-21.16 47.06-47.06 47.06H283.1z"/&gt;&lt;/svg&gt;](https://join.slack.com/t/ipma-2020/shared_invite/zt-j9o8ig7k-LN1~U3d6QJztquYbC6iW7w) Slack (https://bit.ly/3eRDBRL) - &lt;svg style="height:0.8em;top:.04em;position:relative;fill:steelblue;" viewBox="0 0 512 512"&gt;&lt;path d="M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7 22.4 17.4 52.1 39.5 154.1 113.6 21.1 15.4 56.7 47.8 92.2 47.6 35.7.3 72-32.8 92.3-47.6 102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4 132.7-96.3 142.8-104.7 173.4-128.7 5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9 30.6 23.9 40.7 32.4 173.4 128.7 16.8 12.2 50.2 41.8 73.4 41.4z"/&gt;&lt;/svg&gt;](mailto:marco.chiapello@unito.it) marco.chiapello@unito.it]
]
]

---
layout: true

# Last lesson check

---

&lt;img src="images/qr-code_lesson1.png" width="500px" style="display: block; margin: auto;" /&gt;

.center[https://bit.ly/3np9ILA]

---

class: clear, middle, center

# .black[Il microbiota vegetale varia?]

.m0tbp[
<style>html {
  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Helvetica Neue', 'Fira Sans', 'Droid Sans', Arial, sans-serif;
}

#ksfoeigmnt .gt_table {
  display: table;
  border-collapse: collapse;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#ksfoeigmnt .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#ksfoeigmnt .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#ksfoeigmnt .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 0;
  padding-bottom: 4px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#ksfoeigmnt .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#ksfoeigmnt .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#ksfoeigmnt .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#ksfoeigmnt .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#ksfoeigmnt .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#ksfoeigmnt .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#ksfoeigmnt .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#ksfoeigmnt .gt_group_heading {
  padding: 8px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
}

#ksfoeigmnt .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#ksfoeigmnt .gt_from_md > :first-child {
  margin-top: 0;
}

#ksfoeigmnt .gt_from_md > :last-child {
  margin-bottom: 0;
}

#ksfoeigmnt .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#ksfoeigmnt .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 12px;
}

#ksfoeigmnt .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#ksfoeigmnt .gt_first_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
}

#ksfoeigmnt .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#ksfoeigmnt .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#ksfoeigmnt .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#ksfoeigmnt .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#ksfoeigmnt .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#ksfoeigmnt .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding: 4px;
}

#ksfoeigmnt .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#ksfoeigmnt .gt_sourcenote {
  font-size: 90%;
  padding: 4px;
}

#ksfoeigmnt .gt_left {
  text-align: left;
}

#ksfoeigmnt .gt_center {
  text-align: center;
}

#ksfoeigmnt .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#ksfoeigmnt .gt_font_normal {
  font-weight: normal;
}

#ksfoeigmnt .gt_font_bold {
  font-weight: bold;
}

#ksfoeigmnt .gt_font_italic {
  font-style: italic;
}

#ksfoeigmnt .gt_super {
  font-size: 65%;
}

#ksfoeigmnt .gt_footnote_marks {
  font-style: italic;
  font-size: 65%;
}
</style>
<div id="ksfoeigmnt" style="overflow-x:auto;overflow-y:auto;width:auto;height:auto;"><table class="gt_table">
  
  <thead class="gt_col_headings">
    <tr>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1">Opzioni</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1">Risposte</th>
    </tr>
  </thead>
  <tbody class="gt_table_body">
    <tr>
      <td class="gt_row gt_left">Si, varia nel tempo. In diversi momenti della vita della pianta la composizione del microbiota muta</td>
      <td class="gt_row gt_center">1 <svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"><path d="M504 256c0 136.967-111.033 248-248 248S8 392.967 8 256 119.033 8 256 8s248 111.033 248 248zM227.314 387.314l184-184c6.248-6.248 6.248-16.379 0-22.627l-22.627-22.627c-6.248-6.249-16.379-6.249-22.628 0L216 308.118l-70.059-70.059c-6.248-6.248-16.379-6.248-22.628 0l-22.627 22.627c-6.248 6.248-6.248 16.379 0 22.627l104 104c6.249 6.249 16.379 6.249 22.628.001z"/></svg></td>
    </tr>
  </tbody>
  
  
</table></div>
]

---

class: clear, middle, center

# .black[Qual è la forza selettiva maggiore che da forma al microbiota associato alle piante?]

.m0tbp[
<style>html {
  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Helvetica Neue', 'Fira Sans', 'Droid Sans', Arial, sans-serif;
}

#hjnjsfinum .gt_table {
  display: table;
  border-collapse: collapse;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#hjnjsfinum .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#hjnjsfinum .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#hjnjsfinum .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 0;
  padding-bottom: 4px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#hjnjsfinum .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#hjnjsfinum .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#hjnjsfinum .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#hjnjsfinum .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#hjnjsfinum .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#hjnjsfinum .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#hjnjsfinum .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#hjnjsfinum .gt_group_heading {
  padding: 8px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
}

#hjnjsfinum .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#hjnjsfinum .gt_from_md > :first-child {
  margin-top: 0;
}

#hjnjsfinum .gt_from_md > :last-child {
  margin-bottom: 0;
}

#hjnjsfinum .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#hjnjsfinum .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 12px;
}

#hjnjsfinum .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#hjnjsfinum .gt_first_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
}

#hjnjsfinum .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#hjnjsfinum .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#hjnjsfinum .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#hjnjsfinum .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#hjnjsfinum .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#hjnjsfinum .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding: 4px;
}

#hjnjsfinum .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#hjnjsfinum .gt_sourcenote {
  font-size: 90%;
  padding: 4px;
}

#hjnjsfinum .gt_left {
  text-align: left;
}

#hjnjsfinum .gt_center {
  text-align: center;
}

#hjnjsfinum .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#hjnjsfinum .gt_font_normal {
  font-weight: normal;
}

#hjnjsfinum .gt_font_bold {
  font-weight: bold;
}

#hjnjsfinum .gt_font_italic {
  font-style: italic;
}

#hjnjsfinum .gt_super {
  font-size: 65%;
}

#hjnjsfinum .gt_footnote_marks {
  font-style: italic;
  font-size: 65%;
}
</style>
<div id="hjnjsfinum" style="overflow-x:auto;overflow-y:auto;width:auto;height:auto;"><table class="gt_table">
  
  <thead class="gt_col_headings">
    <tr>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1">Opzioni</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1">Risposte</th>
    </tr>
  </thead>
  <tbody class="gt_table_body">
    <tr>
      <td class="gt_row gt_left">La pianta</td>
      <td class="gt_row gt_center">1 <svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"><path d="M504 256c0 136.967-111.033 248-248 248S8 392.967 8 256 119.033 8 256 8s248 111.033 248 248zM227.314 387.314l184-184c6.248-6.248 6.248-16.379 0-22.627l-22.627-22.627c-6.248-6.249-16.379-6.249-22.628 0L216 308.118l-70.059-70.059c-6.248-6.248-16.379-6.248-22.628 0l-22.627 22.627c-6.248 6.248-6.248 16.379 0 22.627l104 104c6.249 6.249 16.379 6.249 22.628.001z"/></svg></td>
    </tr>
  </tbody>
  
  
</table></div>
]

---

class: clear, middle, center

# .black[Per olobiote si intende...]

.m0tbp[
<style>html {
  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Helvetica Neue', 'Fira Sans', 'Droid Sans', Arial, sans-serif;
}

#kntvpbkfff .gt_table {
  display: table;
  border-collapse: collapse;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#kntvpbkfff .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#kntvpbkfff .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#kntvpbkfff .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 0;
  padding-bottom: 4px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#kntvpbkfff .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#kntvpbkfff .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#kntvpbkfff .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#kntvpbkfff .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#kntvpbkfff .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#kntvpbkfff .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#kntvpbkfff .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#kntvpbkfff .gt_group_heading {
  padding: 8px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
}

#kntvpbkfff .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#kntvpbkfff .gt_from_md > :first-child {
  margin-top: 0;
}

#kntvpbkfff .gt_from_md > :last-child {
  margin-bottom: 0;
}

#kntvpbkfff .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#kntvpbkfff .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 12px;
}

#kntvpbkfff .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#kntvpbkfff .gt_first_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
}

#kntvpbkfff .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#kntvpbkfff .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#kntvpbkfff .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#kntvpbkfff .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#kntvpbkfff .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#kntvpbkfff .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding: 4px;
}

#kntvpbkfff .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#kntvpbkfff .gt_sourcenote {
  font-size: 90%;
  padding: 4px;
}

#kntvpbkfff .gt_left {
  text-align: left;
}

#kntvpbkfff .gt_center {
  text-align: center;
}

#kntvpbkfff .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#kntvpbkfff .gt_font_normal {
  font-weight: normal;
}

#kntvpbkfff .gt_font_bold {
  font-weight: bold;
}

#kntvpbkfff .gt_font_italic {
  font-style: italic;
}

#kntvpbkfff .gt_super {
  font-size: 65%;
}

#kntvpbkfff .gt_footnote_marks {
  font-style: italic;
  font-size: 65%;
}
</style>
<div id="kntvpbkfff" style="overflow-x:auto;overflow-y:auto;width:auto;height:auto;"><table class="gt_table">
  
  <thead class="gt_col_headings">
    <tr>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1">Opzioni</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1">Risposte</th>
    </tr>
  </thead>
  <tbody class="gt_table_body">
    <tr>
      <td class="gt_row gt_left">Un organismo vivente alla luce della convivenza con il proprio microbiota</td>
      <td class="gt_row gt_center">1 <svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"><path d="M504 256c0 136.967-111.033 248-248 248S8 392.967 8 256 119.033 8 256 8s248 111.033 248 248zM227.314 387.314l184-184c6.248-6.248 6.248-16.379 0-22.627l-22.627-22.627c-6.248-6.249-16.379-6.249-22.628 0L216 308.118l-70.059-70.059c-6.248-6.248-16.379-6.248-22.628 0l-22.627 22.627c-6.248 6.248-6.248 16.379 0 22.627l104 104c6.249 6.249 16.379 6.249 22.628.001z"/></svg></td>
    </tr>
  </tbody>
  
  
</table></div>
]

---
layout: true

# Last lesson recap

---

.pull-left[
&lt;br&gt;
.m0p[
.h25o[
1. Plant microbiota changes spatially and temporally

1. Plant compartment is a major selective force that shapes the composition of plant-associated microbiota

1. Plants and their associated microorganisms form a holobiont

1. Core and Hub microbiota

]
]
]

---

.pull-left[
&lt;br&gt;
.m0p[
.h25o[
1. Plant microbiota change spatially and temporally

1. .opacity10[ Plant compartment is a major selective force that shapes the composition of plant-associated microbiota]

1. .opacity10[ Plants and their associated microorganisms form a holobiont]

1. .opacity10[ Core and Hub microbiota]

]
]
]

.pull-right[

.m0p[
&lt;img src="images/PlantMicrobiomeComposition.png" width="230px" height="230px" style="display: block; margin: auto;" /&gt;

&lt;img src="images/plantMicrobiotaTemporal.png" width="300px" height="270px" style="display: block; margin: auto;" /&gt;
]
]

---

.pull-left[
&lt;br&gt;
.m0p[
.h25o[
1. .opacity10[ Plant microbiota change spatially and temporally]

1. Plant compartment is a major selective force that shapes the composition of plant-associated microbiota

1. .opacity10[ Plants and their associated microorganisms form a holobiont]

1. .opacity10[ Core and Hub microbiota]

]
]
]

.pull-right[

&lt;img src="images/bacterialDiv.png" width="350px" style="display: block; margin: auto;" /&gt;

&lt;img src="images/fungalDiv.png" width="350px" style="display: block; margin: auto;" /&gt;
]

---

.pull-left[
&lt;br&gt;
.m0p[
.h25o[
1. .opacity10[ Plant microbiota change spatially and temporally]

1. .opacity10[ Plant compartment is a major selective force that shapes the composition of plant-associated microbiota]

1. Plants and their associated microorganisms form a holobiont

1. .opacity10[ Core and Hub microbiota]

]
]
]

.pull-right[
&lt;br&gt;
&lt;img src="images/holobiont1.png" width="800px" style="display: block; margin: auto;" /&gt;

]

---

.pull-left[
&lt;br&gt;
.m0p[
.h25o[
1. .opacity10[ Plant microbiota change spatially and temporally]

1. .opacity10[ Plant compartment is a major selective force that shapes the composition of plant-associated microbiota]

1. .opacity10[Plants and their associated microorganisms form a holobiont]

1. Core and Hub microbiota

]
]
]

.pull-right[
&lt;br&gt;
&lt;img src="images/core1.png" width="200px" style="display: block; margin: auto;" /&gt;

&lt;img src="images/core2.png" width="200px" style="display: block; margin: auto;" /&gt;

]

---
layout: false
class: clear 

.pull-left-wide[
.font180[.bold[Agenda]]



- .font140[How do you study the microbiota composition?]

- .font140[What can the microbiota potentially do?]

- .font140[What are they doing?]
]

???

.h201[
- Questa lezione prevede dei capitoli che riguardano la **pratica** dello studio del microbiota

- Nel primo capitolo vedremo come si rispone alla domanda: "Chi è presente?"

- Nel secondo capitolo risponderemo alla domanda: "Quali funzioni possono avere i microorganismi che compongono il microbiota?"

- Ed infine risponderemo alla domanda, forse, più importante: "Che cosa STANNO facendo i microorganismi presenti nel microbiota"
]


---
layout: true

# How do you study the microbiota composition?


---
class: inverse, middle, center

----

???

.h201[
- Inziamo a vedere come si studia la composizione del microbiota

- Rispondiamo quindi alla domanda: "Chi è presente?"
]

---

.center[**There are two major approaches to assess the microbiota composition**]

.h20[
&lt;img src="images/microbiotaCompositionTecniques.png" width="520px" style="display: block; margin: auto;" /&gt;

.right[.font60[(Lebeis, 2014)]]
]

???

.h201[
- Le 2 metodologie di studio del microbiota si basano su 2 principi diametralmente opposti:
  - Da una parta vedremo un approccio indipendete dalla coltura in vitro ma basato sul materiale genetico     
  - Dall'altra un approccio esclusivamente incentrato sulla coltura in vitro
]

---

.pull-left[
&lt;img src="images/microbiotaCompositionTecniques.png" width="520px" style="display: block; margin: auto;" /&gt;
]

.pull-right[
**Culture dependent**

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; This approach isolates individual microbes 

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm121.6 313.1c4.7 4.7 4.7 12.3 0 17L338 377.6c-4.7 4.7-12.3 4.7-17 0L256 312l-65.1 65.6c-4.7 4.7-12.3 4.7-17 0L134.4 338c-4.7-4.7-4.7-12.3 0-17l65.6-65-65.6-65.1c-4.7-4.7-4.7-12.3 0-17l39.6-39.6c4.7-4.7 12.3-4.7 17 0l65 65.7 65.1-65.6c4.7-4.7 12.3-4.7 17 0l39.6 39.6c4.7 4.7 4.7 12.3 0 17L312 256l65.6 65.1z"/&gt;&lt;/svg&gt; The community diversity estimate is limited

]

???

.h201[

- **Easy Poll**

- Focalizziamo la nostra attenzione sul sistema coltura dipendente

- Questo approccio ha un grande punto di forza: Permette di isolare i microbi in coltura pura (ovviamente tutti quelli coltivabili in vitro - esclusi i simbioti obbligati)

- Il problema di questo approccio è sicuramente la sottostima della diversità all'interno del microbiota
]
---

.pull-left[
**Culture independent**

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt;  The ability to identify and quantify community members, even down to extremely rare taxa

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt;  Insights beyond the information provided by individual microbes

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm121.6 313.1c4.7 4.7 4.7 12.3 0 17L338 377.6c-4.7 4.7-12.3 4.7-17 0L256 312l-65.1 65.6c-4.7 4.7-12.3 4.7-17 0L134.4 338c-4.7-4.7-4.7-12.3 0-17l65.6-65-65.6-65.1c-4.7-4.7-4.7-12.3 0-17l39.6-39.6c4.7-4.7 12.3-4.7 17 0l65 65.7 65.1-65.6c4.7-4.7 12.3-4.7 17 0l39.6 39.6c4.7 4.7 4.7 12.3 0 17L312 256l65.6 65.1z"/&gt;&lt;/svg&gt; This approach does not isolate individual microbes 

]


.pull-right[
&lt;img src="images/microbiotaCompositionTecniques.png" width="520px" style="display: block; margin: auto;" /&gt;
]

???

.h201[
- Il metodo indipendente dalla coltura, che approfondiremo meglio nelle prossime slides, ha 2 indiscutibili vantaggi: 
  - L'abilità non solo di identificare ma anche di quantificare l'abbondanza dei microorganismi presenti. Permettendo di identificare anche organismi estremamente poco abbondanti     
  - Utilizzando approcci bioinformatici è possibile ricavare importati informazioni sul funzionamento dell'intera comunità e non solo sui singoli individui
  
- Questo approccio, però, non permette di isolare i singoli microrganismi

- Isolare i singoli microorganismi può essere importante per la riscostruzione delle Comunità Sintetiche che permettono lo studio in laboratorio o l'applicazione in agricoltura per migliorare la resa delle colture vegetali
]

---

.center[.font150[Microbial DNA metabarcoding]]

&gt; **DNA metabarcoding** is a method of **DNA barcoding** that uses universal genetic markers to identify DNA of a mixture of organisms


???

.h201[

- Vediamo ora come studiare il microbiota con **tecniche coltura indipendenti**!
- Per studiare il microbiota e quindi identificare gli organismi lo compongono si utilizza una tecnica chiamata **DNA METABARCODING**

- Questa tecnica utilizza il **DNA barcoding** per studiare la composizione del microbiota

- **CHE COS'E' il DNA BARCODING"**
]

--

&lt;br&gt;
.content-box-grey[What is it the &lt;br&gt; **DNA barcoding**?]


---

.center[.font150[Microbial DNA metabarcoding]]

- **DNA barcoding** is a method of species identification using a short section of DNA from a specific gene 

- An individual sequence can be used to uniquely identify an organism by comparison with a reference library of classified DNA sequences


&lt;img src="images/DNAbarcoding.png" width="520px" style="display: block; margin: auto;" /&gt;

???

.h201[
- Il DNA Barcoding è una tecnica che permette di identificare l'organismo studiato basandosi su una sua specifica sequenza nucleotidica

- L'idea è utilizzare una sequenza di DNA per identificare un organismo in maniera univoca. 

- Dopo aver identificato la sequenza specifica, questa viene confrontata con un database di riferimento per essere classificata

- Un paragone potrebbe essere quello di utilizzare il vostro codica fiscale per identiifcarvi. La sequenza di lettere e numeri non mi da informazioni, ma cercandolo in un database potrei conosce il vostro nome e cognome  e identificarvi in modo univoco
]

---

.center[.font150[Microbial DNA metabarcoding]]

.pull-left[
**Marker selection**

.h201[
- Markers used for DNA barcoding are called barcodes and their selection is crucial

- Ideally, one gene sequence would be used for developing universal PCR primers for all taxonomic group
  
- However, no such gene region has been found yet!

- Different gene regions are used to identify the different organismal groups using barcoding
]
]

.pull-right[

&lt;img src="images/DNAbarcoding2.png" width="200px" style="display: block; margin: auto;" /&gt;

.right[.font50[(Mosa et al., 2019)]]
]

???

.h201[
- Siccome i microrganismi non sono registrati all'anagrafe e non hanno un loro CF, cosa possiamo usare come sequenza univoca per identificarli?

- La selezione di queste sequenze marker (**chiamate barcode**) è cruciale per l'identificazione dei microorganismi

- In un modo perfetto esisterebbe un unico barcode, cioè un singolo marker che ci permeterebbe si identificare e distinguere tutti i gruppi tassonomici

- Ma ovviamente non viviamo in un mondo ideale! E questo marker non è ancora stato individuato e probabilmente non lo sarà mai

- Quindi, diversi gruppi tassonomici usano diversi tipi di barcodes

- Sulla destra della slide vedete un tipico cromatogramma di sequenziamento. Da quest'immagine è facile capire perchè i markers sono chiamati barcodes
]

---

.center[.font150[Sequencing]]

**Sanger Sequencing**

.pull-left-narrow[
.h25[
&lt;br&gt;&lt;br&gt;
Sanger sequencing, also known as the “chain termination method”, is a method for determining the nucleotide sequence of DNA. The method was developed by two time Nobel Laureate Frederick Sanger and his colleagues in 1977, hence the name the Sanger Sequence
]
]

.pull-right-wide[
&lt;img src="images/sanger.png" width="820px" style="display: block; margin: auto;" /&gt;
]

---

.center[.font150[Microbial DNA metabarcoding]]

**Marker selection**

| Organism | Region |
|----------|--------|
| animals  | cytochrome c oxidase I|
| fungi    |  internal transcribed spacer (ITS) rRNA |
| plants   | RuBisCO |
| prokaryotes | 16S rRNA |
| microbial eukaryotes | 18S rRNA |


.font80[When barcoding is used to identify organisms from a sample containing DNA from more than one organism, the term DNA metabarcoding is used]

???

.h201[
- Vedete qui una lista di barcodes per diversi organismi

- Questi geni sono stati scelti a causa della dell'evoluzione lenta 

- COX1 enzima chiave del metabolismo aerobico

- Internal transcribed spacer (ITS): sono regioni non codificanti di DNA situate tra geni. ITS in partcilare tra le subunità dellRNA ribosomiale

- ribulosio-1,5-bisfosfato carbossilasi/ossigenasi: tramite la fissazione del carbonio, è il principale produttore di materia organica nell'ecosistema terrestre

- RNA ribosomiale 16 S (o rRNA 16 S) è una componente della subunità 30S di un ribosoma procariotico

- Il 18S è l'omologo eucariota del 16S dei procariotu
]

---

.center[.font150[Microbial DNA metabarcoding]]

**Metabarcoding process**

&lt;img src="images/barcoding.png" width="920px" style="display: block; margin: auto;" /&gt;

???

.h201[
- **Easy Poll**
]

---

.center[.font150[Microbial DNA metabarcoding]]

.pull-left[
.m0p[
**Bioinformatic analysis**
]
.h201[
- After sequencing, raw data must be processed (**quality and cleaning step**)

- Samples are clustered in operational taxonomic unit based on identity threshold (**OTU formation**)

- Using "the Basic Local Alignment Search Tool (BLAST)" regions of similarity between sequences and reference databases are identified (**annotation step**)    

- Microbiota composition definition (**richness and diversity step**)

]
]

.pull-right[

.h20[
.m0p[
.right[.font50[(Jesse et al., 2018)]]     
]
&lt;img src="images/OTU.png" width="220px" style="display: block; margin: auto;" /&gt;


]]



???

.h201[
- These databases contain the DNA barcodes assigned to previously identified taxa

- An operational taxonomic unit (OTU) is an operational definition used to classify groups of closely related individuals
]

---
layout: true

# What can the microbiota potentially do?

---

class: inverse, middle, center

----

???

.h201[
- Prendiamo ora in considerazione quella tecnica che oltre a permetterci di identificare i componenti del microbiota ci permette anche di capire cosa "potenzialmente" possono fare!
]
---

.center[.font120[Metagenomics]]

&lt;img src="images/metagenomics1.png" width="720px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Ye, Siddle, Park, and Sabeti, 2019)]]
.h25[
- **Metagenomics** is the study of genetic material recovered directly from environmental samples

- The collection of sequenced genes from the environment could be analyzed as a **single genome**
  
- Metagenomics **does not require the isolation** and lab cultivation of individual species

- Due to its random nature, metagenomics, allows that many **under-represented organisms will be detected**
]

???

.h201[
- La metagenomica studia il DNA isolato da un campione ambientale

- Questo campione può essere considerato come un singolo campione e quindi come tale analizzato

- La figura illustra la procedura di analisi di un campione di metagenomica. Come potete vedere 3 steps sono necessari:
  1. Estrazione
  1. Sequenziamento
  1. Classificazione
]

---

.center[Metagenomics vs Metabarcoding]
.pull-left[
.m0p[
&lt;img src="images/compareBarcodingMetagenome.png" width="450px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Gill, Baldock, Brown, Cresswell, Dicks, Fountain, Garratt, Gough, Heard, Holland, Ollerton, Stone, Tang, Vanbergen, Vogler, Woodward, Arce, Boatman, Brand-Hardy, and Potts, 2016)]]
]]


 .pull-right[
.h21[

- Metagenomics does not need the marker selection step

- Metagenomics sequences all the genes present in the sample from each organism

- Metagenomics can help to reconstruct large fragments or even complete genomes from organisms in a community

- Metagenomics allows the characterization of a large number of coding and non-coding sequences that can be used to decipher the microbial diversity or to understand its metabolic potential
]
]

???

.h201[
- La differenza fondamentale tra Metagenomica e Metabarcoding è che per la Metagenomica non serve alcun barcode (marker)

- Questo perchè in metagenomica tutto il DNA presente nel campione viene sequenziato

- Il risultato del sequenziamento permette di ricostruire gran parte dei geni presenti nel microbiota, permettondo così di capire quali organismi sono presenti ma anche cosa potenzialmente questi geni possono esprimere
]

---



&lt;img src="images/CompareHTS.png" width="920px" style="display: block; margin: auto;" /&gt;

???

.h201[
- Consideriamo il range tra il sanger sequencing method e 
llumina HiSeq
]

---

.center[
.middle[
&lt;iframe width="840" height="473" src="https://www.youtube.com/embed/fCd6B5HRaZ8" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen&gt;&lt;/iframe&gt;
]
https://bit.ly/2IGGiJE
]
---

.center[.font120[Genomes assembly]]

.m0p[
&lt;img src="images/Align.png" width="620px" style="display: block; margin: auto;" /&gt;
]

???

.h201[
- I frammenti generati dal sequenziamento devono poi essere **assemblati** per ricostruire il/i genomi presenti nel campione

- 2 tipi assembli a seconda che sia presente un genoma di riferimento

- Mappaggio per dire chi è più o meno abbondante
]

---
.font120[**Functional Metagenomics Analysis**]

.pull-left[
.m10ulp[
Workflow:

- Binning

- Metagenomic assembly

- Gene prediction

- Gene annotation 

- Reconstruction of metabolic pathways (achieved from enzyme-coding genes)
]]

.pull-right[
&lt;img src="images/metagenomicsPipeline.png" width="380px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Roumpeka, Wallace, Escalettes, Fotheringham, and Watson, 2017)]]
]

???

.h201[

- In questa slide vediamo come facciamo a definire quali sono i potenziali processi del microbiota analizzato

- In metagenomica, il binning è il processo di raggruppamento delle reads o contigs e di assegnarli a unità tassonomiche operative
]

---

.font120[**Functional Metagenomics Analysis**]

.pull-left[
.m10ulp[
Workflow:

- Binning

- Metagenomic assembly

- Gene prediction

- Gene annotation 

- Reconstruction of metabolic pathways (achieved from enzyme-coding genes)
]]

.pull-right[
&lt;img src="images/functionalMetagenomics.png" width="370px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Castañeda and Barbosa, 2017)]]
]

---
layout: true

# What are they doing?

---

class: inverse, center, middle

----

???

.h201[
- In fine vediamo come studiare le funzioni reali del microbiota
]
---

.white[Metatranscriptomics]

&lt;img src="images/Meta1.png" width="620px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Vanwonterghem, Jensen, Ho, Batstone, and Tyson, 2014)]]

???

.h201[
- Fin'ora ci siamo occupati di DNA metabarcoding e Metagenomics

- Abbiamo visto come il DNA Metabercoding risponda alla domanda "Chi è presente?"

- Il Metagenoma risponde sia alla domanda "Chi è presente?" sia alla domanda "Cosa potenzialmente puà fare il microbiota"

- Vediamo ora quali tecniche ci permettono di studiare le funzioni del microbiota:

1. Metatranscriptomics
1. Metaproteomics
1. Metabolomics
1. SIP: Stable isotop probing
]

---

Metatranscriptomics

&lt;img src="images/Meta2.png" width="620px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Vanwonterghem, Jensen, Ho, et al., 2014)]]

---

Metatranscriptomics
.pull-left[
.h25[
- Metatranscriptomics is the science that studies gene expression of microbes within natural environments

- It allows to obtain whole gene expression profiling of complex microbial communities

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; The advantage of metatranscriptomics, over metagenomics, is that it can provide information about differences in the active functions of microbial communities 

-  The overview of the gene expression in a given sample is obtained by capturing the total mRNA of the microbiome and by performing a whole metatranscriptomics shotgun sequencing.
]
]

.pull-right[
.m0p[
&lt;img src="images/metatrranscriptomePL.png" width="380px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Pérez-Pantoja and Tamames, 2015)]]
]
]

???

.h201[
- La metatranscrittomica studia l'espressione genica dei microorganismi nell'ambiente ovvero studia l'**mRNA**

- Essendo una tecnica basata sul sequenziamento permette di ottenere tutti gi mRNA messaggeri presenti nel campione e quindi di avere il profili di espressione dell'ologenoma

- Guardando la figura salla dx vediamo come si sviluppa schematicamente un'analisi di metatrascrittomica

- [...]

- Mi permetto di farvi notate la diversa dimensione dei 2 riquadri e dell'importanza che riveste oggi la parte di bioinformatica
]
---

Metaproteomics

&lt;img src="images/Meta3.png" width="620px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Vanwonterghem, Jensen, Ho, et al., 2014)]]

---

Metaproteomics

.pull-left[
.h25[
- Metaproteomics is the study of all protein samples recovered directly from environmental sources

- Metaproteomics have rapidly increased in recent years due to many technological advances in mass spectrometry (MS)

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; Metaproteomics assess the “expressed” metabolism and physiology of microbial community members, but also allows quantification of per-species biomass to determine community structure

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; Proteomics quantifies the real proteins expression level, while mRNA may be degraded or translated inefficiently, resulting in a incorrect estimation of protein expression level
]
]

.pull.right[
&lt;img src="images/metaproteomics.png" width="480px" style="display: block; margin: auto;" /&gt;
.right[.font30[https://www.ecu.edu.au/schools/science/research-activity/research-support/proteomics/what-is-lc-ms-proteomics]]
]

???

.h201[
- La Metaproteomica studia le proteine espresse dalla comunità di microorganismi presa in esame

- La proteomics lavora sulle proteine espresse, cioè i veri attori del metabolismo. A differenza di quanto fa la trascrittomica

- Come funziona la proteomica
]

---

Metabolomics

&lt;img src="images/Meta4.png" width="600px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Vanwonterghem, Jensen, Ho, et al., 2014)]]

---

Metabolomics

.pull-left[
.m0p[
.h25[

- Metabolomics is the study of the complete set of metabolites composition (the metabolome) of a cell type, tissue, or biological fluid

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; The metabolites (&lt;1.5 kDa) are the intermediates or end products of multiple enzymatic reactions and therefore are the most informative proxies of the biochemical activity of an organism

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm121.6 313.1c4.7 4.7 4.7 12.3 0 17L338 377.6c-4.7 4.7-12.3 4.7-17 0L256 312l-65.1 65.6c-4.7 4.7-12.3 4.7-17 0L134.4 338c-4.7-4.7-4.7-12.3 0-17l65.6-65-65.6-65.1c-4.7-4.7-4.7-12.3 0-17l39.6-39.6c4.7-4.7 12.3-4.7 17 0l65 65.7 65.1-65.6c4.7-4.7 12.3-4.7 17 0l39.6 39.6c4.7 4.7 4.7 12.3 0 17L312 256l65.6 65.1z"/&gt;&lt;/svg&gt; The metabolome consists of multiple compounds. Determine the entire set of metabolites is extremely difficult, further complicating the analyses is the dynamic nature of these metabolites.
]
]]

.pull-rigth[
&lt;br&gt;&lt;br&gt;
&lt;img src="images/metabolomics.png" width="550px" style="display: block; margin: auto;" /&gt;
]
.right[.font70[(Alonso, Marsal, and Julià, 2015)]]


???

.h201[
- La metabolomica è lo studio dei metaboliti

- I metaboliti sono molecole di dimensione inferiore ai 1.5 KDa e rappresentano gli intermedi o i prodotti finali delle reazioni enzimatiche 

- Ci informano per tanto sulle reali attività biochimiche in atto!!

- Esempi di metaboliti sono: carboidrati idrofilici, alcool volatili, chetoni, metaboliti secondari come antibiotici e pigmenti

- Descrizione figura

- Spettroscopia di risonanza magnetica nucleare
]


---

Stable/radioactive isotopes

&lt;img src="images/Meta5.png" width="600px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Vanwonterghem, Jensen, Ho, et al., 2014)]]

---

Stable/radioactive isotopes

.pull-left[
.m0p[
.h25[

- Stable-isotope probing (SIP) is a technique in microbial ecology for tracing fluxes of nutrients in biogeochemical cycling by microorganisms using heavy stable isotopes

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; SIP is an approach that can identify microbial populations with a defined function

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; The use of either DNA-SIP, RNA-SIP or protein-SIP can enable both the phylogenetic identification and the key metabolic genes/proteins

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm121.6 313.1c4.7 4.7 4.7 12.3 0 17L338 377.6c-4.7 4.7-12.3 4.7-17 0L256 312l-65.1 65.6c-4.7 4.7-12.3 4.7-17 0L134.4 338c-4.7-4.7-4.7-12.3 0-17l65.6-65-65.6-65.1c-4.7-4.7-4.7-12.3 0-17l39.6-39.6c4.7-4.7 12.3-4.7 17 0l65 65.7 65.1-65.6c4.7-4.7 12.3-4.7 17 0l39.6 39.6c4.7 4.7 4.7 12.3 0 17L312 256l65.6 65.1z"/&gt;&lt;/svg&gt; A major challenge associated with SIP is a very limited availability and high cost of labeled substrates. SIP is also very labor-intensive and low-throughput using current techniques
]
]]

.pull-rigth[
&lt;img src="images/sip1.png" width="280px" style="display: block; margin: auto;" /&gt;
]
.right[.font50[(Uhlik, Leewis, Strejcek, Musilova, Mackova, Leigh, and Macek, 2013)]]

???

.h201[
- La Stable isotope probing è una tecnica che utilizzando isotopi radioattivi traccia il flusso dei nutrienti utilizzati dai microorganismi

- Tra i pro di questa tecnica c'è la possibilità di studiare in detttaglio specifiche funzioni della popolazione microbica

- Per contro questa tecnica è molto complessa a livello pratico e molto costaosa a livello economico
]

---

Stable/radioactive isotopes

.m0p[
&lt;img src="images/sip2.png" width="570px" style="display: block; margin: auto;" /&gt;
.right[.font50[(el Zahar Haichar, Heulin, Guyonnet, et al., 2016)]]
]

---
layout: false
class: clear, middle, center

.m0p[
&lt;img src="images/sb1.png" width="850px" style="display: block; margin: auto;" /&gt;
]

???
metafora della torta e dei suoi ingredienti

---
class: clear


.center[.font150[System biology]]

.pull-left[
.m0p[
.h25u[
- .font80[Systems biology studies biological systems by systematically perturbing them]    

- .font80[Monitoring the gene, protein, metabolite, and informational pathway responses]    

- .font80[Integrating these data]    

- .font80[Formulating mathematical models that describe the structure of the system]    

- .font80[Predict its response to individual perturbations]
  ]]]

.pull-right[
.m0p[
&lt;img src="images/sb2.png" width="670px" style="display: block; margin: auto;" /&gt;

]
]

???

.h201[
- Qual è la tecnica per studiare al meglio il microbiota?

- La soluzione migliore è usarle tutte e combinarle insieme in quello che viene chiamato System biology

- Tutti questi dati integrati dovrebbero permetterci una migliore comprensione del sistema studiato, aiutarci a create dei modelli e grazie a questi modelli predirre le risposte del sistema
]

---

layout: true

# Questions about the lesson


---


.m0tbp[
.center[.font50px[&lt;svg style="height:0.8em;top:.04em;position:relative;fill:black;" viewBox="0 0 512 512"&gt;&lt;path d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zM262.655 90c-54.497 0-89.255 22.957-116.549 63.758-3.536 5.286-2.353 12.415 2.715 16.258l34.699 26.31c5.205 3.947 12.621 3.008 16.665-2.122 17.864-22.658 30.113-35.797 57.303-35.797 20.429 0 45.698 13.148 45.698 32.958 0 14.976-12.363 22.667-32.534 33.976C247.128 238.528 216 254.941 216 296v4c0 6.627 5.373 12 12 12h56c6.627 0 12-5.373 12-12v-1.333c0-28.462 83.186-29.647 83.186-106.667 0-58.002-60.165-102-116.531-102zM256 338c-25.365 0-46 20.635-46 46 0 25.364 20.635 46 46 46s46-20.636 46-46c0-25.365-20.635-46-46-46z"/&gt;&lt;/svg&gt;]]
]

.marco[
.center[.font90[[[&lt;svg style="height:0.8em;top:.04em;position:relative;fill:steelblue;" viewBox="0 0 448 512"&gt;&lt;path d="M94.12 315.1c0 25.9-21.16 47.06-47.06 47.06S0 341 0 315.1c0-25.9 21.16-47.06 47.06-47.06h47.06v47.06zm23.72 0c0-25.9 21.16-47.06 47.06-47.06s47.06 21.16 47.06 47.06v117.84c0 25.9-21.16 47.06-47.06 47.06s-47.06-21.16-47.06-47.06V315.1zm47.06-188.98c-25.9 0-47.06-21.16-47.06-47.06S139 32 164.9 32s47.06 21.16 47.06 47.06v47.06H164.9zm0 23.72c25.9 0 47.06 21.16 47.06 47.06s-21.16 47.06-47.06 47.06H47.06C21.16 243.96 0 222.8 0 196.9s21.16-47.06 47.06-47.06H164.9zm188.98 47.06c0-25.9 21.16-47.06 47.06-47.06 25.9 0 47.06 21.16 47.06 47.06s-21.16 47.06-47.06 47.06h-47.06V196.9zm-23.72 0c0 25.9-21.16 47.06-47.06 47.06-25.9 0-47.06-21.16-47.06-47.06V79.06c0-25.9 21.16-47.06 47.06-47.06 25.9 0 47.06 21.16 47.06 47.06V196.9zM283.1 385.88c25.9 0 47.06 21.16 47.06 47.06 0 25.9-21.16 47.06-47.06 47.06-25.9 0-47.06-21.16-47.06-47.06v-47.06h47.06zm0-23.72c-25.9 0-47.06-21.16-47.06-47.06 0-25.9 21.16-47.06 47.06-47.06h117.84c25.9 0 47.06 21.16 47.06 47.06 0 25.9-21.16 47.06-47.06 47.06H283.1z"/&gt;&lt;/svg&gt;](https://join.slack.com/t/ipma-2020/shared_invite/zt-j9o8ig7k-LN1~U3d6QJztquYbC6iW7w) Slack (https://bit.ly/3eRDBRL) - &lt;svg style="height:0.8em;top:.04em;position:relative;fill:steelblue;" viewBox="0 0 512 512"&gt;&lt;path d="M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7 22.4 17.4 52.1 39.5 154.1 113.6 21.1 15.4 56.7 47.8 92.2 47.6 35.7.3 72-32.8 92.3-47.6 102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4 132.7-96.3 142.8-104.7 173.4-128.7 5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9 30.6 23.9 40.7 32.4 173.4 128.7 16.8 12.2 50.2 41.8 73.4 41.4z"/&gt;&lt;/svg&gt;](mailto:marco.chiapello@unito.it) marco.chiapello@unito.it]
]
]


    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="libs/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"ratio": "16:9",
"highlightStyle": "github",
"highlightLines": true
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
