<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>Microbiota</title>
    <meta charset="utf-8" />
    <meta name="author" content="Marco Chiapello" />
    <script src="libs/header-attrs/header-attrs.js"></script>
    <link href="libs/tile-view/tile-view.css" rel="stylesheet" />
    <script src="libs/tile-view/tile-view.js"></script>
    <link rel="stylesheet" href="scrollable.css" type="text/css" />
    <link rel="stylesheet" href="mtheme_max.css" type="text/css" />
    <link rel="stylesheet" href="fonts_mtheme_max.css" type="text/css" />
    <link rel="stylesheet" href="lesson3.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">






class: title-slide



# .font170[MICROBIOMA E MICROBIOTA]


&lt;br&gt;&lt;br&gt;&lt;br&gt;&lt;br&gt;
.marco[
.tada[Marco Chiapello]
&lt;br&gt;
2020-11-24 
]


.marco[
.center[.font90[[[&lt;svg style="height:0.8em;top:.04em;position:relative;fill:steelblue;" viewBox="0 0 448 512"&gt;&lt;path d="M94.12 315.1c0 25.9-21.16 47.06-47.06 47.06S0 341 0 315.1c0-25.9 21.16-47.06 47.06-47.06h47.06v47.06zm23.72 0c0-25.9 21.16-47.06 47.06-47.06s47.06 21.16 47.06 47.06v117.84c0 25.9-21.16 47.06-47.06 47.06s-47.06-21.16-47.06-47.06V315.1zm47.06-188.98c-25.9 0-47.06-21.16-47.06-47.06S139 32 164.9 32s47.06 21.16 47.06 47.06v47.06H164.9zm0 23.72c25.9 0 47.06 21.16 47.06 47.06s-21.16 47.06-47.06 47.06H47.06C21.16 243.96 0 222.8 0 196.9s21.16-47.06 47.06-47.06H164.9zm188.98 47.06c0-25.9 21.16-47.06 47.06-47.06 25.9 0 47.06 21.16 47.06 47.06s-21.16 47.06-47.06 47.06h-47.06V196.9zm-23.72 0c0 25.9-21.16 47.06-47.06 47.06-25.9 0-47.06-21.16-47.06-47.06V79.06c0-25.9 21.16-47.06 47.06-47.06 25.9 0 47.06 21.16 47.06 47.06V196.9zM283.1 385.88c25.9 0 47.06 21.16 47.06 47.06 0 25.9-21.16 47.06-47.06 47.06-25.9 0-47.06-21.16-47.06-47.06v-47.06h47.06zm0-23.72c-25.9 0-47.06-21.16-47.06-47.06 0-25.9 21.16-47.06 47.06-47.06h117.84c25.9 0 47.06 21.16 47.06 47.06 0 25.9-21.16 47.06-47.06 47.06H283.1z"/&gt;&lt;/svg&gt;](https://join.slack.com/t/ipma-2020/shared_invite/zt-j9o8ig7k-LN1~U3d6QJztquYbC6iW7w) Slack - &lt;svg style="height:0.8em;top:.04em;position:relative;fill:steelblue;" viewBox="0 0 512 512"&gt;&lt;path d="M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7 22.4 17.4 52.1 39.5 154.1 113.6 21.1 15.4 56.7 47.8 92.2 47.6 35.7.3 72-32.8 92.3-47.6 102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4 132.7-96.3 142.8-104.7 173.4-128.7 5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9 30.6 23.9 40.7 32.4 173.4 128.7 16.8 12.2 50.2 41.8 73.4 41.4z"/&gt;&lt;/svg&gt;](mailto:marco.chiapello@unito.it) marco.chiapello@unito.it]
]
]

???

.n30[

- Sarò uno dei vostri docenti per il corso "Interazioni tra piante, microrganismi e ambiente"
]
---
layout: true

# Last lesson recap

---

.pull-left[
&lt;br&gt;
.m0p[
.h25o[
1. DNA Metabarcoding 

1. Several techniques can be employed to decipher the microbiota functions

1. System biology is the discipline that integrates the -omics techniques to formulate models that describe the structure of the system
]
]
]

---
.pull-left[
&lt;br&gt;
.m0p[
.h25o[
1. DNA Metabarcoding 

1. .opacity10[Several techniques can be employed to decipher the microbiota functions]

1. .opacity10[ System biology is the discipline that integrates the -omics techniques to formulate models that describe the structure of the system]
]
]
]

.pull-right[

.m0p[
&lt;img src="images/DNAbarcoding2.png" width="230px" style="display: block; margin: auto;" /&gt;
]
]

---

.pull-left[
&lt;br&gt;
.m0p[
.h25o[
1. .opacity10[DNA Metabarcoding] 

1. Several techniques can be employed to decipher the microbiota functions

1. .opacity10[ System biology is the discipline that integrates the -omics techniques to formulate models that describe the structure of the system]
]
]
]

.pull-right[

&lt;br&gt;&lt;br&gt;
&lt;img src="images/recap1.png" width="500px" style="display: block; margin: auto;" /&gt;
]

---

.pull-left[
&lt;br&gt;
.m0p[
.h25o[
1. .opacity10[DNA Metabarcoding] 

1. .opacity10[Several techniques can be employed to decipher the microbiota functions]

1. System biology is the discipline that integrates the -omics techniques to formulate models that describe the structure of the system
]
]
]

.pull-right[
&lt;br&gt;&lt;br&gt;
&lt;img src="images/recap2.png" width="300px" style="display: block; margin: auto;" /&gt;

]

---
layout: false
class: clear 

.pull-left-wide[
.font180[.bold[Agenda]]


- .font140[Functions of plant-associated microbiomes]

- .font140[Can the plant modify the composition of the associated microbiota?]

- .font140[Can we exploit the microbiota?]
]



---
layout: true
# Functions of plant-associated microbiomes

---

class: inverse, middle, center 

----

---

.pull-left[
.m0p[
&lt;img src="images/fuctionOverview.png" width="530px" style="display: block; margin: auto;" /&gt;
]]

.pull-right[

.h25up[
&lt;br&gt;
The plant-associated microbiome can provide benefits to the plant through various direct or indirect mechanisms

- nutrient acquisition (blue)
  
- stress control (green)
  
- defence against pathogens and pests (red)


]
]

---
class: clear, middle, center

# .black[Nutrient acquisition]

----

---
layout: true

# Nutrient acquisition

---

.pull-left[
&lt;br&gt;
&lt;img src="images/nutrientUptake.png" width="530px" style="display: block; margin: auto;" /&gt;
]

.pull-right[

.h25up[

**Plant microbiota has essential functions in improving plant nutrition**

- .font90[The molecular mechanisms driving nutrient acquisition have been thoroughly studied for plant symbioses with arbuscular mycorrhizal fungi (AMF) and Rhizobium bacteria]
  
- .font90[Non-symbiotic plant-growth-promoting bacteria can either enhance the bioavailability of insoluble minerals or improve the root system architecture of host plants, thus increasing the exploratory capacity of the root for water and minerals]

]
]


---

&lt;img src="images/nutrientPaper.png" width="830px" style="display: block; margin: auto;" /&gt;

---

## Introduction

.pull-left[
.h202[
.m0p[

- .font70[Nitrogen (N) is a limiting nutrient in many natural and managed ecosystems]

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M64 96H0c0 123.7 100.3 224 224 224v144c0 8.8 7.2 16 16 16h32c8.8 0 16-7.2 16-16V320C288 196.3 187.7 96 64 96zm384-64c-84.2 0-157.4 46.5-195.7 115.2 27.7 30.2 48.2 66.9 59 107.6C424 243.1 512 147.9 512 32h-64z"/&gt;&lt;/svg&gt; .font70[Arbuscular mycorrhizal (AM) fungi can substantially enhance plant N acquisition from soil, thereby potentially alleviating plant N limitation and playing an important role in plant productivity and soil nutrient cycling] 

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M64 96H0c0 123.7 100.3 224 224 224v144c0 8.8 7.2 16 16 16h32c8.8 0 16-7.2 16-16V320C288 196.3 187.7 96 64 96zm384-64c-84.2 0-157.4 46.5-195.7 115.2 27.7 30.2 48.2 66.9 59 107.6C424 243.1 512 147.9 512 32h-64z"/&gt;&lt;/svg&gt; .font70[Other soil biota with decomposer capabilities are key players in AM fungal N acquisition and transfer to plants]

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:grey;" viewBox="0 0 496 512"&gt;&lt;path d="M248 8C111.03 8 0 119.03 0 256s111.03 248 248 248 248-111.03 248-248S384.97 8 248 8zm0 432c-101.69 0-184-82.29-184-184 0-101.69 82.29-184 184-184 101.69 0 184 82.29 184 184 0 101.69-82.29 184-184 184zm0-312c-70.69 0-128 57.31-128 128s57.31 128 128 128 128-57.31 128-128-57.31-128-128-128zm0 192c-35.29 0-64-28.71-64-64s28.71-64 64-64 64 28.71 64 64-28.71 64-64 64z"/&gt;&lt;/svg&gt; .font70[Authors show that multipartite synergies between AM fungi and soil microbial communities substantially enhance plant and fungal N acquisition from organic matter and microbial acquisition of plant photosynthates]
]
]
]

.pull-right[
.m0p[
&lt;img src="images/nutr1.png" width="400px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Jacott, Murray, and Ridout, 2017)]]

]]

???

- Azoto

---

## Experimental design

.pull-left[
.h25up[
.m0tbp[

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:grey;" viewBox="0 0 448 512"&gt;&lt;path d="M437.2 403.5L320 215V64h8c13.3 0 24-10.7 24-24V24c0-13.3-10.7-24-24-24H120c-13.3 0-24 10.7-24 24v16c0 13.3 10.7 24 24 24h8v151L10.8 403.5C-18.5 450.6 15.3 512 70.9 512h306.2c55.7 0 89.4-61.5 60.1-108.5zM137.9 320l48.2-77.6c3.7-5.2 5.8-11.6 5.8-18.4V64h64v160c0 6.9 2.2 13.2 5.8 18.4l48.2 77.6h-172z"/&gt;&lt;/svg&gt; .font70[The experimental design allowed to assess the individual and combined contributions of AM fungi and the rest of the soil microbial community to plant N acquisition from organic matter]

- .font70[The 6 mesocosm treatments included]


.tt[
|Plot|Fungi|soil inocula| soil fertilization |
|---|:---:|:---:|:---:|
|control|&lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zM124 296c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h264c6.6 0 12 5.4 12 12v56c0 6.6-5.4 12-12 12H124z"/&gt;&lt;/svg&gt; | &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zM124 296c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h264c6.6 0 12 5.4 12 12v56c0 6.6-5.4 12-12 12H124z"/&gt;&lt;/svg&gt;| none |
|microbes (N0)| &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zM124 296c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h264c6.6 0 12 5.4 12 12v56c0 6.6-5.4 12-12 12H124z"/&gt;&lt;/svg&gt;| &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt;| 0 kg N / ha per year|
|AM fungi| &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt;| &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zM124 296c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h264c6.6 0 12 5.4 12 12v56c0 6.6-5.4 12-12 12H124z"/&gt;&lt;/svg&gt;| none |
|AM fungi + microbes (N196)| &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt;| &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt;| 196 kg N / ha per year|
|AM fungi + microbes (N28)| &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt;| &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt;| 28 kg N / ha per year|
|AM fungi + microbes (N0)| &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt;| &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt;| 0 kg N / ha per year|


]
]
]
]


.pull-right[
.m0p[

.center[.font80[Mesocosm design]]

&lt;img src="images/nutr2.png" width="200px" style="display: block; margin: auto;" /&gt;

.h20[
.m0tbp[
- Terrestrial ecosystems experience substantial N enrichment due to atmospheric deposition and fertilizer applications

- Long-term N enrichment of grassland soils results in substantial changes in microbial community structure and functional gene representation

- The particular mechanisms through which long-term N enrichment influences plant-biotic interactions and plant productivity are not fully understood
]

]]]


???

.h201[
- model grass Brachypodium distachyon

- inoculum of fresh grassland soil containing whole soil microbial communities that had been exposed to an N enrichment gradient for eight years

- Un mesocosmo ( meso o 'media' e -cosm 'mondo' ) è un qualsiasi sistema sperimentale all'aperto che esamina l'ambiente naturale in condizioni controllate. In questo modo gli studi mesocosmo forniscono un legame tra indagini sul campo e gli esperimenti di laboratorio altamente controllate.
]

---

## Plant N acquisition from organic matter

.pull-left[
.h25up[
.m0tbp[

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:grey;" viewBox="0 0 512 512"&gt;&lt;path d="M440.65 12.57l4 82.77A247.16 247.16 0 0 0 255.83 8C134.73 8 33.91 94.92 12.29 209.82A12 12 0 0 0 24.09 224h49.05a12 12 0 0 0 11.67-9.26 175.91 175.91 0 0 1 317-56.94l-101.46-4.86a12 12 0 0 0-12.57 12v47.41a12 12 0 0 0 12 12H500a12 12 0 0 0 12-12V12a12 12 0 0 0-12-12h-47.37a12 12 0 0 0-11.98 12.57zM255.83 432a175.61 175.61 0 0 1-146-77.8l101.8 4.87a12 12 0 0 0 12.57-12v-47.4a12 12 0 0 0-12-12H12a12 12 0 0 0-12 12V500a12 12 0 0 0 12 12h47.35a12 12 0 0 0 12-12.6l-4.15-82.57A247.17 247.17 0 0 0 255.83 504c121.11 0 221.93-86.92 243.55-201.82a12 12 0 0 0-11.8-14.18h-49.05a12 12 0 0 0-11.67 9.26A175.86 175.86 0 0 1 255.83 432z"/&gt;&lt;/svg&gt; .font70[Synergies emerging from these interactions far exceeded an additive effect on plant N acquisition]

- .font70[Plants grown with either soil microbes or AM fungi acquired twofold and threefold more N from the organic matter than control plants, respectively]

- .font70[Plants grown with both soil microbes and AM fungi acquired ten to twelvefold more N from the organic matter than control plants]

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zm-248 50c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z"/&gt;&lt;/svg&gt; .font70[This ten to twelvefold increase in plant N acquisition is more than double the expected increase in plant N acquisition based on the sum of N taken up by plants grown with free-living soil microbes or AM fungi alone]
]
]
]



.pull-right[
.m0p[
&lt;br&gt;
&lt;img src="images/nutr3.png" width="450px" style="display: block; margin: auto;" /&gt;

]]

---

&lt;br&gt;
&lt;img src="images/nutr4.png" width="1030px" style="display: block; margin: auto;" /&gt;
---

## Introduction

.pull-left[
.h25up[
.m0tbp[

- .font70[Climate change has altered plant phenology]

- .font70[Recent studies also suggest that root microbiota can contribute to phenotypic plasticity, which has important implications for our understanding of plant phenology in a changing climate]

- .font70[Flowering can be triggered by low nitrate levels]
  
- .font70[Auxins can be synthesized by rhizosphere microorganisms, raising the intriguing possibility that root microbiota may regulate plant growth and development through phytohormone production]

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:grey;" viewBox="0 0 496 512"&gt;&lt;path d="M248 8C111.03 8 0 119.03 0 256s111.03 248 248 248 248-111.03 248-248S384.97 8 248 8zm0 432c-101.69 0-184-82.29-184-184 0-101.69 82.29-184 184-184 101.69 0 184 82.29 184 184 0 101.69-82.29 184-184 184zm0-312c-70.69 0-128 57.31-128 128s57.31 128 128 128 128-57.31 128-128-57.31-128-128-128zm0 192c-35.29 0-64-28.71-64-64s28.71-64 64-64 64 28.71 64 64-28.71 64-64 64z"/&gt;&lt;/svg&gt; .font70[Authors identified a novel network of molecular interactions linking the nitrogen cycle, the phytohormone IAA produced from Tryptophan (Trp), and the timing of flowering]
]
]
]


.pull-right[
.m0p[
&lt;img src="images/nutr5.png" width="400px" style="display: block; margin: auto;" /&gt;

]]



???

.n30[

- This has crucial biological, physical, and chemical effects on the biosphere and the earth system
]

---

## Experimenta design

&lt;img src="images/nutr6.png" width="800px" style="display: block; margin: auto;" /&gt;

.pull-left[
.h20[
.m0ul[

- Plants growing in soil for three generations (G1, G2, or G3) inoculated with different soil microbiomes

- The *pgr5* mutant is deficient in antimycin A-sensitive cyclic flow from ferredoxin to plastoquinone, which is one of the most crucial physiological processes for efficient photosynthesis

]
]
]

.pull-right[
.h20[

- The pgr5 mutant produces different exudates from the Wt *Arabidopsis*

]]

---

## The richness and diversity of the rhizosphere 

.pull-left[
.h26[
.m0ul[

- .font80[Species richness and diversity significantly decreased after the two and three generations ]

- .font80[The relative abundances of Proteobacteria and Acidobacteria, the dominant bacterial phyla in the rhizosphere, decreased by generations 2 and 3 relative to generation 1]

- .font80[In contrast, Bacteroidetes and cyanobacterial abundances increased by generation 3]

]
]
]


.pull-right[
.m0p[
&lt;img src="images/nutr7.png" width="400px" style="display: block; margin: auto;" /&gt;
&lt;img src="images/nutr8.png" width="400px" style="display: block; margin: auto;" /&gt;

]]


???

.n30[

- abundance-based coverage (ACE)
]

---

## Flowering time is directly associated with the rhizosphere microbiota

.pull-left[
.f15l18[
- .font120[&lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M64 96H0c0 123.7 100.3 224 224 224v144c0 8.8 7.2 16 16 16h32c8.8 0 16-7.2 16-16V320C288 196.3 187.7 96 64 96zm384-64c-84.2 0-157.4 46.5-195.7 115.2 27.7 30.2 48.2 66.9 59 107.6C424 243.1 512 147.9 512 32h-64z"/&gt;&lt;/svg&gt; Microbiota from the third-generation pgr5 (PM) or Wt cultures (WM) was used to inoculate cultures of three *Arabidopsis* lines ]
  
  - Wt  

  - *pgr5*

  - *pns84*: deficient in cyclic electron flow from NADPH to plastoquinone


- .font120[&lt;svg style="height:0.8em;top:.04em;position:relative;fill:pink;" viewBox="0 0 448 512"&gt;&lt;path d="M413.03 256c40.13-54.89 41.51-98.62 25.14-128-10.91-19.52-40.54-50.73-116.33-41.88C300.36 34.89 267.64 0 224 0s-76.36 34.89-97.84 86.12C50.43 77.34 20.73 108.48 9.83 128c-16.38 29.4-15 73.09 25.14 128-40.13 54.89-41.51 98.62-25.14 128 29.21 52.34 101.68 43.58 116.33 41.88C147.63 477.1 180.36 512 224 512s76.37-34.9 97.84-86.12c14.64 1.7 87.11 10.46 116.33-41.88 16.38-29.4 15-73.09-25.14-128zM63.38 352c-4.03-7.21-.19-24.8 14.95-48.29 6.96 6.53 14.2 12.89 21.87 19.18 1.71 13.71 4 27.08 6.76 40.08-24.56.89-39.89-4.37-43.58-10.97zm36.82-162.88c-7.66 6.29-14.9 12.65-21.87 19.18-15.13-23.5-18.97-41.09-14.95-48.3 3.41-6.14 16.39-11.47 37.92-11.47 1.71 0 3.87.3 5.69.37a472.191 472.191 0 0 0-6.79 40.22zM224 64c9.47 0 22.2 13.52 33.86 37.26-11.19 3.7-22.44 8-33.86 12.86-11.42-4.86-22.67-9.16-33.86-12.86C201.8 77.52 214.53 64 224 64zm0 384c-9.47 0-22.2-13.52-33.86-37.26 11.19-3.7 22.44-8 33.86-12.86 11.42 4.86 22.67 9.16 33.86 12.86C246.2 434.48 233.47 448 224 448zm62.5-157.33c-26.7 19.08-46.14 29.33-62.5 37.48-16.35-8.14-35.8-18.41-62.5-37.48-1.99-27.79-1.99-41.54 0-69.33 26.67-19.05 46.13-29.32 62.5-37.48 16.39 8.17 35.86 18.44 62.5 37.48 1.98 27.78 1.99 41.53 0 69.33zM384.62 352c-3.67 6.62-19 11.82-43.58 10.95 2.76-13 5.05-26.37 6.76-40.06 7.66-6.29 14.9-12.65 21.87-19.18 15.13 23.49 18.97 41.08 14.95 48.29zm-14.95-143.71c-6.96-6.53-14.2-12.89-21.87-19.18a473.535 473.535 0 0 0-6.79-40.22c1.82-.07 3.97-.37 5.69-.37 21.52 0 34.51 5.34 37.92 11.47 4.02 7.22.18 24.81-14.95 48.3zM224 224c-17.67 0-32 14.33-32 32s14.33 32 32 32 32-14.33 32-32-14.33-32-32-32z"/&gt;&lt;/svg&gt; The addition of Wt microbiota delayed flowering by 3.3, 5.5, and 5.7 days in Wt, the pnsB4 mutant, and the pgr5 mutant, respectively  ]

- .font120[The shoot fresh weight of the plants treated with Wt microbiota also increased significantly in the three lines compared to the treatments with the addition of pgr5 microbiota]

- .font120[&lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zm-248 50c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z"/&gt;&lt;/svg&gt; These results clearly indicate that flowering time and shoot fresh weight are affected by the rhizosphere microbiota]

- .font120[&lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 448 512"&gt;&lt;path d="M323.56 51.2c-20.8 19.3-39.58 39.59-56.22 59.97C240.08 73.62 206.28 35.53 168 0 69.74 91.17 0 209.96 0 281.6 0 408.85 100.29 512 224 512s224-103.15 224-230.4c0-53.27-51.98-163.14-124.44-230.4zm-19.47 340.65C282.43 407.01 255.72 416 226.86 416 154.71 416 96 368.26 96 290.75c0-38.61 24.31-72.63 72.79-130.75 6.93 7.98 98.83 125.34 98.83 125.34l58.63-66.88c4.14 6.85 7.91 13.55 11.27 19.97 27.35 52.19 15.81 118.97-33.43 153.42z"/&gt;&lt;/svg&gt; This effect disappeared when the soil slurry was sterilized before inoculation, indicating that heat-stable exudates alone were not modulating *Arabidopsis* flowering time]

]]

.pull-right[
.m0p[
&lt;br&gt;
&lt;img src="images/nutr9.png" width="600px" style="display: block; margin: auto;" /&gt;

]]

---

## Linking N availability to flowering time

.pull-left[
.h202[
.m10bp[

- The enriched microbes in the WM treatment mostly have key roles in rhizosphere N regeneration

- Flowering can be triggered by low nitrate levels

- The authors tested whether WM treatment affects the abundance of genes involved in N cycling in soil

- The *nif* genes are genes encoding enzymes involved in the fixation of atmospheric nitrogen into a form of nitrogen available to living organisms

- The *amo*A gene is a gene involved in nitrification

- *nir*K and *nos*Z genes are involved in denitrification (spep2 and step4 of the nitrate reduction pathway)

]
]
]


.pull-right[
.m0p[
&lt;img src="images/nutr10.png" width="550px" style="display: block; margin: auto;" /&gt;

]]

???

.n30[
- La fissazione dell'azoto atmosferico o azotofissazione consiste nella riduzione, tramite la nitrogenasi, dell'azoto molecolare (N2) in azoto ammonico (NH3)

- La nitrificazione e' la trasformazione dell'ammoniaca (NH3) e ioni ammonio (NH4+) in ioni nitrito (NO2-) e successivamente ioni nitrato (NO3-).

- La denitrificazione e'  un processo nell'ambito del ciclo dell'azoto nel terreno, che comporta la riduzione dell'azoto nitrico con formazione di gas che si liberano nell'atmosfera
]

---
class: clear, middle, center

# .black[Stress control]

----

---
layout: true

# Stress control

---

.pull-left[

**Plant-associated microorganisms could modify plant evolutionary responses to environmental stress** in at least three non-mutually-exclusive pathways

.h202[
.m0ul[

- by altering the fitness of individual plant genotypes

- by altering the expression of plant traits related to fitness 
  
- by altering the strength or direction of natural selection occurring within populations that experience environmental stress

]
]
]

.pull-right[
.m0p[
&lt;br&gt;
&lt;img src="images/stress1.png" width="500px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Balestrini, Chitarra, Fotopoulos, and Ruocco, 2017)]]

]]

---

.pull-left[

**Responses to drought**

.h203[
.m0tp[

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:grey;" viewBox="0 0 448 512"&gt;&lt;path d="M437.2 403.5L320 215V64h8c13.3 0 24-10.7 24-24V24c0-13.3-10.7-24-24-24H120c-13.3 0-24 10.7-24 24v16c0 13.3 10.7 24 24 24h8v151L10.8 403.5C-18.5 450.6 15.3 512 70.9 512h306.2c55.7 0 89.4-61.5 60.1-108.5zM137.9 320l48.2-77.6c3.7-5.2 5.8-11.6 5.8-18.4V64h64v160c0 6.9 2.2 13.2 5.8 18.4l48.2 77.6h-172z"/&gt;&lt;/svg&gt; Experiment examines plant adaptation to drought stress in a multigeneration experiment that manipulated aboveground-belowground feedbacks between plants and soil microbial communitie

- Plant fitness in both drought and nondrought environments was linked strongly to the rapid responses of soil microbial community structure to moisture manipulations

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8c137 0 248 111 248 248S393 504 256 504 8 393 8 256 119 8 256 8zm-28.9 143.6l75.5 72.4H120c-13.3 0-24 10.7-24 24v16c0 13.3 10.7 24 24 24h182.6l-75.5 72.4c-9.7 9.3-9.9 24.8-.4 34.3l11 10.9c9.4 9.4 24.6 9.4 33.9 0L404.3 273c9.4-9.4 9.4-24.6 0-33.9L271.6 106.3c-9.4-9.4-24.6-9.4-33.9 0l-11 10.9c-9.5 9.6-9.3 25.1.4 34.4z"/&gt;&lt;/svg&gt; Plants were most fit when their contemporary environmental conditions (wet vs. dry soil) matched the historical environmental conditions (wet vs. dry soil) of their associated microbial community

- Plants may not be limited to "adapt or migrate" strategies
  
- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zm-248 50c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z"/&gt;&lt;/svg&gt; Plants may benefit from association with interacting species, especially diverse soil microbial communities, that respond rapidly to environmental change
]
]
]


.pull-right[
.m0p[
&lt;img src="images/stress2.png" width="400px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Lau and Lennon, 2012)]]

]]

---

.pull-left[

**Responses to drought**

.h203[
.m0tp[

- **How belowground microbial communities do affect plant fitness responses to drought stress?**

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zm-248 50c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z"/&gt;&lt;/svg&gt; Shifts in microbial community composition and bacterial diversity could be linked to changes in biogeochemical processes that influence the availability of resources, such as nitrogen (N), that commonly limit plant growth and fitness.

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zm-248 50c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z"/&gt;&lt;/svg&gt; Given that drought stress altered the composition of both bacterial and fungal communities, drought stress may have changed the relative abundances of mutualists and pathogens and also may have affected the fitness benefits of associating with mutualists and susceptibility to pathogens
]
]
]


.pull-right[
.m0p[
&lt;img src="images/stress2.png" width="400px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Lau and Lennon, 2012)]]

]]



---


.pull-left[

**Responses to salinity**

.h203[
.m0tp[

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 496 512"&gt;&lt;path d="M248 8C111.03 8 0 119.03 0 256s111.03 248 248 248 248-111.03 248-248S384.97 8 248 8zm0 432c-101.69 0-184-82.29-184-184 0-101.69 82.29-184 184-184 101.69 0 184 82.29 184 184 0 101.69-82.29 184-184 184zm0-312c-70.69 0-128 57.31-128 128s57.31 128 128 128 128-57.31 128-128-57.31-128-128-128zm0 192c-35.29 0-64-28.71-64-64s28.71-64 64-64 64 28.71 64 64-28.71 64-64 64z"/&gt;&lt;/svg&gt; Experiments aim to assess whether soil salinity changes the microbial community in the rhizosphere of *Hibiscus hamabo*, and whether these changes in the microbiome feedback on the growth of the plant

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:grey;" viewBox="0 0 448 512"&gt;&lt;path d="M437.2 403.5L320 215V64h8c13.3 0 24-10.7 24-24V24c0-13.3-10.7-24-24-24H120c-13.3 0-24 10.7-24 24v16c0 13.3 10.7 24 24 24h8v151L10.8 403.5C-18.5 450.6 15.3 512 70.9 512h306.2c55.7 0 89.4-61.5 60.1-108.5zM137.9 320l48.2-77.6c3.7-5.2 5.8-11.6 5.8-18.4V64h64v160c0 6.9 2.2 13.2 5.8 18.4l48.2 77.6h-172z"/&gt;&lt;/svg&gt; *H. hamabo* was grown in pots with a sand-soil mixture at different salt concentrations (0, 15, 40 and 90 mM NaCl). **Authors tested the effects of the salinity on the plants**
  
- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zm-248 50c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z"/&gt;&lt;/svg&gt; The salinity treatment  significantly affected the total biomass of *H. hamabo*. While root weight ratio was slightly affected

  
]
]
]


.pull-right[
.m0p[
&lt;img src="images/stress3.png" width="330px" style="display: block; margin: auto;" /&gt;
.right[.font20px[(Yuan, Brunel, van
Kleunen, Li, and Jin, 2019)]]
]
]

---


.pull-left[

**Responses to salinity**

.h203[
.m0tp[

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 496 512"&gt;&lt;path d="M248 8C111.03 8 0 119.03 0 256s111.03 248 248 248 248-111.03 248-248S384.97 8 248 8zm0 432c-101.69 0-184-82.29-184-184 0-101.69 82.29-184 184-184 101.69 0 184 82.29 184 184 0 101.69-82.29 184-184 184zm0-312c-70.69 0-128 57.31-128 128s57.31 128 128 128 128-57.31 128-128-57.31-128-128-128zm0 192c-35.29 0-64-28.71-64-64s28.71-64 64-64 64 28.71 64 64-28.71 64-64 64z"/&gt;&lt;/svg&gt; Experiments aim to assess whether soil salinity changes the microbial community in the rhizosphere of *Hibiscus hamabo*, and whether these changes in the microbiome feedback on the growth of the plant

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:grey;" viewBox="0 0 448 512"&gt;&lt;path d="M437.2 403.5L320 215V64h8c13.3 0 24-10.7 24-24V24c0-13.3-10.7-24-24-24H120c-13.3 0-24 10.7-24 24v16c0 13.3 10.7 24 24 24h8v151L10.8 403.5C-18.5 450.6 15.3 512 70.9 512h306.2c55.7 0 89.4-61.5 60.1-108.5zM137.9 320l48.2-77.6c3.7-5.2 5.8-11.6 5.8-18.4V64h64v160c0 6.9 2.2 13.2 5.8 18.4l48.2 77.6h-172z"/&gt;&lt;/svg&gt; *H. hamabo* was grown in pots with a sand-soil mixture at different salt concentrations (0, 15, 40 and 90 mM NaCl).  **Authors tested the effects of the salinity on the microbiotas**
  
- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zm-248 50c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z"/&gt;&lt;/svg&gt; There was a slight trend that **bacterial** (a-b) OTU richness and the Shannon indice decreased with increasing salinity (these effects were not significant) 

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zm-248 50c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z"/&gt;&lt;/svg&gt; The overall effects of the salinity treatments on **fungal** (d-e) OTU richness and the Shannon indice were not significant, but Least Significant Difference revealed significant differences between the 0 mM and 90 mM NaCl treatments

]
]
]


.pull-right[
&lt;img src="images/stress4.png" width="530px" style="display: block; margin: auto;" /&gt;
.right[.font20px[(Yuan Brunel, et al., 2019)]]
]

---

.pull-left[

**Responses to salinity**

.h203[
.m0tp[

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 496 512"&gt;&lt;path d="M248 8C111.03 8 0 119.03 0 256s111.03 248 248 248 248-111.03 248-248S384.97 8 248 8zm0 432c-101.69 0-184-82.29-184-184 0-101.69 82.29-184 184-184 101.69 0 184 82.29 184 184 0 101.69-82.29 184-184 184zm0-312c-70.69 0-128 57.31-128 128s57.31 128 128 128 128-57.31 128-128-57.31-128-128-128zm0 192c-35.29 0-64-28.71-64-64s28.71-64 64-64 64 28.71 64 64-28.71 64-64 64z"/&gt;&lt;/svg&gt; Experiments aim to assess whether soil salinity changes the microbial community in the rhizosphere of *Hibiscus hamabo*, and whether these changes in the microbiome feedback on the growth of the plant

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:grey;" viewBox="0 0 448 512"&gt;&lt;path d="M437.2 403.5L320 215V64h8c13.3 0 24-10.7 24-24V24c0-13.3-10.7-24-24-24H120c-13.3 0-24 10.7-24 24v16c0 13.3 10.7 24 24 24h8v151L10.8 403.5C-18.5 450.6 15.3 512 70.9 512h306.2c55.7 0 89.4-61.5 60.1-108.5zM137.9 320l48.2-77.6c3.7-5.2 5.8-11.6 5.8-18.4V64h64v160c0 6.9 2.2 13.2 5.8 18.4l48.2 77.6h-172z"/&gt;&lt;/svg&gt; *H. hamabo* was grown in pots with root-inoculum addition from previously treated soils (i.e. control with sterilized root inoculum vs the treatments with root inoculum)
  
- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zm-248 50c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z"/&gt;&lt;/svg&gt; The addition of a live inoculum significantly increased **germination rate**

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zm-248 50c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z"/&gt;&lt;/svg&gt; The addition of a live inoculum significantly **increased biomass**, effect was stronger at salinities of 0 and 40 mM than at a salinity of 90 mM
]
]
]


.pull-right[
&lt;img src="images/stress5.png" width="330px" style="display: block; margin: auto;" /&gt;
.right[.font20px[(Yuan Brunel, et al., 2019)]]
]

---
class: clear, middle, center

# .black[Disease resistance]

----

---
layout: true

# Disease resistance

---


- **Disease-suppressive soils** are exceptional ecosystems in which crop plants suffer less from specific soil-borne pathogens than expected owing to the activities of other soil microorganisms

  


&lt;img src="images/disease4.png" width="650px" style="display: block; margin: auto;" /&gt;
.right[.font20px[(Berendsen, Pieterse, and Bakker, 2012)]]



---

.pull-left[

**Disease resistence**

.h202[
.m0tp[

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 496 512"&gt;&lt;path d="M248 8C111.03 8 0 119.03 0 256s111.03 248 248 248 248-111.03 248-248S384.97 8 248 8zm0 432c-101.69 0-184-82.29-184-184 0-101.69 82.29-184 184-184 101.69 0 184 82.29 184 184 0 101.69-82.29 184-184 184zm0-312c-70.69 0-128 57.31-128 128s57.31 128 128 128 128-57.31 128-128-57.31-128-128-128zm0 192c-35.29 0-64-28.71-64-64s28.71-64 64-64 64 28.71 64 64-28.71 64-64 64z"/&gt;&lt;/svg&gt; The aim of the study was to decipher the rhizosphere microbiome to identify such disease-suppressive microbes and to unravel the mechanisms by which they protect plants against root diseases.

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:grey;" viewBox="0 0 448 512"&gt;&lt;path d="M437.2 403.5L320 215V64h8c13.3 0 24-10.7 24-24V24c0-13.3-10.7-24-24-24H120c-13.3 0-24 10.7-24 24v16c0 13.3 10.7 24 24 24h8v151L10.8 403.5C-18.5 450.6 15.3 512 70.9 512h306.2c55.7 0 89.4-61.5 60.1-108.5zM137.9 320l48.2-77.6c3.7-5.2 5.8-11.6 5.8-18.4V64h64v160c0 6.9 2.2 13.2 5.8 18.4l48.2 77.6h-172z"/&gt;&lt;/svg&gt; The investigated soil is suppressive to *Rhizoctonia solani*, an economically important fungal pathogen of many crops 
]
]

.h20[
.m0tbp[

- suppressive soil (S)
  
- conducive soil (C)
  
- conducive soil amended with 10% (w/w) of suppressive soil (CS)

- suppressive soil heat-treated at 50C (S50)

- suppressive soil heat-treated at 80C (S80)
  
]
]
]


.pull-right[
&lt;br&gt;
.m0p[
&lt;img src="images/disease2.png" width="530px" style="display: block; margin: auto;" /&gt;
.right[.font20px[(Mendes, Kruijt, de
Bruijn, Dekkers, van der
Voort, Schneider, Piceno, DeSantis, Andersen, Bakker, and Raaijmakers, 2011)]]

]
]


---

.pull-left[

**Disease resistence**

.h202[
.m0tp[

- A total of 33,346 bacterial and archaeal OTUs were detected in the rhizosphere microbiome 

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zM124 296c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h264c6.6 0 12 5.4 12 12v56c0 6.6-5.4 12-12 12H124z"/&gt;&lt;/svg&gt; When comparing the six soil treatments with different levels of disease suppressiveness, no significant differences were found in the number of detected bacterial taxa

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; When the abundance of the detected taxa was taken into account, six clusters of samples that corresponded to the six soil treatments were found

.content-box-grey[
.m0tbp[.font20px[These results suggest that &lt;br&gt;**the relative abundance of several bacterial taxa**&lt;br&gt; is a **more important** indicator of disease suppression&lt;br&gt; than the exclusive **presence of specific bacterial taxa**
]]]
  
]
]
]


.pull-right[
&lt;img src="images/disease3.png" width="340px" style="display: block; margin: auto;" /&gt;
.right[.font20px[(Mendes Kruijt, et al., 2011)]]

]

???

.n30[

- Paragone con ricetta di cucina in cui si ha la lista degli ingredienti MA soprattutto e' importante la quantita'
]

---

&lt;br&gt;&lt;br&gt;&lt;br&gt;
.center[.font150[How soil-born microbes control the plant disease?]]

----

---

.pull-left[
.h25[
.m0ul[
&lt;br&gt;
- (a) **Antibiosis**: The bacterium colonizes the growing root system and delivers antibiotic molecules around the root, thereby harming pathogens that approach the root

- (b) **Induced systemic resistance** (ISR): many bacterial products induce systemic signaling, which can result in protection of the whole plant against diseases caused by different organisms
  
- (c) **Competition for nutrients and niches**: Biocontrol bacteria acting through this mechanism excel in fast chemotactic movement along the growing root in their efficient hunt for root exudate components, thereby outcompeting the pathogen in scavenging nutrients and in occupying niches on the root

]
]
]

.pull-right[
&lt;br&gt;
&lt;img src="images/disease5.png" width="400px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Lugtenberg and Kamilova, 2009)]]

]

---

.pull-left[

**Antibiosis**

.h202[
.m0ul[

- *Dematophora necatrix* is the cause of avocado Dematophora root rot (also called white root rot)

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:grey;" viewBox="0 0 448 512"&gt;&lt;path d="M437.2 403.5L320 215V64h8c13.3 0 24-10.7 24-24V24c0-13.3-10.7-24-24-24H120c-13.3 0-24 10.7-24 24v16c0 13.3 10.7 24 24 24h8v151L10.8 403.5C-18.5 450.6 15.3 512 70.9 512h306.2c55.7 0 89.4-61.5 60.1-108.5zM137.9 320l48.2-77.6c3.7-5.2 5.8-11.6 5.8-18.4V64h64v160c0 6.9 2.2 13.2 5.8 18.4l48.2 77.6h-172z"/&gt;&lt;/svg&gt; A set of eight bacterial strains was selected on the basis of growth inhibitory activity against *D. necatrix*. Upon testing the biocontrol ability of these strains in a newly developed avocado test system and in a tomato test system, it became apparent that PCL1606 exhibited the highest biocontrol ability

- This compound was purified and subsequently identified as 2-hexyl 5-propyl resorcinol (HPR).

]
]
]

.pull-right[
&lt;br&gt;
&lt;img src="images/disease6.png" width="500px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Cazorla, Duckett, Bergström, Noreen, Odijk, Lugtenberg, Thomas-Oates, and Bloemberg, 2006)]]

]

---

.pull-left[

**Antibiosis**

.h202[
.m0ul[

- *Dematophora necatrix* is the cause of avocado Dematophora root rot (also called white root rot)

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:grey;" viewBox="0 0 448 512"&gt;&lt;path d="M437.2 403.5L320 215V64h8c13.3 0 24-10.7 24-24V24c0-13.3-10.7-24-24-24H120c-13.3 0-24 10.7-24 24v16c0 13.3 10.7 24 24 24h8v151L10.8 403.5C-18.5 450.6 15.3 512 70.9 512h306.2c55.7 0 89.4-61.5 60.1-108.5zM137.9 320l48.2-77.6c3.7-5.2 5.8-11.6 5.8-18.4V64h64v160c0 6.9 2.2 13.2 5.8 18.4l48.2 77.6h-172z"/&gt;&lt;/svg&gt; A set of eight bacterial strains was selected on the basis of growth inhibitory activity against *D. necatrix*. Upon testing the biocontrol ability of these strains in a newly developed avocado test system and in a tomato test system, it became apparent that PCL1606 exhibited the highest biocontrol ability

- This compound was purified and subsequently identified as 2-hexyl 5-propyl resorcinol (HPR).

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:grey;" viewBox="0 0 448 512"&gt;&lt;path d="M437.2 403.5L320 215V64h8c13.3 0 24-10.7 24-24V24c0-13.3-10.7-24-24-24H120c-13.3 0-24 10.7-24 24v16c0 13.3 10.7 24 24 24h8v151L10.8 403.5C-18.5 450.6 15.3 512 70.9 512h306.2c55.7 0 89.4-61.5 60.1-108.5zM137.9 320l48.2-77.6c3.7-5.2 5.8-11.6 5.8-18.4V64h64v160c0 6.9 2.2 13.2 5.8 18.4l48.2 77.6h-172z"/&gt;&lt;/svg&gt; To study the role of HPR in biocontrol activity, two mutants of PCL1606 impaired in antagonistic activity were selected

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; These mutants were shown to impair HRP production and showed a decrease in biocontrol activity

]
]
]

.pull-right[
&lt;img src="images/disease8.png" width="350px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Cazorla Duckett, et al., 2006)]]

]

???

.n30[

- Biocontrol strains Pseudomonas fluorescens WCS365 and P. chlororaphis PCL1391 were tested as controls

]

---

.pull-left[

**Induced systemic resistance**

.h27[
- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:#99C6E8;" viewBox="0 0 640 512"&gt;&lt;path d="M384 320H256c-17.67 0-32 14.33-32 32v128c0 17.67 14.33 32 32 32h128c17.67 0 32-14.33 32-32V352c0-17.67-14.33-32-32-32zM192 32c0-17.67-14.33-32-32-32H32C14.33 0 0 14.33 0 32v128c0 17.67 14.33 32 32 32h95.72l73.16 128.04C211.98 300.98 232.4 288 256 288h.28L192 175.51V128h224V64H192V32zM608 0H480c-17.67 0-32 14.33-32 32v128c0 17.67 14.33 32 32 32h128c17.67 0 32-14.33 32-32V32c0-17.67-14.33-32-32-32z"/&gt;&lt;/svg&gt; **Sistemic Acquired Resistance** (SAR) corresponds to a plant "vaccination" against a broad range of pathogens. Induced by local contact with MAMPs, PAMPs or effectors. SAR is mainly mediated and dependent on Salicylic Acid


- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:#F2AD85;" viewBox="0 0 512 512"&gt;&lt;path d="M255.545 8c-66.269.119-126.438 26.233-170.86 68.685L48.971 40.971C33.851 25.851 8 36.559 8 57.941V192c0 13.255 10.745 24 24 24h134.059c21.382 0 32.09-25.851 16.971-40.971l-41.75-41.75c30.864-28.899 70.801-44.907 113.23-45.273 92.398-.798 170.283 73.977 169.484 169.442C423.236 348.009 349.816 424 256 424c-41.127 0-79.997-14.678-110.63-41.556-4.743-4.161-11.906-3.908-16.368.553L89.34 422.659c-4.872 4.872-4.631 12.815.482 17.433C133.798 479.813 192.074 504 256 504c136.966 0 247.999-111.033 248-247.998C504.001 119.193 392.354 7.755 255.545 8z"/&gt;&lt;/svg&gt; **Induced Systemic Resistance** (ISR) is initiated in roots by plant-growth- promoting microbes (PGPM) and leads to resistance priming in distant parts of the plant ISR is mainly mediated and dependent by Jasmonic Acid (JA) and Ethylene (ET).

]
]

.pull-right[
&lt;br&gt;
&lt;img src="images/disease1.png" width="400px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Burketova, Trda, Ott, and Valentova, 2015)]]

]

???

.n30[

- Microbe-Associated Molecular Patterns (MAMPs)

- Pathogen-Associated Molecular Patterns (PAMPs)
  
- Their presence is detected by members of a large family of pattern recognition receptors (PRRs)

- Damage-Associated Molecular Patterns (DAMPs)

- Plant Growth-Promoting Rhizobacteria (PGPR) and Fungi (PGPF)

- Many individual bacterial components induce ISR, such as LPS, flagella, salicylic acid, and siderophores
G
]

---

.pull-left[

**Induced systemic resistance**

.h202[
- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 496 512"&gt;&lt;path d="M248 8C111.03 8 0 119.03 0 256s111.03 248 248 248 248-111.03 248-248S384.97 8 248 8zm0 432c-101.69 0-184-82.29-184-184 0-101.69 82.29-184 184-184 101.69 0 184 82.29 184 184 0 101.69-82.29 184-184 184zm0-312c-70.69 0-128 57.31-128 128s57.31 128 128 128 128-57.31 128-128-57.31-128-128-128zm0 192c-35.29 0-64-28.71-64-64s28.71-64 64-64 64 28.71 64 64-28.71 64-64 64z"/&gt;&lt;/svg&gt; Determine whether root colonisation by *Trichoderma asperellum* isolate T34 (T34) can enhance resistance in *Arabidopsis* against foliar pathogens


- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:grey;" viewBox="0 0 448 512"&gt;&lt;path d="M437.2 403.5L320 215V64h8c13.3 0 24-10.7 24-24V24c0-13.3-10.7-24-24-24H120c-13.3 0-24 10.7-24 24v16c0 13.3 10.7 24 24 24h8v151L10.8 403.5C-18.5 450.6 15.3 512 70.9 512h306.2c55.7 0 89.4-61.5 60.1-108.5zM137.9 320l48.2-77.6c3.7-5.2 5.8-11.6 5.8-18.4V64h64v160c0 6.9 2.2 13.2 5.8 18.4l48.2 77.6h-172z"/&gt;&lt;/svg&gt; The severity of bacterial speck disease inflicted by *Pseudomonas syringae*  was quantified for control, T34-treated plants and *Pseudomonas fluorescens* WCS417r bacteria (non-pathogenic)

]
]

.pull-right[
&lt;br&gt;
&lt;img src="images/disease9.png" width="350px" style="display: block; margin: auto;" /&gt;
.right[.font20px[(Segarra, der
Ent, Trillas, and Pieterse, 2009)]]

]


---

.pull-left[

**Induced systemic resistance**

.h202[
- WCS417r is known to be effective against a broad range of pathogens, including obligate biotrophs and necrotrophic fungi 

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 496 512"&gt;&lt;path d="M248 8C111.03 8 0 119.03 0 256s111.03 248 248 248 248-111.03 248-248S384.97 8 248 8zm0 432c-101.69 0-184-82.29-184-184 0-101.69 82.29-184 184-184 101.69 0 184 82.29 184 184 0 101.69-82.29 184-184 184zm0-312c-70.69 0-128 57.31-128 128s57.31 128 128 128 128-57.31 128-128-57.31-128-128-128zm0 192c-35.29 0-64-28.71-64-64s28.71-64 64-64 64 28.71 64 64-28.71 64-64 64z"/&gt;&lt;/svg&gt; To investigate whether T34-ISR is similarly effective against these types of pathogens 

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:grey;" viewBox="0 0 448 512"&gt;&lt;path d="M437.2 403.5L320 215V64h8c13.3 0 24-10.7 24-24V24c0-13.3-10.7-24-24-24H120c-13.3 0-24 10.7-24 24v16c0 13.3 10.7 24 24 24h8v151L10.8 403.5C-18.5 450.6 15.3 512 70.9 512h306.2c55.7 0 89.4-61.5 60.1-108.5zM137.9 320l48.2-77.6c3.7-5.2 5.8-11.6 5.8-18.4V64h64v160c0 6.9 2.2 13.2 5.8 18.4l48.2 77.6h-172z"/&gt;&lt;/svg&gt; The level of T34-induced protection against the biotrophic oomycete *Hyaloperonospora parasitica* and the necrotrophic fungus *Plectosphaerella cucumerina* was assessed.

.content-box-grey[
.font20px[These results demonstrate that colonisation 
&lt;br&gt;of *Arabidopsis* roots by T34
&lt;br&gt;**triggers a systemic resistance response**
&lt;br&gt;that is effective against different types of foliar pathogens]
]
]
]

.pull-right[
&lt;br&gt;
&lt;img src="images/disease10.png" width="350px" style="display: block; margin: auto;" /&gt;
.pull-left[
.h12[
.m0tbp[
.m0tbul[
- *Class I*: no sporulation

- *Class II*: trailing necrosis

- *Class III*: &lt;50% of the leaf area covered with sporangia

- *Class IV*: &gt;50% of the leaf area covered with sporangia, with additional chlorosis and leaf collapse
]
]
]
]

.pull-right[
.h12[
.m0tbp[
.m0tbul[
- *Class I*: no symptom

- *Class II*: lesion diameter &lt;2 mm

- *Class III*: lesion diameter &gt;2 mm

]
]
]
]
.whole[
.right[.font20px[(Segarra der
Ent, et al., 2009)]]

]
]


---

.pull-left[

**Induced systemic resistance**

.h202[

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 496 512"&gt;&lt;path d="M248 8C111.03 8 0 119.03 0 256s111.03 248 248 248 248-111.03 248-248S384.97 8 248 8zm0 432c-101.69 0-184-82.29-184-184 0-101.69 82.29-184 184-184 101.69 0 184 82.29 184 184 0 101.69-82.29 184-184 184zm0-312c-70.69 0-128 57.31-128 128s57.31 128 128 128 128-57.31 128-128-57.31-128-128-128zm0 192c-35.29 0-64-28.71-64-64s28.71-64 64-64 64 28.71 64 64-28.71 64-64 64z"/&gt;&lt;/svg&gt; How T34 activates ISR response in *Arabidopsis*?

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:grey;" viewBox="0 0 448 512"&gt;&lt;path d="M437.2 403.5L320 215V64h8c13.3 0 24-10.7 24-24V24c0-13.3-10.7-24-24-24H120c-13.3 0-24 10.7-24 24v16c0 13.3 10.7 24 24 24h8v151L10.8 403.5C-18.5 450.6 15.3 512 70.9 512h306.2c55.7 0 89.4-61.5 60.1-108.5zM137.9 320l48.2-77.6c3.7-5.2 5.8-11.6 5.8-18.4V64h64v160c0 6.9 2.2 13.2 5.8 18.4l48.2 77.6h-172z"/&gt;&lt;/svg&gt; Assessed T34-induced resistance in:
  - *sid2-1*: salicylic acid impaired mutant (**SAR**)    
  - *npr1-1*: disrupted both **SAR** and **ISR** response
  - *myb72-1*: specifically affected in early steps of **ISR** signalling, but not in component required for general ET or JA signalling    

.m0tbp[
- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; *sid2-1* developed a similar level of resistance as wildtype indicating that T34-ISR functions independently of SA

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; *npr1-1* is blocked in its ability to mount ISR, indicating that the regulatory protein NPR1 is required for expression of this type of Trichoderma-induced resistance

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; the root-specific transcription factor MYB72 plays an important role in early signalling steps of ISR and is required for T34-ISR
]
]
]

.pull-right[
&lt;br&gt;
&lt;img src="images/disease11.png" width="400px" style="display: block; margin: auto;" /&gt;

.right[.font20px[(Segarra der
Ent, et al., 2009)]]

]


---


.lh18[
**Priming:** .font60[&lt;svg style="height:0.8em;top:.04em;position:relative;fill:grey;" viewBox="0 0 512 512"&gt;&lt;path d="M505 442.7L405.3 343c-4.5-4.5-10.6-7-17-7H372c27.6-35.3 44-79.7 44-128C416 93.1 322.9 0 208 0S0 93.1 0 208s93.1 208 208 208c48.3 0 92.7-16.4 128-44v16.3c0 6.4 2.5 12.5 7 17l99.7 99.7c9.4 9.4 24.6 9.4 33.9 0l28.3-28.3c9.4-9.4 9.4-24.6.1-34zM208 336c-70.7 0-128-57.2-128-128 0-70.7 57.2-128 128-128 70.7 0 128 57.2 128 128 0 70.7-57.2 128-128 128z"/&gt;&lt;/svg&gt; sensitization of the whole plant for enhanced defense; characterized by a faster and stronger activation of cellular defenses upon invasion]
]

.m0tbp[
&lt;img src="images/disease12.png" width="420px" style="display: block; margin: auto;" /&gt;
]
.right[.font80[(Pieterse, Zamioudis, Berendsen, Weller, Wees, and Bakker, 2014)]]

---

.pull-left[

**Competition for nutrients and niches**

.h20[
.m0ul[

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 496 512"&gt;&lt;path d="M248 8C111.03 8 0 119.03 0 256s111.03 248 248 248 248-111.03 248-248S384.97 8 248 8zm0 432c-101.69 0-184-82.29-184-184 0-101.69 82.29-184 184-184 101.69 0 184 82.29 184 184 0 101.69-82.29 184-184 184zm0-312c-70.69 0-128 57.31-128 128s57.31 128 128 128 128-57.31 128-128-57.31-128-128-128zm0 192c-35.29 0-64-28.71-64-64s28.71-64 64-64 64 28.71 64 64-28.71 64-64 64z"/&gt;&lt;/svg&gt; To study tomato foot and root rot, a plant disease caused by the fungus *Fusarium oxysporum f.sp. radicis-lycopersici*, and the disease control through the mechanism "competition for nutrients and niches"

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:grey;" viewBox="0 0 448 512"&gt;&lt;path d="M437.2 403.5L320 215V64h8c13.3 0 24-10.7 24-24V24c0-13.3-10.7-24-24-24H120c-13.3 0-24 10.7-24 24v16c0 13.3 10.7 24 24 24h8v151L10.8 403.5C-18.5 450.6 15.3 512 70.9 512h306.2c55.7 0 89.4-61.5 60.1-108.5zM137.9 320l48.2-77.6c3.7-5.2 5.8-11.6 5.8-18.4V64h64v160c0 6.9 2.2 13.2 5.8 18.4l48.2 77.6h-172z"/&gt;&lt;/svg&gt; Microbes were isolated from the rhizospheres of tomato plants. Grown briefly in liquid and used to inoculate sterile germinated seedlings. After those microbes had reached the 1-cm-long root tip were shaken off the root tip, plated on KB agar, judged for colony diversity, and the cells from the combined colonies were used for another enrichment cycle. After a total of three cycles the bacteria were selected for competitive root colonization experiments.
  
-  &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; The observation that bacteria selected by the enrichment method grow much better on exudate than random rhizobacteria shows that these strains utilize exudates components more efficiently for growth. It suggests that competition for nutrients plays a major role in the biocontrol activity of the enhanced colonizers
]
]
]

.pull-right[
&lt;br&gt;&lt;br&gt;
&lt;img src="images/disease13.png" width="500px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Kamilova, Validov, Azarova, Mulders, and Lugtenberg, 2005)]]

]

???

.n30[
- TSA: tryptic soy agar
]


---
layout: true
# Can the plant modify the composition of the associated microbiota?

---

class: inverse, middle, center 

----

---
layout: true
# Plants modify the composition of microbiota

---

&lt;img src="images/shape1.png" width="900px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Lakshmanan, Selvaraj, and Bais, 2014)]]

---

.pull-left[

**Plant sensing and the initiation of colonization**

.h202o[
.m0ul[

1. Plants reliese exudates

  - **Plant exudates**: complex mixtures of soluble organic substances that are secreted by living plants

1. Plant-associated microorganisms use **chemotaxis** to sense and respond to plant-derived signals

1. Once a signal is perceived, **microorganisms move towards the plant** with the use of flagella

1. Microorganisms attach to the root surface and form a **biofilm**

1. Genes that encode proteins involved in bacterial chemotaxis, flagella assembly, bacterial motility and biofilm formation are **highly abundant in the rhizosphere compared with the bulk soil**

]
]
]

.pull-right[
&lt;br&gt;
&lt;img src="images/shape2.png" width="700px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Trivedi, Leach, Tringe, Sa, and Singh, 2020)]]

]
---

.pull-left[
**How plant can recruit beneficial microbes** 
.m0tp[
.h202[

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:#FAAD18;" viewBox="0 0 352 512"&gt;&lt;path d="M96.06 454.35c.01 6.29 1.87 12.45 5.36 17.69l17.09 25.69a31.99 31.99 0 0 0 26.64 14.28h61.71a31.99 31.99 0 0 0 26.64-14.28l17.09-25.69a31.989 31.989 0 0 0 5.36-17.69l.04-38.35H96.01l.05 38.35zM0 176c0 44.37 16.45 84.85 43.56 115.78 16.52 18.85 42.36 58.23 52.21 91.45.04.26.07.52.11.78h160.24c.04-.26.07-.51.11-.78 9.85-33.22 35.69-72.6 52.21-91.45C335.55 260.85 352 220.37 352 176 352 78.61 272.91-.3 175.45 0 73.44.31 0 82.97 0 176zm176-80c-44.11 0-80 35.89-80 80 0 8.84-7.16 16-16 16s-16-7.16-16-16c0-61.76 50.24-112 112-112 8.84 0 16 7.16 16 16s-7.16 16-16 16z"/&gt;&lt;/svg&gt; The plant-associated **microbiota is shaped** by complex interactions among the plant, microorganisms and the environment

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zm-248 50c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z"/&gt;&lt;/svg&gt; Metabolic diversification of root exudates within the plant kingdom may provide a basis for communication and recognition that directs the **assembly and maintenance** of a distinct microbiota tailored to the needs of the host

- Plant in response to local environments can **recruit distinct plant-associated microbial communities**

]
]
]

.pull-right[
&lt;br&gt;
&lt;img src="images/shape3.png" width="500px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Berendsen Pieterse, et al., 2012)]]
]

---

.pull-left[
**How plant can recruit beneficial microbes** 
.m0tp[
.h202[


- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 496 512"&gt;&lt;path d="M248 8C111.03 8 0 119.03 0 256s111.03 248 248 248 248-111.03 248-248S384.97 8 248 8zm0 432c-101.69 0-184-82.29-184-184 0-101.69 82.29-184 184-184 101.69 0 184 82.29 184 184 0 101.69-82.29 184-184 184zm0-312c-70.69 0-128 57.31-128 128s57.31 128 128 128 128-57.31 128-128-57.31-128-128-128zm0 192c-35.29 0-64-28.71-64-64s28.71-64 64-64 64 28.71 64 64-28.71 64-64 64z"/&gt;&lt;/svg&gt; Demonstration that roots selectively secrete Malic acid and effectively signal beneficial rhizobacteria establishes a regulatory role of root metabolites in recruitment of beneficial microbes

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:grey;" viewBox="0 0 448 512"&gt;&lt;path d="M437.2 403.5L320 215V64h8c13.3 0 24-10.7 24-24V24c0-13.3-10.7-24-24-24H120c-13.3 0-24 10.7-24 24v16c0 13.3 10.7 24 24 24h8v151L10.8 403.5C-18.5 450.6 15.3 512 70.9 512h306.2c55.7 0 89.4-61.5 60.1-108.5zM137.9 320l48.2-77.6c3.7-5.2 5.8-11.6 5.8-18.4V64h64v160c0 6.9 2.2 13.2 5.8 18.4l48.2 77.6h-172z"/&gt;&lt;/svg&gt; Aerial infection with *Pseudomonas syringae* strains Pst DC3000 **correlate with changes in root symbiont colonization** (*Bacillus subtilis* strain FB17) implicated root exudate involvement in the beneficial microbe recruitment
]
]
]

.pull-right[
&lt;br&gt;
&lt;img src="images/shape4.png" width="500px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Rudrappa, Czymmek, Paré, and Bais, 2008)]]
]

???

.n30[
- (NPS3121) nonpathogenic *Pseudomonas syringae* 
]

---

.pull-left[
**How plant can recruit beneficial microbes** 
.m0tp[
.h202[


- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 512 512"&gt;&lt;path d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zM262.655 90c-54.497 0-89.255 22.957-116.549 63.758-3.536 5.286-2.353 12.415 2.715 16.258l34.699 26.31c5.205 3.947 12.621 3.008 16.665-2.122 17.864-22.658 30.113-35.797 57.303-35.797 20.429 0 45.698 13.148 45.698 32.958 0 14.976-12.363 22.667-32.534 33.976C247.128 238.528 216 254.941 216 296v4c0 6.627 5.373 12 12 12h56c6.627 0 12-5.373 12-12v-1.333c0-28.462 83.186-29.647 83.186-106.667 0-58.002-60.165-102-116.531-102zM256 338c-25.365 0-46 20.635-46 46 0 25.364 20.635 46 46 46s46-20.636 46-46c0-25.365-20.635-46-46-46z"/&gt;&lt;/svg&gt; Can leaf infection trigger changes in the composition of root metabolites?

]
]
]

.pull-right[
&lt;br&gt;
&lt;img src="images/shape5.png" width="500px" style="display: block; margin: auto;" /&gt;

.font70[[Spurious correlation](https://www.tylervigen.com/spurious-correlations)]
]

---

.pull-left[
**How plant can recruit beneficial microbes** 
.m0tp[
.h202[


- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 512 512"&gt;&lt;path d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zM262.655 90c-54.497 0-89.255 22.957-116.549 63.758-3.536 5.286-2.353 12.415 2.715 16.258l34.699 26.31c5.205 3.947 12.621 3.008 16.665-2.122 17.864-22.658 30.113-35.797 57.303-35.797 20.429 0 45.698 13.148 45.698 32.958 0 14.976-12.363 22.667-32.534 33.976C247.128 238.528 216 254.941 216 296v4c0 6.627 5.373 12 12 12h56c6.627 0 12-5.373 12-12v-1.333c0-28.462 83.186-29.647 83.186-106.667 0-58.002-60.165-102-116.531-102zM256 338c-25.365 0-46 20.635-46 46 0 25.364 20.635 46 46 46s46-20.636 46-46c0-25.365-20.635-46-46-46z"/&gt;&lt;/svg&gt; Can leaf infection trigger changes in the composition of root metabolites?

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:grey;" viewBox="0 0 448 512"&gt;&lt;path d="M437.2 403.5L320 215V64h8c13.3 0 24-10.7 24-24V24c0-13.3-10.7-24-24-24H120c-13.3 0-24 10.7-24 24v16c0 13.3 10.7 24 24 24h8v151L10.8 403.5C-18.5 450.6 15.3 512 70.9 512h306.2c55.7 0 89.4-61.5 60.1-108.5zM137.9 320l48.2-77.6c3.7-5.2 5.8-11.6 5.8-18.4V64h64v160c0 6.9 2.2 13.2 5.8 18.4l48.2 77.6h-172z"/&gt;&lt;/svg&gt; Root secretions from plants subjected to different aerial bacterial infection treatments were collected and chemically analyzed by HPLC

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:grey;" viewBox="0 0 448 512"&gt;&lt;path d="M437.2 403.5L320 215V64h8c13.3 0 24-10.7 24-24V24c0-13.3-10.7-24-24-24H120c-13.3 0-24 10.7-24 24v16c0 13.3 10.7 24 24 24h8v151L10.8 403.5C-18.5 450.6 15.3 512 70.9 512h306.2c55.7 0 89.4-61.5 60.1-108.5zM137.9 320l48.2-77.6c3.7-5.2 5.8-11.6 5.8-18.4V64h64v160c0 6.9 2.2 13.2 5.8 18.4l48.2 77.6h-172z"/&gt;&lt;/svg&gt; The peak was characterized by liquid chromatography-mass spectrometry and determined to be **malic acid**

&lt;br&gt;&lt;br&gt;
.right[.font50[(Rudrappa Czymmek, et al., 2008)]]
]
]
]

.pull-right[
&lt;img src="images/shape6.png" width="280px" style="display: block; margin: auto;" /&gt;
]

---

.pull-left[
**How plant can recruit beneficial microbes** 
.m0tp[
.h202[


- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 512 512"&gt;&lt;path d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zM262.655 90c-54.497 0-89.255 22.957-116.549 63.758-3.536 5.286-2.353 12.415 2.715 16.258l34.699 26.31c5.205 3.947 12.621 3.008 16.665-2.122 17.864-22.658 30.113-35.797 57.303-35.797 20.429 0 45.698 13.148 45.698 32.958 0 14.976-12.363 22.667-32.534 33.976C247.128 238.528 216 254.941 216 296v4c0 6.627 5.373 12 12 12h56c6.627 0 12-5.373 12-12v-1.333c0-28.462 83.186-29.647 83.186-106.667 0-58.002-60.165-102-116.531-102zM256 338c-25.365 0-46 20.635-46 46 0 25.364 20.635 46 46 46s46-20.636 46-46c0-25.365-20.635-46-46-46z"/&gt;&lt;/svg&gt; Is the malic acid able to selectively recruit the symbiont colonization (*Bacillus subtilis* strain FB17)?

- *B. subtilis*  exhibits positive chemotaxis towards Malic Acid

- L-MA specifically chemoattracts FB17

- None of the tested bacteria shows any significant motility toward L-MA compared to *B. subtilis* 

&lt;br&gt;&lt;br&gt;
.right[.font50[(Rudrappa Czymmek, et al., 2008)]]
]
]
]

.pull-right[
&lt;img src="images/shape7.png" width="300px" style="display: block; margin: auto;" /&gt;
]

---

.pull-left[
**How plant can recruit beneficial microbes** 
.m0tp[
.h202[


- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 512 512"&gt;&lt;path d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zM262.655 90c-54.497 0-89.255 22.957-116.549 63.758-3.536 5.286-2.353 12.415 2.715 16.258l34.699 26.31c5.205 3.947 12.621 3.008 16.665-2.122 17.864-22.658 30.113-35.797 57.303-35.797 20.429 0 45.698 13.148 45.698 32.958 0 14.976-12.363 22.667-32.534 33.976C247.128 238.528 216 254.941 216 296v4c0 6.627 5.373 12 12 12h56c6.627 0 12-5.373 12-12v-1.333c0-28.462 83.186-29.647 83.186-106.667 0-58.002-60.165-102-116.531-102zM256 338c-25.365 0-46 20.635-46 46 0 25.364 20.635 46 46 46s46-20.636 46-46c0-25.365-20.635-46-46-46z"/&gt;&lt;/svg&gt; Does MA transporter mutant fails to recruit *B. subtilis* onto the root surface?

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:grey;" viewBox="0 0 448 512"&gt;&lt;path d="M437.2 403.5L320 215V64h8c13.3 0 24-10.7 24-24V24c0-13.3-10.7-24-24-24H120c-13.3 0-24 10.7-24 24v16c0 13.3 10.7 24 24 24h8v151L10.8 403.5C-18.5 450.6 15.3 512 70.9 512h306.2c55.7 0 89.4-61.5 60.1-108.5zM137.9 320l48.2-77.6c3.7-5.2 5.8-11.6 5.8-18.4V64h64v160c0 6.9 2.2 13.2 5.8 18.4l48.2 77.6h-172z"/&gt;&lt;/svg&gt; Arabidopsis knockout mutant *Atalmt1*  **deficient in root MA secretion** was assayed to confirm the role of MA secretions in recruiting FB17

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:grey;" viewBox="0 0 448 512"&gt;&lt;path d="M437.2 403.5L320 215V64h8c13.3 0 24-10.7 24-24V24c0-13.3-10.7-24-24-24H120c-13.3 0-24 10.7-24 24v16c0 13.3 10.7 24 24 24h8v151L10.8 403.5C-18.5 450.6 15.3 512 70.9 512h306.2c55.7 0 89.4-61.5 60.1-108.5zM137.9 320l48.2-77.6c3.7-5.2 5.8-11.6 5.8-18.4V64h64v160c0 6.9 2.2 13.2 5.8 18.4l48.2 77.6h-172z"/&gt;&lt;/svg&gt; *Atalmt1* x *Ler*-0 line **restored wild-type Col-0 phenotype** under Pst DC3000 infection

&lt;br&gt;&lt;br&gt;
.right[.font50[(Rudrappa Czymmek, et al., 2008)]]
]
]
]

.pull-right[
&lt;img src="images/shape8.png" width="600px" style="display: block; margin: auto;" /&gt;
]

---

.pull-left[
**Microbial interactions** 
.m0tp[
.h202[


- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 496 512"&gt;&lt;path d="M248 8C111.03 8 0 119.03 0 256s111.03 248 248 248 248-111.03 248-248S384.97 8 248 8zm0 432c-101.69 0-184-82.29-184-184 0-101.69 82.29-184 184-184 101.69 0 184 82.29 184 184 0 101.69-82.29 184-184 184zm0-312c-70.69 0-128 57.31-128 128s57.31 128 128 128 128-57.31 128-128-57.31-128-128-128zm0 192c-35.29 0-64-28.71-64-64s28.71-64 64-64 64 28.71 64 64-28.71 64-64 64z"/&gt;&lt;/svg&gt; Plants are able to influence the composition and activation of their rhizosphere microbiota but the **microorganisms have a role** in cooperative or competitive interactions with other members of the microbiome

- The genomes of many bacteria encode interspecific and intraspecific bacteria-killing mechanisms (antibiotics), so they can modulate the abundance, distribution and diversity of other microbes

- **Quorum sensing** is a well-established mechanism of bacterial cell-to-cell communication that involves the production and sensing of signalling molecules

]
]
]

.pull-right[
&lt;br&gt;
&lt;img src="images/shape9.png" width="580px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Berendsen Pieterse, et al., 2012)]]
]


---

.pull-left[
**Eco-evolutionary processes** 
.m0tp[
.h202[

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 496 512"&gt;&lt;path d="M248 8C111.03 8 0 119.03 0 256s111.03 248 248 248 248-111.03 248-248S384.97 8 248 8zm0 432c-101.69 0-184-82.29-184-184 0-101.69 82.29-184 184-184 101.69 0 184 82.29 184 184 0 101.69-82.29 184-184 184zm0-312c-70.69 0-128 57.31-128 128s57.31 128 128 128 128-57.31 128-128-57.31-128-128-128zm0 192c-35.29 0-64-28.71-64-64s28.71-64 64-64 64 28.71 64 64-28.71 64-64 64z"/&gt;&lt;/svg&gt; ecological and evolutionary processes are linked and can occur on the same time-scale.    
  - ecological change can lead to rapid evolutionary change
  - microevolutionary change can leave an ecological signature

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zm-248 50c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z"/&gt;&lt;/svg&gt; Eco-evolutionary processes can influence the assembly of plant-associated microbiomes:
  1. *dispersal*: movement of microorganisms between different niches
  1. *selection*: biotic and abiotic effects causing fitness differences
  1. *ecological drift* stochastic changes in population size that influence the abundance of a particular member
  1. *diversification*: processes that generate genetic variation

]
]
]

.pull-right[
&lt;br&gt;
&lt;img src="images/ecoevo1.png" width="280px" style="display: block; margin: auto;" /&gt;
]

---

.pull-left[
**Dynamics of the plant-associated microbiome** 
.m0tp[
.h202[

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 496 512"&gt;&lt;path d="M248 8C111.03 8 0 119.03 0 256s111.03 248 248 248 248-111.03 248-248S384.97 8 248 8zm0 432c-101.69 0-184-82.29-184-184 0-101.69 82.29-184 184-184 101.69 0 184 82.29 184 184 0 101.69-82.29 184-184 184zm0-312c-70.69 0-128 57.31-128 128s57.31 128 128 128 128-57.31 128-128-57.31-128-128-128zm0 192c-35.29 0-64-28.71-64-64s28.71-64 64-64 64 28.71 64 64-28.71 64-64 64z"/&gt;&lt;/svg&gt; The assembly of a plant-associated microbiome is a **successional, multistep process** that is determined by dispersal, species interactions, the environment and the host

- **Early colonizers** could be **transmitted vertically** &lt;svg style="height:0.8em;top:.04em;position:relative;fill:#2F4659;" viewBox="0 0 256 512"&gt;&lt;path d="M214.059 377.941H168V134.059h46.059c21.382 0 32.09-25.851 16.971-40.971L144.971 7.029c-9.373-9.373-24.568-9.373-33.941 0L24.971 93.088c-15.119 15.119-4.411 40.971 16.971 40.971H88v243.882H41.941c-21.382 0-32.09 25.851-16.971 40.971l86.059 86.059c9.373 9.373 24.568 9.373 33.941 0l86.059-86.059c15.12-15.119 4.412-40.971-16.97-40.971z"/&gt;&lt;/svg&gt; , via the parents, through seed transmission pathways

- Once **seeds germinate**, microbiome assembly is likely to be driven by **horizontal transfer** &lt;svg style="height:0.8em;top:.04em;position:relative;fill:#2F4659;" viewBox="0 0 512 512"&gt;&lt;path d="M377.941 169.941V216H134.059v-46.059c0-21.382-25.851-32.09-40.971-16.971L7.029 239.029c-9.373 9.373-9.373 24.568 0 33.941l86.059 86.059c15.119 15.119 40.971 4.411 40.971-16.971V296h243.882v46.059c0 21.382 25.851 32.09 40.971 16.971l86.059-86.059c9.373-9.373 9.373-24.568 0-33.941l-86.059-86.059c-15.119-15.12-40.971-4.412-40.971 16.97z"/&gt;&lt;/svg&gt;:
  - seed-borne microorganisms preferentially become associated with aboveground plant tissues
  - soil-derived microorganisms are mainly associated with the rhizosphere and roots

]
]
]

.pull-right[
&lt;br&gt;
&lt;img src="images/ecoevo2.png" width="480px" style="display: block; margin: auto;" /&gt;
]

---

.pull-left[
**Dynamics of the plant-associated microbiome** 
.m0tp[
.h202[

- Microbiomes is **highly dynamic** in the early vegetative phase &lt;svg style="height:0.8em;top:.04em;position:relative;fill:#25403B;" viewBox="0 0 512 512"&gt;&lt;path d="M64 96H0c0 123.7 100.3 224 224 224v144c0 8.8 7.2 16 16 16h32c8.8 0 16-7.2 16-16V320C288 196.3 187.7 96 64 96zm384-64c-84.2 0-157.4 46.5-195.7 115.2 27.7 30.2 48.2 66.9 59 107.6C424 243.1 512 147.9 512 32h-64z"/&gt;&lt;/svg&gt;, begins to **converge** throughout vegetative growth &lt;svg style="height:0.8em;top:.04em;position:relative;fill:#25403B;" viewBox="0 0 384 512"&gt;&lt;path d="M384 312.7c-55.1 136.7-187.1 54-187.1 54-40.5 81.8-107.4 134.4-184.6 134.7-16.1 0-16.6-24.4 0-24.4 64.4-.3 120.5-42.7 157.2-110.1-41.1 15.9-118.6 27.9-161.6-82.2 109-44.9 159.1 11.2 178.3 45.5 9.9-24.4 17-50.9 21.6-79.7 0 0-139.7 21.9-149.5-98.1 119.1-47.9 152.6 76.7 152.6 76.7 1.6-16.7 3.3-52.6 3.3-53.4 0 0-106.3-73.7-38.1-165.2 124.6 43 61.4 162.4 61.4 162.4.5 1.6.5 23.8 0 33.4 0 0 45.2-89 136.4-57.5-4.2 134-141.9 106.4-141.9 106.4-4.4 27.4-11.2 53.4-20 77.5 0 0 83-91.8 172-20z"/&gt;&lt;/svg&gt; and **stabilizes** during the reproductive phase &lt;svg style="height:0.8em;top:.04em;position:relative;fill:#F23D4C;" viewBox="0 0 512 512"&gt;&lt;path d="M352.57 128c-28.09 0-54.09 4.52-77.06 12.86l12.41-123.11C289 7.31 279.81-1.18 269.33.13 189.63 10.13 128 77.64 128 159.43c0 28.09 4.52 54.09 12.86 77.06L17.75 224.08C7.31 223-1.18 232.19.13 242.67c10 79.7 77.51 141.33 159.3 141.33 28.09 0 54.09-4.52 77.06-12.86l-12.41 123.11c-1.05 10.43 8.11 18.93 18.59 17.62 79.7-10 141.33-77.51 141.33-159.3 0-28.09-4.52-54.09-12.86-77.06l123.11 12.41c10.44 1.05 18.93-8.11 17.62-18.59-10-79.7-77.51-141.33-159.3-141.33zM256 288a32 32 0 1 1 32-32 32 32 0 0 1-32 32z"/&gt;&lt;/svg&gt; 

- A small fraction of microbial taxa belonging to the **core microbiota** are consistently maintained throughout plant development

- Specific traits (drought tolerance, disease resistance) that are conferred by associations with a particular microbial group can be **passed from mother plants to offspring** &lt;svg style="height:0.8em;top:.04em;position:relative;fill:#2F4659;" viewBox="0 0 256 512"&gt;&lt;path d="M214.059 377.941H168V134.059h46.059c21.382 0 32.09-25.851 16.971-40.971L144.971 7.029c-9.373-9.373-24.568-9.373-33.941 0L24.971 93.088c-15.119 15.119-4.411 40.971 16.971 40.971H88v243.882H41.941c-21.382 0-32.09 25.851-16.971 40.971l86.059 86.059c9.373 9.373 24.568 9.373 33.941 0l86.059-86.059c15.12-15.119 4.412-40.971-16.97-40.971z"/&gt;&lt;/svg&gt;
]
]
]

.pull-right[
&lt;br&gt;
&lt;img src="images/ecoevo2.png" width="480px" style="display: block; margin: auto;" /&gt;
]

---

.center[**Evolutionary Relationship of PGPB and Pathogenic Bacteria**]
&lt;img src="images/ecoevo3.png" width="480px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Berendsen Pieterse, et al., 2012)]]

---

layout: true
# Can we exploit the microbiota?

---

class: inverse, middle, center 

----

---

.center[**Inoculation of microbial consortium**]

.pull-left[
.m0tbp[
.h24[

- Microorganisms have long been applied as inoculants for biocontrol or biostimulation

.center[BUT]

- their field efficacy varies with the climate, soil type and other environmental factors
]
]
]

.pull-right[
&lt;br&gt;
&lt;img src="images/exploit1.png" width="600px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Quiza, St-Arnaud, and Yergeau, 2015)]]
]   

---

.center[**Outcomes of inoculation with a microbial consortium**]

.pull-left[
.m0tbp[
.h20[

- **Outcomes of inoculating** with a five-member consortium in the presence of a diverse environmental microbial pool 

- Three cultivars are depicted growing in **three different regimes**: a normal year, a drought year, and low-input management

- **Genotype effects**:
  - In a normal year, Cultivar A is colonized by all five members of the inoculant consortium
  - Cultivar B is colonized only by yellow spheres
  - Cultivar C is colonized only by teal stars

- **Environment/ management effects**:
  - The drought year shows higher colonization by blue ovals 
  - Under low-input management all cultivars show increased colonization by the green spiky ovals

- **Temporal dynamics** of two communities after a disturbance event:
  - The more resilient community recovers to its initial state after perturbation
  - The less resilient community does not recovers to its initial state after perturbation
]
]
]

.pull-right[
&lt;img src="images/exploit3.png" width="490px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Busby, Soman, Wagner, Friesen, Kremer, Bennett, Morsy, Eisen, Leach, and Dangl, 2017)]]
]
---

.center[**Microbiome engineering using synthetic microbial communities**]

.pull-left[
.m0tp[
.h204[

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 576 512"&gt;&lt;path d="M569.517 440.013C587.975 472.007 564.806 512 527.94 512H48.054c-36.937 0-59.999-40.055-41.577-71.987L246.423 23.985c18.467-32.009 64.72-31.951 83.154 0l239.94 416.028zM288 354c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z"/&gt;&lt;/svg&gt; There is NOT  a "one size fits all" microorganism community"

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:#F2B544;" viewBox="0 0 352 512"&gt;&lt;path d="M96.06 454.35c.01 6.29 1.87 12.45 5.36 17.69l17.09 25.69a31.99 31.99 0 0 0 26.64 14.28h61.71a31.99 31.99 0 0 0 26.64-14.28l17.09-25.69a31.989 31.989 0 0 0 5.36-17.69l.04-38.35H96.01l.05 38.35zM0 176c0 44.37 16.45 84.85 43.56 115.78 16.52 18.85 42.36 58.23 52.21 91.45.04.26.07.52.11.78h160.24c.04-.26.07-.51.11-.78 9.85-33.22 35.69-72.6 52.21-91.45C335.55 260.85 352 220.37 352 176 352 78.61 272.91-.3 175.45 0 73.44.31 0 82.97 0 176zm176-80c-44.11 0-80 35.89-80 80 0 8.84-7.16 16-16 16s-16-7.16-16-16c0-61.76 50.24-112 112-112 8.84 0 16 7.16 16 16s-7.16 16-16 16z"/&gt;&lt;/svg&gt; Synthetic microbial communities (synComs) of varied complexity have been constructed using bottom-up combinations 

- To build a synCom selection of core functions are more important than taxonomical selection 

- synComs with different complexities can be designed through predictive modelling that evaluates trait redundancy, dominance, modularity, interactions and assembly
]
]
]

.pull-rigth[
&lt;br&gt;
&lt;img src="images/exploit2.png" width="500px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Song, Ding, Jia, Ma, and Yuan, 2014)]]
]

---
class: clear, middle, center

# .black[Companies Examples]

----

---

.center[
&lt;img src="images/bioc1.png" width="500px" style="display: block; margin: auto;" /&gt;
&lt;br&gt;
BioConsortia Inc. is devoted to developing solutions through **enhanced crop productivity** and
is aware that **microbial products have a major role** to play in future mainstream agricultural crops

&lt;br&gt;
.marco[.tada[https://bioconsortia.com/]]
]

---


background-image: url(images/bioc2.png)
background-size: cover

.content-box-grey2[
.p20[
**Founded in New Zeland in 1994**

The company specializes in the discovery and development of natural microbial products
]
]
&lt;br&gt;
.content-box-grey2[
.p20[
**Breakthrough in Advanced Microbial Selection (AMS) process in 2009**

Focused on perfecting the innovative R&amp;D platform for AMS
]
]
&lt;br&gt;
.content-box-grey2[
.p20[**In 2014, the decision was made to globalize**

BioConsortia raised $15 million from Khosla Ventures and Otter Capital, and establishing the headquarters and labs in Davis, CA
]
]
&lt;br&gt;
.content-box-grey2[
.p20[**Strong R&amp;D Experise**

30 highly experienced experts from industry and academia, including 12 PhDs
]
]
&lt;br&gt;
.content-box-grey2[
.p20[**Highest level of competency in ultra-high-throughput biological screening**

They now hold one of the world’s largest collections of pre-screened and characterized microorganisms, comprising over 45,000 microbes, including 9,000 endophytes
]]

---

&lt;img src="images/bioc3.png" width="850px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/bioc4.png" width="850px" style="display: block; margin: auto;" /&gt;

???

.n30[
- Genetics x Envirinment x Microbiome = Phenotype
]

---

&lt;img src="images/bioc5.png" width="850px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/bioc6.png" width="850px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/bioc8.png" width="850px" style="display: block; margin: auto;" /&gt;
---

&lt;img src="images/bioc7.png" width="850px" style="display: block; margin: auto;" /&gt;

---

.center[
&lt;img src="images/ma1.png" width="400px" style="display: block; margin: auto;" /&gt;

&lt;br&gt;
At Mycorrhizal Applications, we harness the power and wisdom of natural systems to promote living soils and increase quality, productivity, and health in all industries involving soils, plants, and people


.marco[.tada[https://mycorrhizae.com/]]
]

---

background-image: url(images/ma2.png)
background-size: cover

.content-box-grey3[
.p201[
**Founded in  USA**&lt;br&gt;
Mycorrhizal Applications was established in the Dr. Amaranthuses’ garage,
with a focus on producing ectomycorrhizal slurries for use in forest restoration
]
]
&lt;br&gt;
.content-box-grey3[
.p201[
**Large-scale production in 2002**&lt;br&gt;
The company moved to its first large-scale production and office building 
]
]
&lt;br&gt;
.content-box-grey3[
.p201[
**Entered into the agriculture industry in 2004**&lt;br&gt;
Its MycoApply® Micronized Endo product being utilized as a seed coat, kick starting the next growth phase for the company
]
]
&lt;br&gt;
.content-box-grey3[
.p201[
**Products for international markets in 2016**&lt;br&gt;
New highly-concentrated MycoApply® products for the Agriculture markets and non-Agriculture sectors like Horticulture, Nurseries and Turf Management
]
]
&lt;br&gt;
.content-box-grey3[
.p201[
**Development and launch of new formulations in 2020**&lt;br&gt;
New advanced carrier technology for agriculture and for horticulture and turf
]
]
&lt;br&gt;

---

.pull-left[
&lt;img src="images/ma3.png" width="450px" style="display: block; margin: auto;" /&gt;
]

.pull-right[
&lt;img src="images/ma4.png" width="450px" style="display: block; margin: auto;" /&gt;
]

---

.center[
&lt;img src="images/bm1.png" width="900px" style="display: block; margin: auto;" /&gt;
&lt;br&gt;
.marco[.tada[https://biomemakers.com]]
]

???

.h202[
- Our technological platform helps farmers and agronomists to discover the whole microbial community impacting your crop. From there, we can help to determine how to improve production and maintain soil sustainability for many future harvests to come.
]

---

background-image: url(images/bm2.jpg)
background-size: cover

&lt;br&gt;
.content-box-grey2[
.p20[
**2015**&lt;br&gt;
Biome Markers is born in San Francisco, taking action to recover soil health through technology
]
]
&lt;br&gt;
.content-box-grey2[
.p20[
**2016**&lt;br&gt;
Tails and market validation for advance soil analysis in vineyard
]
]
&lt;br&gt;
.content-box-grey2[
.p20[
**2017**&lt;br&gt;
Lauched the operational portal with all the features to be an easy and friendly platform
]
]
&lt;br&gt;
.content-box-grey2[
.p20[
**Community recognition**&lt;br&gt;
Many awards for providing the best technology for any crop and soil analysis
]
]
&lt;br&gt;
.content-box-grey2[
.p20[
**2018**&lt;br&gt;
Launch BeCrop and Gheom platforms (even on mobile) 
]
]
&lt;br&gt;



---

&lt;img src="images/bm14.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/bm4.png" width="1200px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/bm5.png" width="1200px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/bm6.png" width="1200px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/bm7.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;br&gt;
&lt;img src="images/bm15.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/bm8.png" width="1200px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/bm9.png" width="1200px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/bm10.png" width="1200px" style="display: block; margin: auto;" /&gt;

???

.h202[
- The Agricultural Input Model (AgInput) was developed to predict a crop’s irrigation
water and nutrients needs and to estimate the corresponding pesticide application to the
crop under average local meteorological and soil conditions
]
---

&lt;img src="images/bm11.png" width="1200px" style="display: block; margin: auto;" /&gt;


---

layout: true

# Questions about the lesson


---


.m0tbp[
.center[.font50px[&lt;svg style="height:0.8em;top:.04em;position:relative;fill:black;" viewBox="0 0 512 512"&gt;&lt;path d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zM262.655 90c-54.497 0-89.255 22.957-116.549 63.758-3.536 5.286-2.353 12.415 2.715 16.258l34.699 26.31c5.205 3.947 12.621 3.008 16.665-2.122 17.864-22.658 30.113-35.797 57.303-35.797 20.429 0 45.698 13.148 45.698 32.958 0 14.976-12.363 22.667-32.534 33.976C247.128 238.528 216 254.941 216 296v4c0 6.627 5.373 12 12 12h56c6.627 0 12-5.373 12-12v-1.333c0-28.462 83.186-29.647 83.186-106.667 0-58.002-60.165-102-116.531-102zM256 338c-25.365 0-46 20.635-46 46 0 25.364 20.635 46 46 46s46-20.636 46-46c0-25.365-20.635-46-46-46z"/&gt;&lt;/svg&gt;]]
]

.center[.font90[[[&lt;svg style="height:0.8em;top:.04em;position:relative;fill:steelblue;" viewBox="0 0 512 512"&gt;&lt;path d="M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7 22.4 17.4 52.1 39.5 154.1 113.6 21.1 15.4 56.7 47.8 92.2 47.6 35.7.3 72-32.8 92.3-47.6 102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4 132.7-96.3 142.8-104.7 173.4-128.7 5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9 30.6 23.9 40.7 32.4 173.4 128.7 16.8 12.2 50.2 41.8 73.4 41.4z"/&gt;&lt;/svg&gt;](mailto:chiapello.m@gmail.com) chiapello.m@gmail.com - &lt;svg style="height:0.8em;top:.04em;position:relative;fill:steelblue;" viewBox="0 0 448 512"&gt;&lt;path d="M424.7 299.8c2.9-14 4.7-28.9 4.7-43.8 0-113.5-91.9-205.3-205.3-205.3-14.9 0-29.7 1.7-43.8 4.7C161.3 40.7 137.7 32 112 32 50.2 32 0 82.2 0 144c0 25.7 8.7 49.3 23.3 68.2-2.9 14-4.7 28.9-4.7 43.8 0 113.5 91.9 205.3 205.3 205.3 14.9 0 29.7-1.7 43.8-4.7 19 14.6 42.6 23.3 68.2 23.3 61.8 0 112-50.2 112-112 .1-25.6-8.6-49.2-23.2-68.1zm-194.6 91.5c-65.6 0-120.5-29.2-120.5-65 0-16 9-30.6 29.5-30.6 31.2 0 34.1 44.9 88.1 44.9 25.7 0 42.3-11.4 42.3-26.3 0-18.7-16-21.6-42-28-62.5-15.4-117.8-22-117.8-87.2 0-59.2 58.6-81.1 109.1-81.1 55.1 0 110.8 21.9 110.8 55.4 0 16.9-11.4 31.8-30.3 31.8-28.3 0-29.2-33.5-75-33.5-25.7 0-42 7-42 22.5 0 19.8 20.8 21.8 69.1 33 41.4 9.3 90.7 26.8 90.7 77.6 0 59.1-57.1 86.5-112 86.5z"/&gt;&lt;/svg&gt; marpello - &lt;svg style="height:0.8em;top:.04em;position:relative;fill:steelblue;" viewBox="0 0 512 512"&gt;&lt;path d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"/&gt;&lt;/svg&gt;](https://twitter.com/marpello1980) @marpello1980 ]]





    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="libs/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"ratio": "16:9",
"highlightStyle": "github",
"highlightLines": true
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
