<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>Microbiota</title>
    <meta charset="utf-8" />
    <meta name="author" content="Marco Chiapello" />
    <script src="libs/header-attrs/header-attrs.js"></script>
    <link rel="stylesheet" href="scrollable.css" type="text/css" />
    <link rel="stylesheet" href="mtheme_max.css" type="text/css" />
    <link rel="stylesheet" href="fonts_mtheme_max.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">




class: title-slide



# .font170[MICROBIOMA E MICROBIOTA]


&lt;br&gt;&lt;br&gt;&lt;br&gt;&lt;br&gt;
.marco[
Marco Chiapello
&lt;br&gt;
2020-10-06
]


.marco[
.font90[[&lt;svg style="height:0.8em;top:.04em;position:relative;fill:steelblue;" viewBox="0 0 512 512"&gt;&lt;path d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"/&gt;&lt;/svg&gt;](https://twitter.com/marpello1980) @marpello1980 - [&lt;svg style="height:0.8em;top:.04em;position:relative;fill:steelblue;" viewBox="0 0 512 512"&gt;&lt;path d="M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7 22.4 17.4 52.1 39.5 154.1 113.6 21.1 15.4 56.7 47.8 92.2 47.6 35.7.3 72-32.8 92.3-47.6 102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4 132.7-96.3 142.8-104.7 173.4-128.7 5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9 30.6 23.9 40.7 32.4 173.4 128.7 16.8 12.2 50.2 41.8 73.4 41.4z"/&gt;&lt;/svg&gt;](mailto:chiapello.m@gmail.com) chiapello.m@gmail.com - &lt;svg style="height:0.8em;top:.04em;position:relative;fill:steelblue;" viewBox="0 0 448 512"&gt;&lt;path d="M424.7 299.8c2.9-14 4.7-28.9 4.7-43.8 0-113.5-91.9-205.3-205.3-205.3-14.9 0-29.7 1.7-43.8 4.7C161.3 40.7 137.7 32 112 32 50.2 32 0 82.2 0 144c0 25.7 8.7 49.3 23.3 68.2-2.9 14-4.7 28.9-4.7 43.8 0 113.5 91.9 205.3 205.3 205.3 14.9 0 29.7-1.7 43.8-4.7 19 14.6 42.6 23.3 68.2 23.3 61.8 0 112-50.2 112-112 .1-25.6-8.6-49.2-23.2-68.1zm-194.6 91.5c-65.6 0-120.5-29.2-120.5-65 0-16 9-30.6 29.5-30.6 31.2 0 34.1 44.9 88.1 44.9 25.7 0 42.3-11.4 42.3-26.3 0-18.7-16-21.6-42-28-62.5-15.4-117.8-22-117.8-87.2 0-59.2 58.6-81.1 109.1-81.1 55.1 0 110.8 21.9 110.8 55.4 0 16.9-11.4 31.8-30.3 31.8-28.3 0-29.2-33.5-75-33.5-25.7 0-42 7-42 22.5 0 19.8 20.8 21.8 69.1 33 41.4 9.3 90.7 26.8 90.7 77.6 0 59.1-57.1 86.5-112 86.5z"/&gt;&lt;/svg&gt; marpello]
]

???

.n30[

- Sarò uno dei vostri docenti per il corso "Interazioni tra piante, microrganismi e ambiente"
]
---
layout: true
class: clear

---

.font180[.bold[Agenda]]


.h35[
- .font140[Introduzione]

  - .font120[Il microbiota umano]
  
- .font140[Il microbiota delle piante]

  - .font120[Da chi è composto]
  
  - .font120[Che funzioni ha]
   
  - .font120[Come prende forma]
]

???

- .n[**Intro**: spiegazione dei concetti generali e dei termini]
---


.pull-left[.center[.font180[.bold[microBIOTA]]]
&lt;br&gt;
&lt;img src="images/microbiota.jpg" width="400px" style="display: block; margin: auto;" /&gt;

&lt;br&gt;
&gt; .font80[si riferisce a una popolazione di microrganismi che colonizza un determinato luogo]

]

.pull-right[.center[.font180[.bold[microBIOMA]]]
&lt;br&gt;
&lt;img src="images/microbioma.jpg" width="400px" style="display: block; margin: auto;" /&gt;

&gt; .font80[indica la totalità del patrimonio genetico posseduto dal microbiota, cioè i geni che quest’ultimo è in grado di esprimere]

]

???

- .n[si riferisce a una popolazione di microrganismi che colonizza un determinato luogo]

- .n[batteri, funghi, protozoi, elminti, neamtodi, virus, ]
 
- .n[indica la totalità del patrimonio genetico posseduto dal microbiota, cioè i geni che quest’ultimo è in grado di esprimere]

- .n[la differenza che passa tra i due termini è la stessa che esiste tra popolazione umana e genoma umano]


---


.pull-left[.center[.font180[.bold[microBIOTA]]]]

.pull-right[.center[.font180[.bold[microBIOMA]]]]

&lt;br&gt;
&lt;img src="images/microPublications.gif" width="550px" style="display: block; margin: auto;" /&gt;

---
&lt;br&gt;&lt;br&gt;&lt;br&gt;
.center[.font600[.fontW7[HMP]]]
.center[.font140[Human Microbiome Project]]

---

Il **progetto microbioma umano** è una iniziativa dei National Institutes of Health statunitensi con il fine di **identificare e caratterizzare i microrganismi ed il loro rapporto con lo stato di salute e di malattia dell'uomo**. 

&lt;img src="images/microNubers.jpg" width="700px" style="display: block; margin: auto;" /&gt;

???


- .n[**RAPPORTO** cellule microbiche e cellule umane]

- .n[Il numero totale di cellule microbiche presenti in un organismo umano può superare di dieci volte il numero di cellule dell'organismo stesso]

- .n[i geni di origine microbica possono superare di cento volte il numero di geni presenti nel genoma umano (composto da circa 20.000 geni).]

---

&lt;br&gt;&lt;br&gt;&lt;br&gt;
.content-box-grey[.font180[.fontW7[L'essere umano va concepito come composto da cellule umane e microbiche]]]

???

- .n[Visto l'alto numero di cellule microbiche e della quantità del loro materiale genetico]

- .n[L'essere umano va concepito come composto da cellule umane e microbiche]

- .n[MA **quali** sono i micro-organismi]


---
.center[Temporal human microbiota]
&lt;img src="images/humanEvo.jpg" width="780px" style="display: block; margin: auto;" /&gt;

???

- .n[La figura mostra i phyla chiave della composizione del microbiota umano in diversi stadi della vita]

---

.center[Spatial human microbiota]
&lt;img src="images/humanDiversity.png" width="750px" height="580px" style="display: block; margin: auto;" /&gt;

???

- .n[Vediamo ora **dove** sono distribuiti i batteri]

- .n[Partiamo dalla parte più interna della figura, dall'albero filigenetico]



---
layout: true
class: clear
.center[.font120[Human microbiota functions] .font80[(Kho and Lal, 2018)]]

---
&lt;img src="images/KhoPaperT3-1.png" width="800px" height="520px" style="display: block; margin: auto;" /&gt;

---
&lt;br&gt;&lt;br&gt;
&lt;img src="images/KhoPaperT3-2.png" width="900px" height="400px" style="display: block; margin: auto;" /&gt;
---
  
&lt;img src="images/KhoPaperT3-3.png" width="900px" height="530px" style="display: block; margin: auto;" /&gt;

---
layout: false
class: inverse, middle, center

# Il microbiota delle piante

----


---
layout: true 
class: clear

---


&lt;img src="images/PlantmicroPublications.gif" width="630px" style="display: block; margin: auto;" /&gt;

---
layout: true

# Composizione del microbiota

---
class: inverse, middle, center

----


---

## Glossario


- .font80[**Rhizosphere**: The region of soil in the vicinity of plant roots that is influenced by plant-derived nutrients and oxygen availability; it is not a region of definable size or shape, but instead consists of a gradient in chemical, biological and physical properties that change both radially and longitudinally along the root.]

--

- .font80[**Phyllosphere**: All the aboveground organs of plants, including the leaf, flower, stem and fruit.]

--

- .font80[**Endophytes**: The microorganisms residing within plant tissues (the endosphere), such as leaves, roots or stems.]

--

- .font80[**Bulk soil**:  is soil outside the rhizosphere. Bulk soil is not penetrated by plant roots]

---

layout: true
# Composizione del microbiota



---

.center[
.h20[
**Spatial plant microbiota**    
.font50[(Singh, Raina, Kumar, Singh, and Prasad, 2019)]
]

&lt;img src="images/PlantMicrobiomeComposition.png" width="470px" height="470px" style="display: block; margin: auto;" /&gt;

]

---

.center[
.h20[
**Temporal plant microbiota**    
.font50[(Sapkota, Jørgensen, and Nicolaisen, 2017)]
]

&lt;img src="images/plantMicrobiotaTemporal.png" width="530px" height="470px" style="display: block; margin: auto;" /&gt;

]

???

.n30[

- La composzione del microbiota vegetale è, come quella umana, dipendente dalla fase evolutiva della pianta e dal comparto che si sceglie di studiare.

- La figura sulla sinistra dello schermo cerca di schematizzare i recenti studi fatti sul microbiota delle piante

]

---


**Microbiota reservoir**    

.pull-left[
- .n[Although the assemblies of root-associated bacteria and fungi differ substantially from the above-ground communities, both represent a **subset of the microbiota derived from soil communities** and enriched in different plant-associated niches]

- .n[This suggests that **soil functions as a common reservoir** for both belowground and aboveground plant microbiota]
]

.pull-rigth[

.h20[
&lt;img src="images/reservoir.png" width="450px" height="450px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Agnello, Morelli, and Del Panno, 2020)]]
]
]

???

.n30[
- Un'importante considerazione da fare è che sebbene le comunità batteriche e fungine che sono associate alla piante siano molto diverse tra la parte ipogea e la parte epigea della pianta, entrambe rappresentano un sottogruppo del microbiota derivato dalle comunità del suolo

- Possiamo quindi dire che il suolo funziona come riserva comune per il microbiota della pianta epigeo e ipogeo
]

---

.pull-left[

&lt;img src="images/bacterialDiv.png" width="350px" style="display: block; margin: auto;" /&gt;

&lt;img src="images/fungalDiv.png" width="350px" style="display: block; margin: auto;" /&gt;
]

.pull-right[

- .n[There are clear differences among the microbial communities in different plant compartments, which indicates that the **plant compartment is a major selective force** that shapes the composition of plant-associated microbiota]

]

???

- .n[**Shannon diversity index**: combina ricchezza e diversità]

- .n[Misura sia il numero delle specie presenti e la differenza tra l'abbondanza delle diverse specie.]

- .n[Un valore grande significa che sono presenti molte specie con un'abbondanza simile. Il range del valore varia da 1 (una specie dominante) fino al numero totale di tutte le species (nel caso che tutte le specie abbiano la stessa abbondanza)]

---

.pull-left[

**CORE MICROBIOTA**

.h202[
.m0ul[

- The plant core microbiota consists of members of the microbial community that are **persistent and are ubiquitous in almost all the communities** associated with a particular host

- The core microbiota contains key microbial taxa that carry **genes with functions that are essential** for host fitness

- Many members of the core microbiota of different plant species are **common at the genus level**

- The identification of a co-occurring core of plant-associated microorganisms also provides a useful starting point for studies of how to build **SynComs** to manipulate plant–microbiome interactions for increased **growth and productivity**

]
]
]

.pull-right[
&lt;br&gt;
&lt;img src="images/core1.png" width="400px" style="display: block; margin: auto;" /&gt;
]

???
    
.h25[
    - .n[Il **core microbiota** è composto da alcuni membri del microbiota che sono presenti in tutte, o quasi, le comunità associate ad una determinato ospite]
    
    - .n[E i suoi membri esprimono geni le cui funzioni sono di primaria importanza per la salute del loro ospite]
    
    - .n[Seppur diverse piante abbiamo microbiota diversi, un certo numero di membri è lo stesso]
    
    - .n[L'identificazione di microorganismi che fanno parte di diversi **core microbiota** è di grande importanza per la produzione delle comunità sintetiche, da usare in agricoltura per aumentare la crescita e la produttività delle piante]
]
---

.pull-left[
&lt;br&gt;
&lt;img src="images/core2.png" width="400px" style="display: block; margin: auto;" /&gt;

]



.pull-right[
**HUB MICROBIOTA**
.h202[
- Within the core microbiota, a few members, such as ‘hub microorganisms’, can **influence the community structure** through strong biotic interactions with the host or with other microbial species, rather than simply by their own high abundance

- These hub species may represent **keystone species** that can exert strong direct and indirect effects on microbiome assembly and that **function as mediators** between the plant and its associated microbiome.

- For example, *Albugo laibachii* and *Dioszegia spp.* have been identified as highly interactive hubs in the phyllosphere of *A. thaliana* [Agler, et al. 2016]

]]

???

.h35[

- .n[All'interno del core microbiota, alcuni membri possono influenzare la struttura dell'intera comunità, non per la loro abbondanza, ma per le loro interazione dirette od indirette con la pianta o gli altri microorganismi. Vengono definiti: **hub microorganism**]

- .n[**hub** letteralmente significa: centro, perno, fulcro]

- .n[Esempio di hubs identificati nella fillosfera di arabidopsis. Albugo è un oomicete obligato mentre Dioszegia è un basidiomicete]


]


---
layout: true

# Plant Holobiont

---

class: inverse, middle, center

----

---

.m10bp[
&lt;img src="images/holobiont1.png" width="800px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Berg, Köberl, Rybakova, Müller, Grosch, and Smalla, 2017)]]
]

.m0p[
.h20[
.font80[Plants and their associated microorganisms form a holobiont and have to be considered as co-evolved species assemblages consisting of bacterial, archaeal and diverse eukaryotic species]
]
]

---

.pull-left[
### Holobiont functions
.m0p[
&lt;img src="images/holobiont2.png" width="500px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Theis, Dheilly, Klassen, Brucker, Baines, Bosch, Cryan, Gilbert, Goodnight, Lloyd, Sapp, Vandenkoornhuyse, Zilber-Rosenberg, Rosenberg, and Bordenstein, 2016)]]
]
]

.pull-right[
.m0p[
.h26[
- Microbial diversity and balance is a key for healthy plants

- Microbial species have the ability to contribute multiple aspects to the system    

    - seed germination and growth    
    - nutrient supply    
    - resistance against biotic stress factors    
    - resistance against abiotic factors    
    - physiology and production of bioactive metabolites

- The presence of microbes in plants became initially noticeable when diseases appeared. However, the vast majority of microbes in plants are not causal agents of damage in plants
]
]
]

---

.pull-left[
### Holobiont co-evolution
.m0p[
&lt;img src="images/holobiont3.png" width="600px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Hassani, Durán, and Hacquard, 2018)]]


]
]

.pull-right[
.m0p[
.h26[
- Holobiont also contributes to genetic variation and to the origin and evolution of species

- Evolution proceeds both via cooperation and competition, working in parallel
  
- Acquisition of microbes and microbial genes is a powerful mechanism for driving the evolution of complexity
  - Horizontal gene transfer (HGT) refers to the transfer of genes between organisms in a
    manner other than the vertical transmission of genes from parent
    to offspring

]

]
]


---
layout: true

# How do you study the microbiota composition?



---
class: inverse, middle, center

----

---

.center[**There are two major approaches to assess the microbiota composition**]

.h20[
&lt;img src="images/microbiotaCompositionTecniques.png" width="520px" style="display: block; margin: auto;" /&gt;

.right[.font60[(Lebeis, 2014)]]
]

---

.pull-left[
&lt;img src="images/microbiotaCompositionTecniques.png" width="520px" style="display: block; margin: auto;" /&gt;
]

.pull-right[
**Culture dependent**

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; This approach isolates individual microbes 

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm121.6 313.1c4.7 4.7 4.7 12.3 0 17L338 377.6c-4.7 4.7-12.3 4.7-17 0L256 312l-65.1 65.6c-4.7 4.7-12.3 4.7-17 0L134.4 338c-4.7-4.7-4.7-12.3 0-17l65.6-65-65.6-65.1c-4.7-4.7-4.7-12.3 0-17l39.6-39.6c4.7-4.7 12.3-4.7 17 0l65 65.7 65.1-65.6c4.7-4.7 12.3-4.7 17 0l39.6 39.6c4.7 4.7 4.7 12.3 0 17L312 256l65.6 65.1z"/&gt;&lt;/svg&gt; The community diversity estimate is limited

]

---

.pull-left[
**Culture independent**

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt;  The ability to identify and quantify community members, even down to extremely rare taxa

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt;  Insights beyond the information provided by individual microbes

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm121.6 313.1c4.7 4.7 4.7 12.3 0 17L338 377.6c-4.7 4.7-12.3 4.7-17 0L256 312l-65.1 65.6c-4.7 4.7-12.3 4.7-17 0L134.4 338c-4.7-4.7-4.7-12.3 0-17l65.6-65-65.6-65.1c-4.7-4.7-4.7-12.3 0-17l39.6-39.6c4.7-4.7 12.3-4.7 17 0l65 65.7 65.1-65.6c4.7-4.7 12.3-4.7 17 0l39.6 39.6c4.7 4.7 4.7 12.3 0 17L312 256l65.6 65.1z"/&gt;&lt;/svg&gt; This approach does not isolate individual microbes 

]


.pull-right[
&lt;img src="images/microbiotaCompositionTecniques.png" width="520px" style="display: block; margin: auto;" /&gt;
]

---

.center[.font150[Microbial DNA metabarcoding]]

&gt; **DNA metabarcoding** is a method of **DNA barcoding** that uses universal genetic markers to identify DNA of a mixture of organisms

--

&lt;br&gt;
.content-box-grey[What is it the &lt;br&gt; **DNA barcoding**?]

---

.center[.font150[Microbial DNA metabarcoding]]

- **DNA barcoding** is a method of species identification using a short section of DNA from a specific gene 

- An individual sequence can be used to uniquely identify an organism by comparison with a reference library of classified DNA sequences


&lt;img src="images/DNAbarcoding.png" width="520px" style="display: block; margin: auto;" /&gt;

---

.center[.font150[Microbial DNA metabarcoding]]

.pull-left[
**Marker selection**

.n30[
- Markers used for DNA barcoding are called barcodes and their selection is crucial

- Ideally, one gene sequence would be used for developing universal PCR primers for all taxonomic group
  
- However, no such gene region has been found yet!

- Different gene regions are used to identify the different organismal groups using barcoding
]
]

.pull-right[

&lt;img src="images/DNAbarcoding2.png" width="200px" style="display: block; margin: auto;" /&gt;

.right[.font50[(Mosa et al., 2019)]]
]

---

.center[.font150[Microbial DNA metabarcoding]]

**Marker selection**

| Organism | Region |
|----------|--------|
| animals  | cytochrome c oxidase I|
| fungi    |  internal transcribed spacer (ITS) rRNA |
| plants   | RuBisCO |
| prokaryotes | 16S rRNA |
| microbial eukaryotes | 18S rRNA |


.font80[When barcoding is used to identify organisms from a sample containing DNA from more than one organism, the term DNA metabarcoding is used]


---

.center[.font150[Microbial DNA metabarcoding]]

.pull-left[
.m0p[
**Bioinformatic analysis**
]
.n30[
- After sequencing, raw data must be processed (**quality and cleaning step**)

- Samples are clustered in operational taxonomic unit based on identity threshold (**OTU formation**)

- Using "the Basic Local Alignment Search Tool (BLAST)" regions of similarity between sequences and reference databases are identified (**annotation step**)    

- Microbiota composition definition (**richness and diversity step**)

]
]

.pull-right[

.h20[
.m0p[
.right[.font50[(Jesse et al., 2018)]]     
]
&lt;img src="images/OTU.png" width="220px" style="display: block; margin: auto;" /&gt;


]]



???

.n30[
- These databases contain the DNA barcodes assigned to previously identified taxa

- An operational taxonomic unit (OTU) is an operational definition used to classify groups of closely related individuals
]

---
layout: true

# What can the microbiota potentially do?

---

class: inverse, middle, center

----

---

.center[.font120[Metagenomics]]

&lt;img src="images/metagenomics1.png" width="720px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Ye, Siddle, Park, and Sabeti, 2019)]]
.h25[
- **Metagenomics** is the study of genetic material recovered directly from environmental samples

- The collection of sequenced genes from the environment could be analyzed as a **single genome**
  
- Metagenomics **does not require the isolation** and lab cultivation of individual species

- Due to its random nature, metagenomics, allows that many **under-represented organisms will be detected**
]

---

.center[Metagenomics vs Barcoding]
.pull-left[
.m0p[
&lt;img src="images/compareBarcodingMetagenome.png" width="450px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Gill, Baldock, Brown, Cresswell, Dicks, Fountain, Garratt, Gough, Heard, Holland, Ollerton, Stone, Tang, Vanbergen, Vogler, Woodward, Arce, Boatman, Brand-Hardy, and Potts, 2016)]]
]]


 .pull-right[
.h21[

- Metagenomics does not need the marker selection step

- Metagenomics sequences all the genes present in the sample from each organism

- Metagenomics can help to reconstruct large fragments or even complete genomes from organisms in a community

- Metagenomics allows the characterization of a large number of coding and non-coding sequences that can be used to decipher the microbial diversity or to understand its metabolic potential
]
]

---



&lt;img src="images/CompareHTS.png" width="920px" style="display: block; margin: auto;" /&gt;

---

.center[
.middle[
&lt;iframe width="840" height="473" src="https://www.youtube.com/embed/fCd6B5HRaZ8" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen&gt;&lt;/iframe&gt;
]
]
---

.center[.font120[Genomes assembly]]

.m0p[
&lt;img src="images/Align.png" width="620px" style="display: block; margin: auto;" /&gt;
]
---
.font120[**Functional Metagenomics Analysis**]

.pull-left[
.m10ulp[
Workflow:

- Metagenomic assembly

- Open Reading Frames (ORF) prediction

- Annotation step performed by homology detection (at proteomic level)

- Reconstruction of metabolic pathways is achieved from enzyme-coding genes 
]]

.pull-right[
&lt;img src="images/metagenomicsPipeline.png" width="380px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Roumpeka, Wallace, Escalettes, Fotheringham, and Watson, 2017)]]
]

---

.font120[**Functional Metagenomics Analysis**]

.pull-left[
.m10ulp[
Workflow:

- Metagenomic assembly

- Open Reading Frames (ORF) prediction

- Annotation step performed by homology detection (at proteomic level)

- Reconstruction of metabolic pathways is achieved from enzyme-coding genes 
]]

.pull-right[
&lt;img src="images/functionalMetagenomics.png" width="370px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Castañeda and Barbosa, 2017)]]
]

---
layout: true

# What are they doing?

---

class: inverse, center, middle

----

---

.white[Metatranscriptomics]

&lt;img src="images/Meta1.png" width="620px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Vanwonterghem, Jensen, Ho, Batstone, and Tyson, 2014)]]

---

Metatranscriptomics

&lt;img src="images/Meta2.png" width="620px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Vanwonterghem, Jensen, Ho, et al., 2014)]]

---

Metatranscriptomics
.pull-left[
.h25[
- Metatranscriptomics is the science that studies gene expression of microbes within natural environments

- It allows to obtain whole gene expression profiling of complex microbial communities

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; The advantage of metatranscriptomics, over metagenomics, is that it can provide information about differences in the active functions of microbial communities 

-  The overview of the gene expression in a given sample is obtained by capturing the total mRNA of the microbiome and by performing a whole metatranscriptomics shotgun sequencing.
]
]

.pull-right[
.m0p[
&lt;img src="images/metatrranscriptomePL.png" width="380px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Pérez-Pantoja and Tamames, 2015)]]
]
]

---

Metaproteomics

&lt;img src="images/Meta3.png" width="620px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Vanwonterghem, Jensen, Ho, et al., 2014)]]

---

Metaproteomics

.pull-left[
.h25[
- Metaproteomics is the study of all protein samples recovered directly from environmental sources

- Metaproteomics have rapidly increased in recent years due to many technological advances in mass spectrometry (MS)

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; Metatranscriptomics assess the “expressed” metabolism and physiology of microbial community members, but also allows quantification of per-species biomass to determine community structure

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; Proteomics quantifies the real proteins expression level, while mRNA may be degraded or translated inefficiently, resulting in a incorrect estimation of protein expression level
]
]

.pull.right[
&lt;img src="images/metaproteomics.png" width="480px" style="display: block; margin: auto;" /&gt;
.right[.font30[https://www.ecu.edu.au/schools/science/research-activity/research-support/proteomics/what-is-lc-ms-proteomics]]
]

---

Metabolomics

&lt;img src="images/Meta4.png" width="600px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Vanwonterghem, Jensen, Ho, et al., 2014)]]

---

Metabolomics

.pull-left[
.m0p[
.h25[

- Metabolomics is the study of the complete set of metabolites composition (the metabolome) of a cell type, tissue, or biological fluid

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; The metabolites (&lt;1.5 kDa) are the intermediates or end products of multiple enzymatic reactions and therefore are the most informative proxies of the biochemical activity of an organism

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm121.6 313.1c4.7 4.7 4.7 12.3 0 17L338 377.6c-4.7 4.7-12.3 4.7-17 0L256 312l-65.1 65.6c-4.7 4.7-12.3 4.7-17 0L134.4 338c-4.7-4.7-4.7-12.3 0-17l65.6-65-65.6-65.1c-4.7-4.7-4.7-12.3 0-17l39.6-39.6c4.7-4.7 12.3-4.7 17 0l65 65.7 65.1-65.6c4.7-4.7 12.3-4.7 17 0l39.6 39.6c4.7 4.7 4.7 12.3 0 17L312 256l65.6 65.1z"/&gt;&lt;/svg&gt; The metabolome consists of multiple compounds. Determine the entire set of metabolites is extremely difficult, further complicating the analyses is the dynamic nature of these metabolites.
]
]]

.pull-rigth[
&lt;br&gt;&lt;br&gt;
&lt;img src="images/metabolomics.png" width="480px" style="display: block; margin: auto;" /&gt;
]
.right[.font50[(Alonso, Marsal, and Julià, 2015)]]


???

.n30[
- hydrophilic
  carbohydrates, volatile alcohols, ketones, amino and non- amino organic acids, hydrophobic lipids and complex natural compounds (secondary metabolites such as antibiotics, pigments, non-ribosomal peptides, cofactors)
]

---

Stable/radioactive isotopes

&lt;img src="images/Meta5.png" width="600px" style="display: block; margin: auto;" /&gt;
.right[.font50[(Vanwonterghem, Jensen, Ho, et al., 2014)]]

---

Stable/radioactive isotopes

.pull-left[
.m0p[
.h25[

- Stable-isotope probing (SIP) is a technique in microbial ecology for tracing fluxes of nutrients in biogeochemical cycling by microorganisms using heavy stable isotopes

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; SIP is an approach that can identify microbial populations with a defined function

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:green;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"/&gt;&lt;/svg&gt; The use of either DNA-SIP, RNA-SIP or protein-SIP can enable both the phylogenetic identification and the key metabolic genes/proteins

- &lt;svg style="height:0.8em;top:.04em;position:relative;fill:red;" viewBox="0 0 512 512"&gt;&lt;path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm121.6 313.1c4.7 4.7 4.7 12.3 0 17L338 377.6c-4.7 4.7-12.3 4.7-17 0L256 312l-65.1 65.6c-4.7 4.7-12.3 4.7-17 0L134.4 338c-4.7-4.7-4.7-12.3 0-17l65.6-65-65.6-65.1c-4.7-4.7-4.7-12.3 0-17l39.6-39.6c4.7-4.7 12.3-4.7 17 0l65 65.7 65.1-65.6c4.7-4.7 12.3-4.7 17 0l39.6 39.6c4.7 4.7 4.7 12.3 0 17L312 256l65.6 65.1z"/&gt;&lt;/svg&gt; A major challenge associated with SIP is a very limited availability and high cost of labeled substrates. SIP is also very labor-intensive and low-throughput using current techniques
]
]]

.pull-rigth[
&lt;img src="images/sip1.png" width="280px" style="display: block; margin: auto;" /&gt;
]
.right[.font50[(Uhlik, Leewis, Strejcek, Musilova, Mackova, Leigh, and Macek, 2013)]]

---

Stable/radioactive isotopes

.m0p[
&lt;img src="images/sip2.png" width="570px" style="display: block; margin: auto;" /&gt;
.right[.font50[(el Zahar Haichar, Heulin, Guyonnet, et al., 2016)]]
]

---
layout: false
class: clear, middle, center

.m0p[
&lt;img src="images/sb1.png" width="850px" style="display: block; margin: auto;" /&gt;
]

???
metafora della torta e dei suoi ingredienti

---
class: clear


.center[.font150[System biology]]

.pull-left[
.m0p[
.h25u[
- .font80[Systems biology studies biological systems by systematically perturbing them]    

- .font80[Monitoring the gene, protein, metabolite, and informational pathway responses]    

- .font80[Integrating these data]    

- .font80[Formulating mathematical models that describe the structure of the system]    

- .font80[Predict its response to individual perturbations]
  ]]]

.pull-right[
.m0p[
&lt;img src="images/sb2.png" width="670px" style="display: block; margin: auto;" /&gt;

]
]

---

layout: true

# Questions about the lesson


---


.m0tbp[
.center[.font50px[&lt;svg style="height:0.8em;top:.04em;position:relative;fill:black;" viewBox="0 0 512 512"&gt;&lt;path d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zM262.655 90c-54.497 0-89.255 22.957-116.549 63.758-3.536 5.286-2.353 12.415 2.715 16.258l34.699 26.31c5.205 3.947 12.621 3.008 16.665-2.122 17.864-22.658 30.113-35.797 57.303-35.797 20.429 0 45.698 13.148 45.698 32.958 0 14.976-12.363 22.667-32.534 33.976C247.128 238.528 216 254.941 216 296v4c0 6.627 5.373 12 12 12h56c6.627 0 12-5.373 12-12v-1.333c0-28.462 83.186-29.647 83.186-106.667 0-58.002-60.165-102-116.531-102zM256 338c-25.365 0-46 20.635-46 46 0 25.364 20.635 46 46 46s46-20.636 46-46c0-25.365-20.635-46-46-46z"/&gt;&lt;/svg&gt;]]
]

.center[.font90[[&lt;svg style="height:0.8em;top:.04em;position:relative;fill:steelblue;" viewBox="0 0 512 512"&gt;&lt;path d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"/&gt;&lt;/svg&gt;](https://twitter.com/marpello1980) @marpello1980 - [&lt;svg style="height:0.8em;top:.04em;position:relative;fill:steelblue;" viewBox="0 0 512 512"&gt;&lt;path d="M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7 22.4 17.4 52.1 39.5 154.1 113.6 21.1 15.4 56.7 47.8 92.2 47.6 35.7.3 72-32.8 92.3-47.6 102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4 132.7-96.3 142.8-104.7 173.4-128.7 5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9 30.6 23.9 40.7 32.4 173.4 128.7 16.8 12.2 50.2 41.8 73.4 41.4z"/&gt;&lt;/svg&gt;](mailto:chiapello.m@gmail.com) chiapello.m@gmail.com - &lt;svg style="height:0.8em;top:.04em;position:relative;fill:steelblue;" viewBox="0 0 448 512"&gt;&lt;path d="M424.7 299.8c2.9-14 4.7-28.9 4.7-43.8 0-113.5-91.9-205.3-205.3-205.3-14.9 0-29.7 1.7-43.8 4.7C161.3 40.7 137.7 32 112 32 50.2 32 0 82.2 0 144c0 25.7 8.7 49.3 23.3 68.2-2.9 14-4.7 28.9-4.7 43.8 0 113.5 91.9 205.3 205.3 205.3 14.9 0 29.7-1.7 43.8-4.7 19 14.6 42.6 23.3 68.2 23.3 61.8 0 112-50.2 112-112 .1-25.6-8.6-49.2-23.2-68.1zm-194.6 91.5c-65.6 0-120.5-29.2-120.5-65 0-16 9-30.6 29.5-30.6 31.2 0 34.1 44.9 88.1 44.9 25.7 0 42.3-11.4 42.3-26.3 0-18.7-16-21.6-42-28-62.5-15.4-117.8-22-117.8-87.2 0-59.2 58.6-81.1 109.1-81.1 55.1 0 110.8 21.9 110.8 55.4 0 16.9-11.4 31.8-30.3 31.8-28.3 0-29.2-33.5-75-33.5-25.7 0-42 7-42 22.5 0 19.8 20.8 21.8 69.1 33 41.4 9.3 90.7 26.8 90.7 77.6 0 59.1-57.1 86.5-112 86.5z"/&gt;&lt;/svg&gt; marpello]]

---
    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="libs/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"ratio": "16:9",
"highlightStyle": "github",
"highlightLines": true
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
